<?xml version="1.0" encoding="UTF-8"?>
<div class="TEXT" chunkType="TEXT" id="t.1.0" xmlns="http://www.w3.org/1999/xhtml">
 <span>
  <b> [g.1.1] </b>
 </span>
 <div id="g.1.1" last="g.1.0" next="g.1.2">
  <svg width="1000.0" height="800.0" xmlns="http://www.w3.org/2000/svg">
   <g transform="matrix(0.5,0.0,0.0,0.5,0.0,0.0)">
    <path stroke="black" fill="#1999ff" d="M466.498 60.844 C463.13 55.314 462.944 48.129 466.624 42.27 C471.809 34.019 482.558 31.662 490.811 36.848 C493.799 38.725 495.939 42.501 495.939 42.501 C494.776 40.651 493.188 39.024 491.213 37.784 C484.182 33.367 474.902 35.487 470.485 42.518 C467.318 47.56 467.512 53.76 470.469 58.492 L466.498 60.844 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M473.437 48.272 L475.467 48.272 C477.024 48.272 478.098 48.777 478.098 50.162 C478.098 51.193 477.496 51.709 476.551 51.945 L476.551 51.966 C477.572 52.052 478.291 52.718 478.291 53.771 C478.291 55.381 476.959 55.768 475.434 55.768 L473.437 55.768 L473.437 48.272 ZM474.511 51.472 L475.102 51.472 C476.143 51.472 477.024 51.376 477.024 50.269 C477.024 49.378 476.261 49.152 475.639 49.152 L474.511 49.152 L474.511 51.472 ZM474.511 54.887 L475.316 54.887 C476.283 54.887 477.153 54.683 477.153 53.674 C477.153 52.514 476.133 52.353 475.209 52.353 L474.511 52.353 L474.511 54.887 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M480.074 48.272 L481.846 48.272 L484.198 54.501 L486.507 48.272 L488.279 48.272 L488.279 55.768 L487.205 55.768 L487.205 49.281 L487.183 49.281 L484.702 55.768 L483.65 55.768 L481.169 49.281 L481.148 49.281 L481.148 55.768 L480.074 55.768 L480.074 48.272 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M495.377 49.507 C494.904 49.228 494.379 49.088 493.831 49.088 C492.102 49.088 490.952 50.334 490.952 52.02 C490.952 53.781 492.22 54.952 493.777 54.952 C494.314 54.952 495.034 54.812 495.41 54.587 L495.474 55.628 C494.904 55.865 494.206 55.897 493.766 55.897 C491.393 55.897 489.814 54.372 489.814 51.988 C489.814 49.679 491.489 48.143 493.777 48.143 C494.357 48.143 494.926 48.229 495.463 48.465 L495.377 49.507 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M473.512 61.082 L475.284 61.082 L477.636 67.311 L479.945 61.082 L481.717 61.082 L481.717 68.578 L480.643 68.578 L480.643 62.092 L480.621 62.092 L478.141 68.578 L477.088 68.578 L474.607 62.092 L474.586 62.092 L474.586 68.578 L473.512 68.578 L473.512 61.082 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M484.745 61.877 L483.607 61.877 L483.607 60.739 L484.745 60.739 L484.745 61.877 M483.671 63.101 L484.68 63.101 L484.68 68.578 L483.671 68.578 L483.671 63.101 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M502.035 60.524 L503.044 60.524 L503.044 63.853 L503.066 63.853 C503.377 63.349 503.925 62.973 504.881 62.973 C506.47 62.973 507.244 64.272 507.244 65.744 C507.244 67.247 506.545 68.707 504.892 68.707 C503.914 68.707 503.292 68.213 503.023 67.806 L503.002 67.806 L503.002 68.579 L502.035 68.579 L502.035 60.524 ZM504.666 63.789 C503.571 63.789 503.044 64.831 503.044 65.851 C503.044 66.764 503.517 67.891 504.655 67.891 C505.74 67.891 506.191 66.678 506.17 65.818 C506.191 64.831 505.762 63.789 504.666 63.789 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M509.799 61.877 L508.66 61.877 L508.66 60.739 L509.799 60.739 L509.799 61.877 M508.725 63.101 L509.734 63.101 L509.734 68.578 L508.725 68.578 L508.725 63.101 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M518.273 68.578 L519.282 68.578 L519.282 60.523 L518.273 60.523 L518.273 68.578 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M490.007 64.025 C489.62 63.864 489.234 63.788 488.912 63.788 C487.784 63.788 487.15 64.798 487.15 65.84 C487.15 66.817 487.677 67.891 488.976 67.891 C489.32 67.891 489.749 67.784 490.061 67.58 L490.136 68.481 C489.717 68.675 489.234 68.707 488.869 68.707 C487.215 68.707 486.076 67.408 486.076 65.84 C486.076 64.175 487.193 62.973 488.869 62.973 C489.277 62.973 489.803 63.069 490.093 63.187 L490.007 64.025 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M491.296 63.101 L492.241 63.101 L492.241 63.949 L492.262 63.949 C492.531 63.38 493.111 62.972 493.691 62.972 C493.991 62.972 494.142 63.004 494.357 63.047 L494.357 63.949 C494.174 63.874 493.97 63.853 493.798 63.853 C492.896 63.853 492.305 64.712 492.305 66.086 L492.305 68.578 L491.296 68.578 L491.296 63.101 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M497.9 62.973 C499.565 62.973 500.682 64.176 500.682 65.84 C500.682 67.408 499.543 68.707 497.9 68.707 C496.246 68.707 495.108 67.408 495.108 65.84 C495.108 64.176 496.225 62.973 497.9 62.973 ZM497.9 67.892 C499.082 67.892 499.608 66.818 499.608 65.84 C499.608 64.798 498.974 63.789 497.9 63.789 C496.816 63.789 496.182 64.798 496.182 65.84 C496.182 66.818 496.708 67.892 497.9 67.892 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M514.009 62.973 C515.674 62.973 516.791 64.176 516.791 65.84 C516.791 67.408 515.652 68.707 514.009 68.707 C512.355 68.707 511.217 67.408 511.217 65.84 C511.217 64.176 512.333 62.973 514.009 62.973 ZM514.009 67.892 C515.191 67.892 515.717 66.818 515.717 65.84 C515.717 64.798 515.082 63.789 514.009 63.789 C512.924 63.789 512.291 64.798 512.291 65.84 C512.291 66.818 512.817 67.892 514.009 67.892 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M523.556 62.973 C525.221 62.973 526.337 64.176 526.337 65.84 C526.337 67.408 525.199 68.707 523.556 68.707 C521.902 68.707 520.764 67.408 520.764 65.84 C520.764 64.176 521.881 62.973 523.556 62.973 ZM523.556 67.892 C524.737 67.892 525.264 66.818 525.264 65.84 C525.264 64.798 524.63 63.789 523.556 63.789 C522.471 63.789 521.837 64.798 521.837 65.84 C521.837 66.818 522.364 67.892 523.556 67.892 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M532.534 68.342 C532.534 70.006 531.675 70.962 529.71 70.962 C529.13 70.962 528.625 70.887 527.744 70.586 L527.852 69.652 C528.625 70.006 529.087 70.146 529.828 70.146 C530.891 70.146 531.525 69.426 531.525 68.267 L531.525 67.73 L531.503 67.73 C531.073 68.299 530.365 68.578 529.667 68.578 C528.099 68.578 527.326 67.3 527.326 65.883 C527.326 64.476 528.12 62.972 529.774 62.972 C530.741 62.972 531.278 63.326 531.578 63.917 L531.6 63.917 L531.6 63.101 L532.534 63.101 L532.534 68.342 ZM531.525 65.904 C531.525 64.766 530.998 63.788 529.924 63.788 C528.904 63.788 528.399 64.894 528.399 65.828 C528.399 66.859 528.979 67.762 529.924 67.762 C530.869 67.762 531.525 66.924 531.525 65.904 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M533.844 70.06 C533.973 70.103 534.145 70.146 534.317 70.146 C535.219 70.146 535.519 68.901 535.519 68.793 C535.519 68.675 535.39 68.396 535.326 68.213 L533.468 63.101 L534.574 63.101 L536.046 67.633 L536.067 67.633 L537.571 63.101 L538.591 63.101 L536.572 68.697 C536.196 69.749 535.799 70.963 534.467 70.963 C534.145 70.963 533.962 70.919 533.78 70.887 L533.844 70.06 " stroke-width="1.0"/>
   </g>
  </svg>
 </div>
 <span>
  <b> [g.1.2] </b>
 </span>
 <div id="g.1.2" last="g.1.1" next="g.1.3">
  <div>
   <p>
    <b>RESEARCH ARTICLE Open Access</b>
   </p>
  </div>
  <div>
   <div>
    <svg width="1000.0" height="800.0" xmlns="http://www.w3.org/2000/svg">
     <g transform="matrix(0.5,0.0,0.0,0.5,0.0,0.0)">
      <path stroke="black" fill="#1f6fb2" d="M56.693 91.936 L56.75 91.426 L56.863 90.916 L57.09 90.462 L57.373 90.009 L57.77 89.668 L58.167 89.385 L58.677 89.158 L59.187 88.988 L536.088 88.988 C537.225 89.322 537.96 89.782 538.413 90.916 L538.526 91.426 L538.583 91.936 L538.583 102.934 L538.526 103.445 C538.461 104.388 537.495 105.458 536.598 105.712 L536.088 105.883 L59.187 105.883 L58.677 105.712 L58.167 105.486 L57.77 105.202 L57.373 104.805 L57.09 104.408 L56.863 103.955 L56.75 103.445 L56.693 102.934 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M56.693 91.936 L56.75 91.426 C56.964 90.063 57.913 89.332 59.187 88.988 L59.698 88.988 L59.698 89.952 L59.358 90.009 C58.516 90.065 57.889 90.814 57.714 91.596 L57.714 91.936 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M56.693 91.936 L56.75 91.426 C56.964 90.063 57.913 89.332 59.187 88.988 L59.698 88.988 L59.698 89.952 L59.358 90.009 C58.516 90.065 57.889 90.814 57.714 91.596 L57.714 91.936 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M59.698 88.988 L535.578 88.988 L535.578 89.952 L59.698 89.952 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M59.698 88.988 L535.578 88.988 L535.578 89.952 L59.698 89.952 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M535.578 88.988 L536.088 88.988 C537.362 89.332 538.312 90.063 538.526 91.426 L538.583 91.936 L537.562 91.936 L537.562 91.596 C537.387 90.814 536.76 90.065 535.918 90.008 L535.578 89.952 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M535.578 88.988 L536.088 88.988 C537.362 89.332 538.312 90.063 538.526 91.426 L538.583 91.936 L537.562 91.936 L537.562 91.596 C537.387 90.814 536.76 90.065 535.918 90.008 L535.578 89.952 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M56.693 102.934 L56.693 91.936 L57.713 91.936 L57.713 102.934 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M56.693 102.934 L56.693 91.936 L57.713 91.936 L57.713 102.934 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M538.583 91.936 L538.583 102.934 L537.563 102.934 L537.563 91.936 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M538.583 91.936 L538.583 102.934 L537.563 102.934 L537.563 91.936 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M59.698 105.882 L59.188 105.882 C57.983 105.606 56.917 104.699 56.75 103.444 L56.693 102.934 L57.714 102.934 L57.714 103.274 C57.889 104.056 58.516 104.805 59.358 104.861 L59.698 104.918 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M59.698 105.882 L59.188 105.882 C57.983 105.606 56.917 104.699 56.75 103.444 L56.693 102.934 L57.714 102.934 L57.714 103.274 C57.889 104.056 58.516 104.805 59.358 104.861 L59.698 104.918 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M537.562 104.918 L57.713 104.918 L57.713 102.934 L537.562 102.934 L537.562 104.918 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M537.562 104.918 L57.713 104.918 L57.713 102.934 L537.562 102.934 L537.562 104.918 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M538.583 102.934 L538.526 103.444 C538.36 104.699 537.293 105.606 536.088 105.882 L535.578 105.882 L535.578 104.918 L535.918 104.862 C536.76 104.805 537.387 104.056 537.562 103.274 L537.562 102.934 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M538.583 102.934 L538.526 103.444 C538.36 104.699 537.293 105.606 536.088 105.882 L535.578 105.882 L535.578 104.918 L535.918 104.862 C536.76 104.805 537.387 104.056 537.562 103.274 L537.562 102.934 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M535.578 105.882 L59.698 105.882 L59.698 104.918 L535.578 104.918 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M535.578 105.882 L59.698 105.882 L59.698 104.918 L535.578 104.918 " stroke-width="1.0"/>
     </g>
    </svg>
   </div>
  </div>
 </div>
 <span>
  <b> [g.1.3] </b>
 </span>
 <div id="g.1.3" last="g.1.2" next="g.1.4">
  <p>Factors influencing lysis time stochasticity in bacteriophage λ</p>
 </div>
 <span>
  <b> [g.1.4] </b>
 </span>
 <p id="g.1.4" last="g.1.3" next="g.1.5">John J Dennehy
  <sup>1,2*</sup>and Ing-Nang Wang
  <sup>1</sup>
 </p>
 <span>
  <b> [g.1.6] </b>
 </span>
 <div id="g.1.6" last="g.1.5" next="g.1.7">
  <p>
   <b>Background Some phenotypic variation arises from randomness in cellular processes despite identical environments and genotypes [1-9]. Population heterogeneity, resulting from such molecular stochasticity, has been documented in many microbial organisms including bacteriophage (phage) λ [10-13], </b>
   <i>Escherichia coli</i> [14-16], 
   <i>Bacillus subtilis </i>[17,18]
   <i> </i>and
   <i> Saccharomyces cerevisiae </i>[19-24].
   <i> </i>This within-population variation can have far reaching life history consequences. For example, experimentally reducing noise in the expression of ComK decreased the number of competent 
   <i>B. subtilis</i> cells in one study [18]. In another study, mutants of 
   <i>S. cerevisiae</i> showing greater heterogeneity in survival had higher rates of occasional-cell survival during high stress conditions than did wild-type cells [25].
  </p>
 </div>
 <span>
  <b> [g.1.7] </b>
 </span>
 <div id="g.1.7" last="g.1.6" next="g.1.8">
  <div>
   <div>
    <p>* Correspondence: john.dennehy@qc.cuny.edu</p>
    <p>
     <sup>1</sup>Department of Biological Sciences, University at Albany, 1400 Washington Avenue, Albany, NY 12222, USA Full list of author information is available at the end of the article
    </p>
   </div>
  </div>
  <div>
   <div>
    <svg width="1000.0" height="800.0" xmlns="http://www.w3.org/2000/svg">
     <g transform="matrix(0.5,0.0,0.0,0.5,0.0,0.0)">
      <path fill="none" stroke="#000000" d="M56.693 663.57 L290.551 663.57 " stroke-width="1.0"/>
     </g>
    </svg>
   </div>
  </div>
 </div>
 <span>
  <b> [g.1.8] </b>
 </span>
 <div id="g.1.8" last="g.1.7" next="g.1.9">
  <p>Because of their simplicity and ease of manipulation, phages are excellent models to explore the life history consequences of molecular stochasticity. Many phages use a “ holin-endolysin” system to compromise two physical barriers, the cell membrane and the peptidoglycan layer, in order to lyse an infected host cell [26,27]. Although there are some variations on the theme, holin usually forms a hole(s) in the inner membrane, thus either allowing soluble endolysin into the periplasmic space [28,29] or activating the membrane-tethered endolysin already translocated to the periplasm [30-32]. Endolysin then digests the peptidoglycan, causing host cell lysis.</p>
  <p>The most extensively studied lysis system is that of phage l, which consists of four genes: 
   <i>S</i> (encodes holin and antiholin), 
   <i>R</i> (encodes endolysin), 
   <i>Rz</i>, and 
   <i>Rz1 (encode an integral inner membrane protein and an outer membrane lipoprotein, respectively). All genes are co-transcribed from the late promoter </i>
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>during
   <i> </i>the late phase of the lytic cycle [26,27,33,34]. Under typical
  </p>
 </div>
 <span>
  <b> [g.1.9] </b>
 </span>
 <div id="g.1.9" last="g.1.8" next="g.1.10">
  <svg width="1000.0" height="800.0" xmlns="http://www.w3.org/2000/svg">
   <g transform="matrix(0.5,0.0,0.0,0.5,0.0,0.0)">
    <path stroke="black" fill="#15ffea" d="M73.07 710.795 C76.575 712.512 78.982 716.065 78.982 720.168 C78.982 725.952 74.234 730.53 68.338 730.53 C66.203 730.53 63.891 729.306 63.891 729.306 C65.083 729.87 66.42 730.184 67.831 730.184 C72.854 730.184 76.925 726.191 76.925 721.264 C76.925 717.73 74.831 714.676 71.793 713.233 L73.07 710.795 " stroke-width="1.0"/>
    <path stroke="black" fill="#aef9ff" d="M62.606 731.745 C59.101 730.029 56.693 726.476 56.693 722.371 C56.693 716.587 61.441 712.011 67.338 712.011 C69.473 712.011 71.785 713.233 71.785 713.233 C70.593 712.671 69.256 712.355 67.845 712.355 C62.822 712.355 58.751 716.349 58.751 721.276 C58.751 724.81 60.845 727.864 63.884 729.309 L62.606 731.745 " stroke-width="1.0"/>
    <path stroke="black" fill="#aef9ff" d="M86.103 723.938 L86.95 723.938 C87.542 723.938 88.183 723.724 88.183 723.083 C88.183 722.373 87.578 722.242 86.973 722.242 L86.103 722.242 L86.103 723.938 ZM86.103 720.534 L86.913 720.534 C87.481 720.534 88.098 720.368 88.098 719.739 C88.098 719.122 87.566 718.933 86.985 718.933 L86.103 718.933 L86.103 720.534 ZM83.056 717.368 L88.364 717.368 C89.827 717.368 91.266 717.829 91.266 719.407 C91.266 720.546 90.649 721.151 89.525 721.435 L89.525 721.46 C90.698 721.614 91.423 722.514 91.423 723.345 C91.423 725.432 89.706 725.647 87.953 725.647 L83.056 725.647 L83.056 717.368 " stroke-width="1.0"/>
    <path stroke="black" fill="#aef9ff" d="M91.859 719.467 L94.906 719.467 L94.906 725.647 L91.859 725.647 L91.859 719.467 M94.906 718.814 L91.859 718.814 L91.859 716.751 L94.906 716.751 L94.906 718.814 " stroke-width="1.0"/>
    <path stroke="black" fill="#15ffea" d="M103.803 717.368 L108.591 717.368 L109.692 721.898 L109.716 721.898 L110.804 717.368 L115.58 717.368 L115.58 725.647 L112.679 725.647 L112.63 719.146 L112.606 719.146 L110.95 725.647 L108.253 725.647 L106.644 719.146 L106.596 719.146 L106.559 725.647 L103.803 725.647 L103.803 717.368 " stroke-width="1.0"/>
    <path stroke="black" fill="#15ffea" d="M126.977 721.175 C126.336 721.175 126.227 721.815 126.227 722.456 C126.227 723.107 126.336 723.796 127.002 723.796 C127.654 723.796 127.751 723.107 127.751 722.526 C127.751 721.815 127.654 721.175 126.977 721.175 ZM127.884 725.647 L127.884 724.519 L127.86 724.519 C127.437 725.445 126.82 725.788 125.828 725.788 C123.809 725.788 123.059 724.247 123.059 722.456 C123.059 720.784 123.845 719.325 125.744 719.325 C126.554 719.325 127.17 719.597 127.643 720.262 L127.666 720.262 L127.666 716.751 L130.713 716.751 L130.713 725.647 L127.884 725.647 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M140.138 725.13 C139.728 725.347 138.944 725.552 137.955 725.552 C135.989 725.552 134.108 724.286 134.108 721.404 C134.108 718.991 135.663 717.158 138.22 717.158 C139.257 717.158 139.861 717.387 140.114 717.52 L139.921 718.002 C139.511 717.798 138.92 717.653 138.232 717.653 C136.037 717.653 134.699 719.076 134.699 721.415 C134.699 723.622 135.965 725.046 138.136 725.046 C138.835 725.046 139.511 724.901 139.969 724.672 L140.138 725.13 " stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M140.138 725.13 C139.728 725.347 138.944 725.552 137.955 725.552 C135.989 725.552 134.108 724.286 134.108 721.404 C134.108 718.991 135.663 717.158 138.22 717.158 C139.257 717.158 139.861 717.387 140.114 717.52 L139.921 718.002 C139.511 717.798 138.92 717.653 138.232 717.653 C136.037 717.653 134.699 719.076 134.699 721.415 C134.699 723.622 135.965 725.046 138.136 725.046 C138.835 725.046 139.511 724.901 139.969 724.672 L140.138 725.13 Z" stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M165.627 725.42 L166.194 725.42 L166.194 716.857 L165.627 716.857 L165.627 725.42 " stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M165.627 716.857 L166.194 716.857 L166.194 725.42 L165.627 725.42 L165.627 716.857 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M154.136 718.352 L154.136 719.63 L155.788 719.63 L155.788 720.089 L154.136 720.089 L154.136 723.852 C154.136 724.587 154.365 725.058 154.98 725.058 C155.282 725.058 155.499 725.009 155.644 724.961 L155.716 725.407 C155.523 725.492 155.258 725.552 154.908 725.552 C154.486 725.552 154.136 725.407 153.919 725.13 C153.654 724.829 153.557 724.346 153.557 723.755 L153.557 720.089 L152.58 720.089 L152.58 719.63 L153.557 719.63 L153.557 718.569 L154.136 718.352 " stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M154.136 718.352 L154.136 719.63 L155.788 719.63 L155.788 720.089 L154.136 720.089 L154.136 723.852 C154.136 724.587 154.365 725.058 154.98 725.058 C155.282 725.058 155.499 725.009 155.644 724.961 L155.716 725.407 C155.523 725.492 155.258 725.552 154.908 725.552 C154.486 725.552 154.136 725.407 153.919 725.13 C153.654 724.829 153.557 724.346 153.557 723.755 L153.557 720.089 L152.58 720.089 L152.58 719.63 L153.557 719.63 L153.557 718.569 L154.136 718.352 Z" stroke-width="1.0"/>
    <path stroke="black" fill="#aef9ff" d="M99.041 724.01 C99.634 724.01 99.718 723.346 99.718 722.444 C99.718 721.709 99.634 721.104 99.041 721.104 C98.449 721.104 98.376 721.709 98.376 722.444 C98.376 723.346 98.449 724.01 99.041 724.01 ZM99.041 719.326 C101.58 719.326 102.959 720.369 102.959 722.444 C102.959 724.58 101.58 725.789 99.041 725.789 C96.49 725.789 95.136 724.58 95.136 722.444 C95.136 720.369 96.49 719.326 99.041 719.326 " stroke-width="1.0"/>
    <path stroke="black" fill="#15ffea" d="M120.355 721.969 C120.355 721.494 120.283 720.748 119.594 720.748 C118.892 720.748 118.735 721.484 118.772 721.969 L120.355 721.969 ZM118.784 723.274 C118.832 724.045 119.569 724.223 120.464 724.223 C121.298 724.223 122.049 724.081 122.532 723.831 L122.532 725.432 C121.722 725.681 120.754 725.788 119.69 725.788 C117.139 725.788 115.785 724.579 115.785 722.444 C115.785 720.369 117.139 719.325 119.69 719.325 C122.133 719.325 122.991 720.819 122.991 722.906 L122.991 723.274 L118.784 723.274 " stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M145.117 722.007 C145.129 721.199 144.792 719.969 143.392 719.969 C142.114 719.969 141.56 721.115 141.463 722.007 L145.117 722.007 ZM141.427 722.465 C141.427 724.323 142.44 725.046 143.634 725.046 C144.478 725.046 144.912 724.877 145.238 724.72 L145.382 725.179 C145.165 725.3 144.574 725.553 143.561 725.553 C141.897 725.553 140.86 724.335 140.86 722.634 C140.86 720.704 141.969 719.499 143.465 719.499 C145.358 719.499 145.72 721.283 145.72 722.116 C145.72 722.272 145.72 722.369 145.696 722.465 L141.427 722.465 " stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M141.427 722.465 C141.427 724.322 142.44 725.046 143.634 725.046 C144.478 725.046 144.912 724.877 145.238 724.72 L145.383 725.179 C145.166 725.3 144.575 725.552 143.562 725.552 C141.897 725.552 140.86 724.334 140.86 722.634 C140.86 720.704 141.97 719.498 143.465 719.498 C145.359 719.498 145.721 721.283 145.721 722.115 C145.721 722.272 145.721 722.369 145.697 722.465 L141.427 722.465 Z" stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M147.034 721.018 C147.034 720.487 147.01 720.089 146.986 719.631 L147.517 719.631 L147.565 720.68 L147.589 720.68 C147.902 720.029 148.626 719.498 149.579 719.498 C150.134 719.498 151.593 719.776 151.593 721.947 L151.593 725.42 L151.026 725.42 L151.026 721.995 C151.026 720.945 150.616 719.993 149.422 719.993 C148.602 719.993 147.915 720.572 147.685 721.319 C147.637 721.464 147.601 721.657 147.601 721.838 L147.601 725.42 L147.034 725.42 L147.034 721.018 " stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M147.034 721.018 C147.034 720.487 147.01 720.089 146.986 719.631 L147.517 719.631 L147.565 720.68 L147.589 720.68 C147.902 720.029 148.626 719.498 149.579 719.498 C150.134 719.498 151.593 719.776 151.593 721.947 L151.593 725.42 L151.026 725.42 L151.026 721.995 C151.026 720.945 150.616 719.993 149.422 719.993 C148.602 719.993 147.915 720.572 147.685 721.319 C147.637 721.464 147.601 721.657 147.601 721.838 L147.601 725.42 L147.034 725.42 L147.034 721.018 Z" stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M156.909 721.367 C156.909 720.788 156.885 720.185 156.861 719.63 L157.391 719.63 L157.415 720.8 L157.44 720.8 C157.693 720.077 158.296 719.498 159.092 719.498 C159.164 719.498 159.249 719.51 159.321 719.522 L159.321 720.077 C159.237 720.053 159.152 720.053 159.044 720.053 C158.248 720.053 157.681 720.74 157.524 721.669 C157.5 721.837 157.476 722.03 157.476 722.223 L157.476 725.419 L156.909 725.419 L156.909 721.367 " stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M156.909 721.367 C156.909 720.788 156.885 720.185 156.861 719.63 L157.391 719.63 L157.415 720.8 L157.44 720.8 C157.693 720.077 158.296 719.498 159.092 719.498 C159.164 719.498 159.249 719.51 159.321 719.522 L159.321 720.077 C159.237 720.053 159.152 720.053 159.044 720.053 C158.248 720.053 157.681 720.74 157.524 721.669 C157.5 721.837 157.476 722.03 157.476 722.223 L157.476 725.419 L156.909 725.419 L156.909 721.367 Z" stroke-width="1.0"/>
    <path stroke="black" fill="#000000" d="M163.337 722.32 C162.071 722.284 160.406 722.477 160.406 723.864 C160.406 724.696 160.961 725.07 161.564 725.07 C162.529 725.07 163.072 724.479 163.277 723.912 C163.313 723.791 163.337 723.671 163.337 723.562 L163.337 722.32 ZM163.904 724.045 C163.904 724.503 163.928 724.973 163.988 725.42 L163.47 725.42 L163.385 724.588 L163.349 724.588 C163.072 725.034 162.433 725.552 161.504 725.552 C160.346 725.552 159.804 724.732 159.804 723.96 C159.804 722.622 160.985 721.814 163.337 721.838 L163.337 721.693 C163.337 721.114 163.228 719.968 161.854 719.981 C161.347 719.981 160.817 720.113 160.406 720.415 L160.225 719.992 C160.744 719.643 161.396 719.498 161.914 719.498 C163.591 719.498 163.904 720.752 163.904 721.79 L163.904 724.045 " stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M163.904 724.045 C163.904 724.503 163.928 724.974 163.988 725.42 L163.47 725.42 L163.385 724.588 L163.349 724.588 C163.072 725.034 162.433 725.552 161.504 725.552 C160.346 725.552 159.804 724.732 159.804 723.96 C159.804 722.622 160.985 721.814 163.337 721.838 L163.337 721.693 C163.337 721.114 163.228 719.968 161.853 719.981 C161.347 719.981 160.816 720.113 160.406 720.415 L160.225 719.993 C160.744 719.643 161.396 719.498 161.914 719.498 C163.59 719.498 163.904 720.752 163.904 721.79 L163.904 724.045 Z" stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M145.117 722.007 C145.129 721.199 144.792 719.969 143.392 719.969 C142.114 719.969 141.56 721.115 141.463 722.007 L145.117 722.007 Z" stroke-width="1.0"/>
    <path fill="none" stroke="#000000" d="M163.337 722.32 C162.071 722.284 160.406 722.477 160.406 723.864 C160.406 724.696 160.961 725.07 161.564 725.07 C162.529 725.07 163.072 724.479 163.277 723.912 C163.313 723.791 163.337 723.671 163.337 723.562 L163.337 722.32 Z" stroke-width="1.0"/>
   </g>
  </svg>
 </div>
 <span>
  <b> [g.2.2] </b>
 </span>
 <div id="g.2.2" last="g.2.1" next="g.2.3">
  <p>laboratory conditions, only 
   <i>S</i> and 
   <i>R</i> are needed for host lysis, though both 
   <i>Rz</i> and 
   <i>Rz1</i> are essential in the presence of high concentrations of divalent cations [33-35].
  </p>
  <p>The lytic pathway of phage λ is commonly divided into the early, delayed early, and late phases. Transitions between stages are triggered by well-characterized molecular actions involving gene transcription and translation [36]. Consequently, the timing of when individual cells enter each phase greatly influences the length of individual lysis times. A recent study by Amir 
   <i>et al.</i> [10] showed that 69% of the total lysis time variance is due to variation in the time interval between the onset of the 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter
   <i> </i>and
   <i> </i>the
   <i> </i>eventual
   <i> </i>lysis
   <i> </i>(see
   <i> </i>APPENDIX A). This observation suggests that a large portion of the observed lysis time stochasticity is a 
   <i>de novo</i> phenomenon, confined to the production and accumulation of holin proteins in the cell membrane, rather than a direct carryover from the various upstream stochastic events.
  </p>
  <p>The formation of the λ holin hole in the membrane is hypothesized to be a multi-step process that starts with the transcription of the late mRNA and the translation of the S holin protein. The resulting holin monomers are then inserted into the cell membrane, where they dimerize, then oligomerize [37], eventually leading to the formation of higher-order holin aggregates, or rafts, in the cell membrane. At a time that is specific to the holin protein sequence, the holin rafts are transformed into a membrane lesion(s) &gt; 300 nm across [38], which is large enough for the passage of a 500 KDa protein [28,29]. Lysis ensues after endolysin digests the peptidoglycan. Thus, by
   <span>
    <b> [g.2.3] </b>
   </span>regulating endolysin’ s access to the peptidoglycan, holin controls the timing of lysis [26,27].
  </p>
 </div>
 <p>To formalize the heuristic model of holin hole formation described by Wang 
  <i>et al.</i> [28], Ryan and Rutenberg [39] proposed a two-stage nucleation model, in which the production rate of the holin monomers and holin self-affinity contribute to the aggregation of holin rafts. Raft aggregation is opposed by thermal Brownian motion which tends to disintegrate rafts into their holin constituents. As the rafts grow and then exceed a certain critical size (the first stage of nucleation), the probability of a second stage nucleation (triggering to hole formation) increases (Figure 1). According to this model, lysis time stochasticity is the inevitable outcome of each infected cell in the population following its own time course of growth in holin raft size. However, a recent study [40] using C-terminus GFP-fused λ S holin protein showed that, for most of the latent period, holin proteins are distributed uniformly in a relatively mobile state in the cell membrane. At a time that coincided with the triggering time, large immobile holin rafts suddenly appeared in the membrane. The transition from uniformly distributed holin to holin rafts occurred in less than a minute. Although it is not clear whether these large rafts correspond to the membrane holes observed by cryoelectron microscopy [38], this study nevertheless casts doubt on the previously hypothesized importance of holin raft size growth as the determining factor in lysis timing [28,39]. Rather, it is proposed that the lysis time is determined by when a critical holin concentration is reached in the cell membrane (Figure 1). According to this model, lysis time stochasticity is mainly the result of variation in
  <span>
   <b> [g.3.2] </b>
  </span>the timing of reaching the critical holin concentration in the membrane.
 </p>
 <p>Typically, the lysis time of a phage is estimated using a one-step growth curve [41-43]. In the case of phage λ , however, the availability of thermally-inducible 
  <i>E. coli</i> λ lysogens allows a more precise determination of the lysis time by following the decline of culture turbidity [26,44]. Direct observation of the lysis of individual λ lysogenic cells [45] confirmed that the precipitous decline of culture turbidity, commonly observed among thermallyinduced λ lysogen cultures, is a reflection of the saltatory nature of individual lysis events at the microscopic level. However, it is not clear to what extent the seemingly high synchronicity of lysis is influenced by various aspects of phage biology and host growth conditions. In this study, we used a simple experimental setup to assess how lysis time stochasticity is affected by allelic variation in the S protein, late promoter 
  <i>p</i>
  <sub>
   <i>R</i>
  </sub>
  <i>’ </i>activity,
  <i> </i>host
  <i> </i>growth rate, and the timing of energy poison KCN addition. Our
  <span>
   <b> [g.3.3] </b>
  </span>results establish the ranges and limits of lysis time stochasticity under various conditions.
 </p>
 <span>
  <b> [g.3.4] </b>
 </span>
 <div id="g.3.4" last="g.3.3" next="g.3.5">
  <p>
   <b>Results Using a microscope-mounted, temperature-controlled perfusion chamber, we observed and recorded individual lysis events of thermally-induced </b>
   <i>Escherichia coli</i> l lysogens (Figure 2A). These observations revealed a considerable amount of variation in lysis time for the wildtype (WT) λ phage (Table 1; Figure 2B). Although the mean lysis time for the WT λ phage was 65.1 min, lysis times for individual lysogenic cells ranged from 45.4 to 74.5 min. Given that phage progeny accumulate linearly at ~7.7 phage per minute beginning ~28 min after lysis induction [46], the ~30 min range of lysis times could result in a three-fold difference in burst size between phages that lyse early and those that lyse late. This result motivated further exploration of variation in lysis time among other λ strains.
  </p>
 </div>
 <span>
  <b> [g.4.3] </b>
 </span>
 <div id="g.4.3" last="g.4.2" next="g.4.4">
  <p>Effect of allelic variation in holin sequence It has long been known that different holin alleles show different lysis times [37,46,47]. However, it is not clear to what extent allelic differences in holin protein would affect the lysis timing of individual cells. To gain further insight, we determined the MLTs (mean lysis times) and SDs (standard deviations) of lysis time for 14 isogenic l lysogens differing in their 
   <i>S</i> holin sequences (see APPENDIX B for our rationale for using SD as the measure for lysis time stochasticity). The directly observed MLTs (Table 1) were longer than those reported previously [46]. This discrepancy was mainly due to the fact that, in previous work, lysis time was defined by the time point when the turbidity of the lysogen culture began to decline, whereas in our current measurement, it was the mean of all individual lysis times observed for a particular phage strain. Figure 3A revealed a significant positive relationship
  </p>
  <p>between MLT and SD (
   <i>F = 8.42, </i>
   <i>p</i> = 0.0133). How[1,12] ever, we did not observe a significant relationship
   <span>
    <b> [g.5.2] </b>
   </span>between MLT and another commonly used measure of stochasticity, the coefficient of variation (CV, defined as SD/MLT; [15,25,48,49]) (
   <i>F</i>
   <sub>[1,12]</sub>= 1.50, 
   <i>p</i> = 0.2445), indicating a proportional increase of the SD with the MLT. Figure 3A also reveals a relatively scattered relationship between the MLTs and the SDs (adjusted 
   <i>R</i>
   <sup>2</sup>= 0.363), with several instances in which strains with similar MLTs are accompanied by very different SDs. For example, the mean lysis times for IN56 and IN71 were 65.1 and 68.8 min, but the SDs were 3.2 and 7.7 min, respectively. Apparently the observed positive relationship is only a general trend, not an absolute. The scattering of the plot also suggests that different missense mutations in the holin sequence can influence MLT and SD somewhat independently.
  </p>
 </div>
 <span>
  <b> [g.5.3] </b>
 </span>
 <div id="g.5.3" last="g.5.2" next="g.5.4">
  <p>Effect of late promoter 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity Transcription of the late genes, including the holin gene S,
   <i> </i>from
   <i> </i>the
   <i> p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter
   <i> </i>marks
   <i> </i>the
   <i> </i>beginning
   <i> </i>of
   <i> </i>the late stage of the λ lytic development [36]. Since the major determinant of lysis time is thought to be when a critical holin concentration is reached in the cell membrane [40], reduced promoter activity should not only lengthen the lysis time, as shown in a previous study [50], but should also increase the lysis time stochasticity
  </p>
  <p>[51,52]. As shown in Figure 3B, our data showed a negative relationship between the 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity,
   <i> </i>and
   <i> </i>the
   <i> </i>MLTs, SDs, and CVs. However, the increase of the 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity had a diminishing influence on both the MLTs, as has been shown previously [50], and the associated SDs and CVs (see Table 2). Interestingly, linear regressions (Figure 3C) showed a much tighter, positive relationship between the MLTs and the SDs (
   <i>F</i>
   <sub>[1,3]</sub>= 81.04, 
   <i>p</i> = 0.0029; adjusted 
   <i>R</i>
   <sup>
    <i>2</i>
   </sup>= 0.952; 
   <i>y</i> = -15.7 + 0.3
   <i>x</i>) and a significant positive relationship between the MLTs and CVs (
   <i>F</i>
   <sub>[1,3]</sub>= 14.51, 
   <i>p</i> = 0.0318, result not shown in the figure). That is, for the WT 
   <i>S</i> gene, every 1 minute increase in the MLT corresponds to 0.3 minute increase in lysis time stochasticity.
  </p>
 </div>
 <span>
  <b> [g.5.4] </b>
 </span>
 <div id="g.5.4" last="g.5.3" next="g.5.5">
  <p>Effect of Host Growth Rates In general, cells growing at a faster rate have higher concentrations of various biosynthesis machineries [53]. Since the expression of the phage holin gene is entirely dependent on the host, we hypothesized that a lower host growth rate would lead to a lower rate of holin protein synthesis, thus resulting in a longer lysis time and increased lysis time stochasticity. In the phage T4, it was shown that lysis time was negatively correlated</p>
  <p>with host growth rate [54]. We determined the MLTs and SDs for wild-type l lysogen grown in four different growth media: standard LB (lysogeny broth [55]), 20% LB, Davis minimal salts
   <span>
    <b> [g.5.6] </b>
   </span>medium (DM) with 20 mM glucose, and DM with 40 mM glycerol, resulting in growth rates of 1.01 ± 0.07, 0.93 ± 0.05, 0.49 ± 0.04, and 0.35 ± 0.01 h
   <sup>-1</sup>(mean ± 95% confidence limits), respectively (see Table 2). As shown in Figure 3D, lower growth rates led to increased lysis time SDs (
   <i>F</i>
   <sub>[1,2]</sub>= 24.50, 
   <i>p</i> = 0.0385) and CVs (
   <i>F</i>
   <sub>[1,2]</sub>
  </p>
 </div>
 <p>= 46.24, 
  <i>p</i> = 0.0209). A similar negative relationship was
 </p>
 <p>also apparent for the MLTs. However, because of the</p>
 <p>case of the LB medium, in which the higher growth rate</p>
 <p>actually resulted in a slightly longer MLT, the observed</p>
 <p>negative relationship was not significant (
  <i>F</i>
  <sub>[1,2]</sub>= 6.44, 
  <i>p</i>
 </p>
 <p>= 0.1265). Interestingly, neither the SDs (
  <i>F</i>
  <sub>[1,2]</sub>= 16.11, 
  <i>p = 0.0568) nor the CVs (</i>
  <i>F</i>
  <sub>[1,2]</sub>= 6.04, 
  <i>p</i> = 0.133) was sig-
 </p>
 <p>nificantly associated with the MLTs.</p>
 <span>
  <b> [g.5.7] </b>
 </span>
 <div id="g.5.7" last="g.5.6" next="g.6.0">
  <p>Effects of KCN Addition The energy poison potassium cyanide, KCN, has long been used in phage research to trigger premature lysis [43]. Typically, after KCN addition, culture turbidity declines precipitously [44], indicating that individual lysis events are relatively synchronous. The KCNinduced premature lysis is thought to be mediated through a collapsed proton motive force (PMF) resulting from a inhibition of the bacterial respiratory chain. As has been shown with λ S holin, a 40% drop in the PMF
   <span>
    <b> [g.6.2] </b>
   </span>triggers lysis [45]. Without a constant supply of ATP, the production of holin protein would also be terminated. If KCN is added soon after thermal induction of the lysogen culture, few holin proteins would have been made before the termination of holin production. Consequently, it should take a longer time for the holin proteins in the membrane to transition from a diffused state to aggregated rafts. Therefore, after the cessation of holin production by KCN addition, it may take a longer time, on average, before any lysis events are observed. On the other hand, if KCN is added late, a larger proportion of the thermally-induced lysogenic cells should have accumulated enough holin proteins in the cell membrane such that they could be triggered to form holin holes quickly. That is, the addition of KCN should prompt the rapid formation of holin holes, thus resulting in an almost immediate and synchronous lysis of most of the cells in the population. Based on the aforementioned scenarios, we expected that (1) the time delay between the time of KCN addition (
   <i>t</i>
  </p>
 </div>
 <p>) and the KCN eventual mean lysis time (
  <i>t</i>
  <sub>L</sub>) (
  <i>i.e.</i>, 
  <i>t</i>
  <sub>L</sub>- 
  <i>t</i>
  <sub>KCN</sub>) would be negatively correlated with the timing of KCN addition, and (2) 
  <i>t</i>
  <sub>KCN</sub>would be negatively correlated with lysis
 </p>
 <p>time stochasticity. Figure 4A shows a significant negative relationship between 
  <i>t</i>
  <sub>L</sub>- 
  <i>t</i>
  <sub>KCN</sub>and 
  <i>t</i>
  <sub>KCN</sub>. As KCN was added later in time (
  <i>i. e.</i>, closer to the normal lysis time of 65.1 min), the time delay between addition of KCN and the MLT was reduced (a quadratic fit, 
  <i>F</i>
  <sub>[2,4]</sub>= 12.87, 
  <i>p</i> = 0.0181, adjusted 
  <i>R</i>
  <sup>2</sup>= 0.798). In fact, when added 55 min after induction (
  <i>i.e.</i>, 10 min before the normal MLT), the time delay was only 2.6 min, almost instantaneous when compared to the 2 min sampling rate of the sipperequipped spectrophotometer method of lysis time determination [46]. Interestingly, a theoretical study of lysis time stochasticity by Ryan and Rutenberg also showed a
  <span>
   <b> [g.6.3] </b>
  </span>seemingly convex relationship between 
  <i>t</i>
  <sub>L</sub>- 
  <i>t</i>
  <sub>KCN</sub>and t
  <sub>KCN</sub>[[39], their figure five].
 </p>
 <p>The effects of 
  <i>t</i>
  <sub>KCN</sub>on lysis time SDs and CVs are shown in Figure 4B. Again, we witnessed the expected pattern of a significant negative relationship between t
  <sub>KCN</sub>and the SDs (a quadratic fit, 
  <i>F</i>
  <sub>[2,4]</sub>= 9.91, 
  <i>p</i> = CVs (a quadratic fit, 
  <i>F</i>
  <sub>[2,4]</sub>= 16.03, 
  <i>p</i> = 0.0282, adjusted R
  <sup>2</sup>= 0.834). These results showed that the later in time KCN was added, the less variation there was in individual lysis times. In fact, the lowest SD (1.45 min) and lowest CV (2.53%) were observed when KCN was added 55 min after induction. This was a significant two-fold reduction in the SD when compared normal lysis conditions (see Table 1 for strain IN56 with the SD = 3.24 min; Student’ s 
  <i>t</i> = 15.45, 
  <i>p</i> &lt; 0.0001, using the standard deviation for the SD in Box 7.1 of [56]). This observation indicated that individual triggering for hole formation during the normal progression of cell lysis was relatively asynchronous when compared to the artificial method of acute triggering by KCN addition. Similar to the effect of growth rate, a linear regression
 </p>
 <p>of the SDs (
  <i>F</i>
  <sub>[1,5]</sub>= 0.60, 
  <i>p</i> = 0.4726) or CVs (
  <i>F</i>
  <sub>[1,5]</sub>= 0.328, 
  <i>p</i> = 0.5917) against the MLTs did not yield significant result. Another interesting aspect of the relationship between 
  <i>t</i>
  <sub>KCN</sub>and the lysis time SDs is that the SDs drop precipitously when KCN is added about 35 min after induction. This observation suggests that, approximately 35 min after thermal induction, the majority of the lysogenic cells have accumulated enough holin proteins in the cell membrane to form holes immediately if triggered.
 </p>
 <span>
  <b> [g.6.4] </b>
 </span>
 <div id="g.6.4" last="g.6.3" next="g.6.5">
  <p>
   <b>Discussion The current model of holin hole formation hypothesizes that λ phage lysis timing is mainly determined by when</b>
   <span>
    <b> [g.7.2] </b>
   </span>a critical concentration of holin proteins is reached in the cell membrane [40] (Figure 1, dark arrows). According to this model, any factor that influences the holin protein production should also affect the timing of lysis. Furthermore, the realized rate of holin production in each cell should also be subjected to stochastic influences impacting the various upstream biochemical reactions, such as gene transcription and translation, that lead to holin production. As has been shown by others, the lower the average rates of the biochemical reactions, the more prominent the cell-to-cell variation is [51,52].
  </p>
 </div>
 <span>
  <b> [g.7.3] </b>
 </span>
 <div id="g.7.3" last="g.7.2" next="g.7.4">
  <p>Manipulation of holin production rate In our study, we manipulated the holin production rate</p>
  <p>by manipulating the λ 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity
   <i> </i>and
   <i> </i>the
   <i> </i>lysogen growth rate. We observed that, in general, treatments expected to result in higher holin production rates (
   <i>e.g.</i>, high 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity
   <i> </i>or
   <i> </i>high
   <i> </i>lysogen
   <i> </i>growth
   <i> </i>rate)
   <i> </i>also resulted in shorter MLTs and smaller SDs (Figure 3B and 3D). Furthermore, it was surprising that the combined MLTs and SDs, despite being from two different experimental treatments, namely 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity
   <i> </i>and
   <i> </i>lysogen growth rate, showed almost identical positive correlations, even after excluding the far-flung data point with the longest MLT and largest SD (obtained with strain SYP028, see Table 2) from the analysis (Figure 3C). This result suggests that, irrespective of how the MLT was achieved, as long as the MLTs are the same, we should expect to observe similar SDs. For the wild-type λ S holin sequence, any factor that results in 1.0 min increase in MLT would be accompanied by a concomitant 0.3 min increase in the SD. It would be interesting to conduct a similar experiment with different holin sequences to see if the rate of SD increase is sequence-
  </p>
  <p>specific. Regarding the effects of host growth rate on lysis time stochasticity, it is interesting to note the following. Amir et al. [10]
   <i> </i>found
   <i> </i>that
   <i> </i>the
   <i> </i>MLTs,
   <i> </i>SDs,
   <i> </i>and
   <i> </i>CVs,
   <i> </i>following UV induction, ranged from 72 min, 9 min, and 12.5% respectively for λ lysogens alone to 99 min, 14 min, and 14.1% respectively for λ lysogens carrying pRGFP reporter plasmid and 117 min, 19 min, and 15.8% respectively for λ lysogens carrying pR’ -tR’ -GFP reporter plasmid (all values are extracted from their figures six A and B). Since their λ lysogens were grown in M9 minimal salts medium plus various growth factors and 0.4% glucose at 37°C, it is similar to our Davis minimal salts medium with glucose, from which we obtained the comparable values of 70.3 min, 6.3 min, and 8.96% respectively (see Table 2). It is not clear whether the difference between these two SDs is the result of different methods used for lysogen induction (thermal vs. UV induction) or different growth media, but the MLTs are virtually identical. Their result also indirectly confirmed
   <span>
    <b> [g.7.4] </b>
   </span>our current result that host physiology (which is presumably somewhat perturbed in their lysogen strains carrying the medium-copy reporter plasmids) would affect the overall MLTs and SDs of lysis time.
  </p>
 </div>
 <span>
  <b> [g.7.5] </b>
 </span>
 <div id="g.7.5" last="g.7.4" next="g.7.6">
  <p>Manipulation of holin protein sequence Barring potential post-translational modifications due to differences in holin protein sequence (
   <i>e.g.</i>, differential rate in proteolysis), isogenic λ strains expressing different holin sequences would have a similar average rate of holin accumulation in the membrane and consequently the same distribution of holin proteins among the cells across different lysogen populations. That is, at any given moment, we would expect a certain proportion of cells to accumulate a certain number of holin molecules in the membrane, irrespective of the holin sequences. The observed differences in MLTs, as the result of differences in holin sequence, can be seen as a reflection of different set-points for critical concentrations in an increasingly crowded cell membrane [40]. Presumably, a sequence with a lower set-point would not only result in a shorter MLT, but also a smaller SD as well. However, the existence of similar MLTs, but very different SDs, suggests that missense mutations in the holin sequence not only affect the set-point for spontaneous triggering, but also impact the robustness of the setpoint. For example, some mutations may be relatively insensitive to the critical holin concentration, thus resulting in proportionally more cells that are triggered earlier and later than expected, hence greater lysis time stochasticity.
  </p>
 </div>
 <span>
  <b> [g.7.6] </b>
 </span>
 <div id="g.7.6" last="g.7.5" next="g.8.0">
  <p>Effect of energy poison KCN It is well known that addition of the energy poison, KCN, to induced lysogen cultures will accelerate the onset of lysis [44]. Our results also confirmed this observation (see Table 2). However, it is not clear how this accelerated lysis would affect the lysis time stochasticity. From anecdotal observations, the addition of KCN seems to synchronize lysis, thus resulting in a precipitous decline of lysogen culture turbidity. Our study showed that the timing of KCN addition was inversely related to lysis time stochasticity (see Figure 4B). In fact, the smallest SD (1.45 min) was achieved by adding KCN at 55 min after thermal induction (see Table 2), a time where normally only about 1% of the cells have lysed. The almost synchronous lysis when KCN was added 55 min post thermal induction suggests that most cells would have already accumulated enough holin proteins</p>
  <p>in the cell membrane to form a hole. Besides collapsing the PMF, the addition of KCN should also halt the production of holin protein, thus “ fixes” the amount of holin proteins on the cell membrane at the time of addition. The progressive decline in lysis time
   <span>
    <b> [g.8.2] </b>
   </span>stochasticity as KCN was added later in time (see Figure 4B) strongly suggests that a larger supply of holin protein is a key factor in ensuring synchronous lysis. As more holin proteins are inserted into the cell membrane, the kinetics of raft formation gradually shifts from stochastic to deterministic and synchronous. In fact, there was a nearly five-fold decrease in lysis time stochasticity when the PMF was collapsed at 55 min after lysogen induction when compared to collapse at 25 min (see Table 2). It is also noted that the properties of the normally triggered and the prematurely triggered holin holes are quite distinct, with the prematurely triggered holes being much smaller than the normally triggered holes [28].
  </p>
 </div>
 <span>
  <b> [g.8.3] </b>
 </span>
 <div id="g.8.3" last="g.8.2" next="g.8.4">
  <p>Evolutionary implication of lysis time stochasticity Both theoretical and experimental studies have demonstrated the importance of lysis timing on phage fitness [46,57-61]. However, it is not clear if lysis time stochasticity would have any impact on phage fitness. All else being equal, genotypes with reduced variances in offspring number would, in the long run, have higher fitness than genotypes that have the same mean offspring number, but larger variances [62,63]. Overall, this suggests that natural selection would tend to minimize stochasticity in phenotypes that are closely linked to Darwinian fitness. If the phage burst size is positively linked with the lysis time, as has been shown previously [46], then selection for reduced burst size stochasticity should lead to reduced lysis time stochasticity as well. Presumably, this hypothesis can be tested by competing two isogenic phage strains that have the same MLTs but very different lysis time SDs. Interestingly, inspection of Table 1 revealed that mutations introduced into WT λ holin sequence usually result in increased stochasticity, except in one case. It is not clear if this observation implies that the WT holin sequences have already been selected for reduced stochasticity in the wild as well. Experiments with more phage holins should provide some hints in this respect.</p>
 </div>
 <span>
  <b> [g.8.4] </b>
 </span>
 <div id="g.8.4" last="g.8.3" next="g.8.5">
  <p>
   <b>Conclusions Even in a seemingly uniform environment, the lysis time</b>
  </p>
  <p>can vary greatly among individual λ lysogenic cells (lysis</p>
  <p>time stochasticity). The extent of stochasticity, as quan-</p>
  <p>tified by the standard deviation, depends on the quality</p>
  <p>(due to isogenic λ lysogens expressing different S pro-</p>
  <p>tein alleles) and quantity (manipulated by having differ-</p>
  <p>ent 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activities
   <i> </i>and
   <i> </i>lysogen
   <i> </i>growth
   <i> </i>rates)
   <i> </i>of
   <i> </i>the
   <i> </i>holin
  </p>
  <p>protein, the major determinant of lysis timing in large-</p>
  <p>genome phages. There is a general positive trend</p>
  <p>between the mean lysis time and the degree of stochasti-</p>
  <p>city. However, this positive relationship is much tighter</p>
  <p>when difference in mean lysis time is due to holin quantity rather than quality. The pattern of lysis time
   <span>
    <b> [g.8.5] </b>
   </span>stochasticity obtained by addition of KCN at various time points after lysogen induction showed a negative relationship between the timing of KCN addition and the level of lysis time stochasticity.
  </p>
 </div>
 <span>
  <b> [g.8.6] </b>
 </span>
 <div id="g.8.6" last="g.8.5" next="g.8.7">
  <p>
   <b>Appendix A This section provides the rationale for partitioning lysis</b>
  </p>
  <p>time variance found in the study by Amir 
   <i>et al.</i> [10]. For each UV-induced λ lysogenic cell, the lysis time 
   <i>T can be divided into three time intervals: (1) </i>
   <i>t</i>
   <sub>1</sub>, the time interval between lysogen induction and the onset of 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>promoter, (2) 
   <i>t</i>
   <sub>2</sub>, the time interval between the onset of the 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>promoter and the onset of the 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter,
   <i> </i>and (3) 
   <i>t</i>
   <sub>3</sub>, the time interval between the onset of the 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ promoter and the eventual lysis. The following relationships describe the above time intervals and the empirically determined time intervals by Amir </i>
   <i>et al.</i> [10]: 
   <i>t</i>
   <sub>1</sub>= t
   <sub>pR</sub>, 
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>= 
   <i>t</i>
   <sub>pR’ -tR’</sub>, 
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>+ 
   <i>t</i>
   <sub>3</sub>= 
   <i>t</i>
   <sub>lysis</sub>, and 
   <i>t</i>
   <sub>3</sub>= Δ 
   <i>t</i> = 
   <i>t</i>
   <sub>ly-</sub>
  </p>
  <p>
   <sub>sis</sub>- 
   <i>t</i>
   <sub>pR’ -tR’</sub>. For, 
   <i>T</i> = 
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>+ 
   <i>t</i>
   <sub>3</sub>, the variance for the lysis time can be expressed as 
   <i>VAR</i>(
   <i>T</i>) = 
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>) + 
   <i>VAR</i>(
   <i>t</i>
   <sub>2</sub>) + VAR(
   <i>t</i>
   <sub>3</sub>) + 2
   <i>COV</i> (
   <i>t</i>
   <sub>1</sub>, 
   <i>t</i>
   <sub>2</sub>) + 2
   <i>COV</i> (
   <i>t</i>
   <sub>2</sub>, 
   <i>t</i>
   <sub>3</sub>) + 2
   <i>COV</i> (
   <i>t</i>
   <sub>1</sub>, 
   <i>t</i>
   <sub>3</sub>). While the authors did not provide all possible combinations of covariance, it is empirically determined that COV(
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>, 
   <i>t</i>
   <sub>3</sub>) = 0, as shown in their figure seven E (
   <i>i. e.</i>,
   <i> </i>no
   <i> </i>correlation
   <i> </i>between
   <i> t</i>
   <sub>pR’ -tR’</sub>and Δ 
   <i>t</i>). That is, 
   <i>COV (</i>
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>, 
   <i>t</i>
   <sub>3</sub>) = 
   <i>COV</i>(
   <i>t</i>
   <sub>1</sub>, 
   <i>t</i>
   <sub>3</sub>) + 
   <i>COV</i>(
   <i>t</i>
   <sub>2</sub>, 
   <i>t</i>
   <sub>3</sub>) = 0. Although not empirically demonstrated, it seems unlikely that the timing of turning on either the 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>or 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter would have a positive or negative effect on the assembly of lysis apparatus such that their effects would cancel each other out, resulting in the observed 
   <i>COV</i>(
   <i>t</i>
   <sub>1</sub>, 
   <i>t</i>
   <sub>3</sub>) + COV(
   <i>t</i>
   <sub>2</sub>, 
   <i>t</i>
   <sub>3</sub>) = 0. Most likely, time intervals are mutually
  </p>
  <p>independent, 
   <i>i.e.</i>, 
   <i>COV</i>(
   <i>t</i>
   <sub>1</sub>, 
   <i>t</i>
   <sub>3</sub>) = 
   <i>COV</i>(
   <i>t</i>
   <sub>2</sub>, 
   <i>t</i>
   <sub>3</sub>) = 0. The standard deviations ("absolute noise” in their terminology) for 
   <i>t</i>
   <sub>pR’ -tR’</sub>and 
   <i>t</i>
   <sub>lysis</sub>can be extracted from their figure six A using data determined from cells carrying the pR’ -tR’ -GFP plasmid. The estimated SDs for t
   <sub>pR’ -tR’</sub>and 
   <i>t</i>
   <sub>lysis</sub>are ~10 min and ~18 min, respectively; therefore, 
   <i>VAR</i>(
   <i>t</i>
   <sub>pR’ -tR’</sub>) = ~100 and 
   <i>VAR</i>(
   <i>t</i>
   <sub>lysis</sub>) = ~324. The SD for 
   <i>t</i>
   <sub>pR</sub>can be estimated by extrapolating the line connecting between lysis and 
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>onset
   <i> </i>to
   <i> </i>the
   <i> </i>20 min mean time at the 
   <i>x</i>-axis (based on the result from cells carrying the pR-GFP plasmid in their figure six A). The corresponding SD for 
   <i>t</i>
   <sub>pR</sub>is ~7 min, thus 
   <i>VAR</i>(
   <i>t</i>
   <sub>pR</sub>) = ~49. Taken together, 
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>) = 49, 
   <i>VAR</i>(
   <i>t</i>
   <sub>2</sub>) = 51 (= VAR(
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>) - 
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>) = 100 - 49 ), and 
   <i>VAR</i>(
   <i>t</i>
   <sub>3</sub>) = 224 (= 
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>+ 
   <i>t</i>
   <sub>3</sub>) - 
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>+ 
   <i>t</i>
   <sub>2</sub>) = 324 - 100). That is, 
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>), 
   <i>VAR</i>(
   <i>t</i>
   <sub>2</sub>), and 
   <i>VAR</i>(
   <i>t</i>
   <sub>3</sub>) contributed to 15%, 16%, and 69% of total lysis time variance, respectively.
  </p>
 </div>
 <span>
  <b> [g.8.7] </b>
 </span>
 <div id="g.8.7" last="g.8.6" next="g.9.0">
  <p>
   <b>Appendix B Studies of molecular stochasticity typically use the coefficient of variation (CV) as the measurement for the</b>
   <span>
    <b> [g.9.2] </b>
   </span>degree of stochasticity [15,25,48,49]. Since CV is a composite statistic (defined as standard deviation/mean), it is sometimes difficult to discern whether an increase in the observed stochasticity (as quantified by CV) is due to decrease in mean or increase in SD. In some cases, a different metric, such as phenotypic noise strength (defined as variance/mean) [17,20], or a slight variant of it (defined as variance/squared mean) [19], has been used as well. Many times, it is not clear why a particular metric is used, except in the instance where the phenotypic noise strength is used to test against an 
   <i>a priori expectation of a Poisson distribution, for which variance/mean = 1.</i>
  </p>
 </div>
 <p>It is understandable why the CV, or a variant, is used in certain situations. For example, if the means are drastically different from each other or a comparison is made between measurements using different units [[56], pp. 57-59.]. In our study, however, the means were not very different and the same measuring unit (
  <i>i.e.</i>, min) was used. Therefore, we presented our means and SDs separately and then jointly as CVs. Except in one instance where presenting stochasticity as SD or CV makes a difference (
  <i>i.e.</i>, effect of genotype on SD or CV vs. MLT), all the other results showed that SD and CV followed the same trend. Since CV can be derived from SD and mean, no information is lost by presenting them separately. Furthermore, when the stochasticity (or noise distribution) is not apparent, it may be advisable to present results as means and SDs (and if necessary, higher moments like skewness and kurtosis) separately, so that the impact of various experimental treatments on the noise distribution can be fully appreciated.
 </p>
 <span>
  <b> [g.9.3] </b>
 </span>
 <div id="g.9.3" last="g.9.2" next="g.9.4">
  <p>
   <b>Methods Bacterial strains All bacteria and phage strains used in this study are listed in Table 3. The copy number of λ genome was checked by PCR following the method of Powell </b>
   <i>et al.</i> [64].
  </p>
 </div>
 <span>
  <b> [g.9.4] </b>
 </span>
 <div id="g.9.4" last="g.9.3" next="g.9.5">
  <p>Experimental instrumentation</p>
  <p>
   <i>E. coli </i>cells
   <i> </i>lysogenic
   <i> </i>for
   <i> </i>λ
   <i> </i>phage
   <i> </i>were
   <i> </i>induced
   <i> </i>and
  </p>
  <p>observed to lyse in a temperature-controlled perfusion</p>
  <p>chamber. The experimental apparatus consisted of a 250</p>
  <p>mL side-arm (on bottom) medium bottle clamped to an</p>
  <p>elevated support with tubing leading to an inline heater</p>
  <p>(SH-27B, Warner Instruments, New Haven, CT) that</p>
  <p>was controlled by a dual channel heater controller (TC-</p>
  <p>344B, Warner Instruments, New Haven, CT). The</p>
  <p>growth medium, flowing at a rate of ~1 mL/min (driven</p>
  <p>by gravity) and heated by the inline heater to the desired</p>
  <p>temperature, was introduced to a 358 μ L perfusion</p>
  <p>chamber (RC-21B, Warner Instruments, New Haven,</p>
  <p>CT) mounted on a heating platform (PM2, Warner Instruments, New Haven, CT) that was controlled by
   <span>
    <b> [g.9.6] </b>
   </span>the same dual channel heater controller to maintain the desired temperature. The internal temperature of the perfusion chamber was independently monitored by a thermistor. Waste flowed out of the perfusion chamber, pooled in a reservoir, and was siphoned into a 2 L bottle by a vacuum source. Both the perfusion chamber and the heating platform were placed on the stage of an inverted microscope (TS100, Nikon) for observation at 400× magnification. One of the microscope’ s ocular lenses was replaced with a 10X MiniVID™ microscope camera (LW Scientific, Norcross, GA) to record individual lysis events onto a laptop computer at the rate of 1 frame per second. All data were collected in unit of seconds, though the results were presented in minutes.
  </p>
 </div>
 <span>
  <b> [g.9.7] </b>
 </span>
 <div id="g.9.7" last="g.9.6" next="g.10.0">
  <p>Sample preparation and lysis time determination Lysogens were cultured overnight in LB or minimal salts media (see below) at 30°C on a rolling drum. Stationary phase cultures were diluted 100-fold in LB or minimal</p>
  <p>salts media, then grown to A
   <sub>550</sub>~ 0.2. 200 μ L of expo-
  </p>
  <p>nentially growing cells were immobilized on a 22 mm square glass coverslip that has been pretreated with
   <span>
    <b> [g.10.2] </b>
   </span>0.01% tissue-culture tested poly-L-lysine (mol. wt. 150 K - 300 K, Sigma, St. Louis, MO) at room temperature for 30 min. After assembling the perfusion chamber, the device was immediately placed on the heating platform and infused with heated medium to maintain the chamber temperature at 30°C for 30 min to stabilize the cells. To induce lysis, the chamber temperature was raised to 42°C for 15 min, and then dropped to 37°C for the duration of the observation period (
   <i>i.e.</i>, until ~95% of cells are lysed). Video recording was initiated at the time when the temperature was raised to 42°C. Under these conditions, it usually takes less than 5 min for the temperature to rise from 30°C to 42°C, a transition comparable to shifting culture flasks from a 30°C to 42°C waterbath shaker. Some experiments were performed by adding KCN to the growth medium in the sidearm feeder bottle to a final concentration of 20 mM.
  </p>
 </div>
 <p>Videos were subsequently analyzed using Windows Media Player™ playback. The times of individual lysis events were then noted visually and recorded manually. The lysis time was defined as the time from the initiation of the first temperature shift to when the image of the cell disappeared from view. In general, it takes about a few seconds (frames) for lysing cells to fully disappear from view (Figure 1A).</p>
 <span>
  <b> [g.10.3] </b>
 </span>
 <div id="g.10.3" last="g.10.2" next="g.10.4">
  <p>Determination of lysogen growth rate Lysogen growth rate was manipulated by using different growth medium formulations: (i) full-strength LB (10 g tryptone, 5 g yeast extract, 10 g NaCl per L dH
   <sub>2</sub>O), (ii) one-fifth-strength LB (2 g tryptone, 1 g yeast extract, 10 g NaCl per L dH
   <sub>2</sub>O), (iii) 20 mM glucose in Davis minimal salts (7 g K
   <sub>2</sub>HPO
   <sub>4</sub>, 2 g KH
   <sub>2</sub>PO
   <sub>4</sub>, 1 g (NH
   <sub>4</sub>)
   <sub>2</sub>SO
   <sub>4</sub>, 0.5 g sodium citrate• 2H
   <sub>2</sub>O, and 0.2 g MgSO
   <sub>4</sub>• 7H
   <sub>2</sub>O), and (iv) 40 mM glycerol in Davis minimal salts. We assessed the growth of the lysogen strain IN56 by culturing it overnight at 30°C in each growth media. The next day, 90 μ L of the overnight culture was used to inoculate 25 mL growth medium and the culture was placed in a 30°C waterbath shaker at 220 rpm. Culture growth was followed
  </p>
  <p>with a sipper-equipped spectrophotometer at A . 550 The growth rate was calculated as the slope of the linear regression of natural-logarithm transformed A 550 values over time.</p>
 </div>
 <span>
  <b> [g.10.4] </b>
 </span>
 <div id="g.10.4" last="g.10.3" next="g.10.5">
  <p>Statistical analysis In most cases, data collection for a given strain or treatment spanned several days. Therefore, even for the same lysogen strain or experimental treatment the means and/or variances may be significantly different among data collected from different dates. We conducted a preliminary exploration of our data set using the software package JMP version 7.0.2, as
   <span>
    <b> [g.10.5] </b>
   </span>implemented in MacOS operating system. For each lysogen strain or experimental treatment, the means and standard deviations (SDs) were extracted from the data set according to the date the data were collected and were treated as replicates. Pairwise comparisons of the means (using the Tukey-Kramer HSD test) showed that, for more than half of the cases, at least one mean was significantly different from the others. Since we were mainly interested in the variation, we subsequently converted all values into their corresponding residuals (centered by their corresponding means). We also tested the homogeneity of variance from each date replicate, using O’ Brien’ s test, Brown-Forsythe test, Levene’ s test, and Bartlett’ s test, all implemented in
  </p>
 </div>
 <p>JMP. Not surprisingly, more than half of the cases</p>
 <p>showed that at least one replicate variance was signifi-</p>
 <p>cantly different from the others. Although we did not</p>
 <p>have an 
  <i>a priori</i> expectation of lysis time distribution,
 </p>
 <p>we nonetheless tested to see if the lysis time in each</p>
 <p>replicate is normally distributed or not, using the Sha-</p>
 <p>piro-Wilk W test. Again, in many cases, the replicates</p>
 <p>do not show a normal distribution. Despite variability</p>
 <p>in our data set, none of our conclusions were funda-</p>
 <p>mentally changed. Therefore, for the presented results,</p>
 <p>the mean and standard deviation for each lysogen</p>
 <p>strain or experimental treatment were calculated based</p>
 <p>on the following criteria: (
  <i>i</i>) if the means and variances
 </p>
 <p>were the same among all blocks, then all the data</p>
 <p>would be pooled together to estimate the combined</p>
 <p>means and SDs, (
  <i>ii</i>) if the means were significantly different, but the variances were the same among all blocks, then the mean would be estimated by averaging the block means while the SDs would be estimated by pooled residuals, and (
  <i>iii</i>) if the means and variances were significantly different among all blocks, then the means and SDs would be estimated by averaging block means and SDs. For details of our data set, see additional file 1.
 </p>
 <span>
  <b> [g.10.6] </b>
 </span>
 <p id="g.10.6" last="g.10.5" next="g.10.7">
  <b>Additional material</b>
 </p>
 <span>
  <b> [g.10.7] </b>
 </span>
 <div id="g.10.7" last="g.10.6" next="g.10.8">
  <div>
   <div>
    <p>Additional file 1: Sample sizes and standard deviations. More detailed data sets for both Table 1 and Table 2.</p>
   </div>
  </div>
  <div>
   <div>
    <svg width="1000.0" height="800.0" xmlns="http://www.w3.org/2000/svg">
     <g transform="matrix(0.5,0.0,0.0,0.5,0.0,0.0)">
      <path stroke="black" fill="#000000" d="M304.724 578.587 L304.781 578.077 C304.941 576.823 306.005 575.857 307.219 575.639 L307.729 575.582 L307.729 575.866 L307.219 575.922 C306.129 576.055 305.173 577.071 305.007 578.133 L304.951 578.587 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M304.724 578.587 L304.781 578.077 C304.941 576.823 306.005 575.857 307.219 575.639 L307.729 575.582 L307.729 575.866 L307.219 575.922 C306.129 576.055 305.173 577.071 305.007 578.133 L304.951 578.587 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M307.729 575.583 L535.578 575.583 L535.578 575.866 L307.729 575.866 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M307.729 575.583 L535.578 575.583 L535.578 575.866 L307.729 575.866 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M535.578 575.583 L536.088 575.64 C537.302 575.858 538.365 576.823 538.526 578.077 L538.583 578.588 L538.356 578.588 L538.299 578.134 C538.134 577.072 537.177 576.056 536.088 575.923 L535.578 575.866 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M535.578 575.583 L536.088 575.64 C537.302 575.858 538.365 576.823 538.526 578.077 L538.583 578.588 L538.356 578.588 L538.299 578.134 C538.134 577.072 537.177 576.056 536.088 575.923 L535.578 575.866 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M304.724 597.523 L304.724 578.588 L304.951 578.588 L304.951 597.523 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M304.724 597.523 L304.724 578.588 L304.951 578.588 L304.951 597.523 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M538.583 578.587 L538.583 597.522 L538.356 597.522 L538.356 578.587 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M538.583 578.587 L538.583 597.522 L538.356 597.522 L538.356 578.587 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M307.729 600.528 L307.219 600.471 C306.082 600.389 304.882 599.165 304.781 598.034 L304.724 597.523 L304.951 597.523 L305.008 597.977 C305.158 599.105 306.151 599.973 307.219 600.245 L307.729 600.245 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M307.729 600.528 L307.219 600.471 C306.082 600.389 304.882 599.165 304.781 598.034 L304.724 597.523 L304.951 597.523 L305.008 597.977 C305.158 599.105 306.151 599.973 307.219 600.245 L307.729 600.245 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M538.583 597.523 L538.526 598.033 C538.425 599.165 537.225 600.389 536.088 600.471 L535.578 600.528 L535.578 600.244 L536.088 600.244 C537.156 599.972 538.149 599.106 538.3 597.977 L538.356 597.523 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M538.583 597.523 L538.526 598.033 C538.425 599.165 537.225 600.389 536.088 600.471 L535.578 600.528 L535.578 600.244 L536.088 600.244 C537.156 599.972 538.149 599.106 538.3 597.977 L538.356 597.523 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M535.578 600.528 L307.729 600.528 L307.729 600.245 L535.578 600.245 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M535.578 600.528 L307.729 600.528 L307.729 600.245 L535.578 600.245 " stroke-width="1.0"/>
     </g>
    </svg>
   </div>
  </div>
 </div>
 <span>
  <b> [g.11.5] </b>
 </span>
 <div id="g.11.5" last="g.11.4" next="g.11.6">
  <p>Received: 1 December 2010 Accepted: 2 August 2011 Published: 2 August 2011
   <span>
    <b> [g.11.7] </b>
   </span>24. Blake WJ, Balazsi G, Kohanski MA, Isaacs FJ, Murphy KF, Kuang Y, Cantor CR,
  </p>
 </div>
 <p>Walt DR, Collins JJ: Phenotypic consequences of promoter-mediated</p>
 <p>transcriptional noise. 
  <i>Mol Cell</i> 2006, 24:853-865. 25. Bishop AL, Rab FA, Sumner ER, Avery SV: Phenotypic heterogeneity can
 </p>
 <p>enhance rare-cell survival in ‘ stress-sensitive’ yeast populations. 
  <i>Mol</i>
 </p>
 <p>
  <i>Microbiol </i>2007,
  <i> </i>63:507-520. 26. Wang IN, Smith DL, Young R: HOLINS: The Protein Clocks of
 </p>
 <p>Bacteriophage Infections. 
  <i>Annu Rev Microbiol</i> 2000, 54:799-825. 27. Young R, Wang IN, Roof WD: Phages will out: strategies of host cell lysis.
 </p>
 <p>
  <i>Trends Microbiol </i>2000,
  <i> </i>8:120-128. 28. Wang IN, Deaton J, Young R: Sizing the holin lesion with an endolysin-β -
 </p>
 <p>galactosidase fusion. 
  <i>J Bacteriol</i> 2003, 185:779-787. 29. Savva CG, Dewey JS, Deaton J, White RL, Struck DK, Holzenburg A,
 </p>
 <p>Young R: The holin of bacteriophage lambda forms rings with large</p>
 <p>diameter. 
  <i>Mol Microbiol</i> 2008, 69:784-793. 30. Park T, Struck DK, Dankenbring CA, Young R: The pinholin of lambdoid
 </p>
 <p>phage 21: control of lysis by membrane depolarization. 
  <i>J Bacteriol</i> 2007,
 </p>
 <p>189:9135-9139. 31. Xu M, Arulandu A, Struck DK, Swanson S, Sacchettini JC, Young R: Disulfide</p>
 <p>isomerization after membrane release of its SAR domain activates P1</p>
 <p>lysozyme. 
  <i>Science</i> 2005, 307:113-117. 32. Xu M, Struck DK, Deaton J, Wang IN, Young R: A signal-arrest-release
 </p>
 <p>sequence mediates export and control of the phage P1 endolysin. 
  <i>Proc</i>
 </p>
 <p>
  <i>Natl Acad Sci USA </i>2004,
  <i> </i>101:6415-6420. 33. Zhang N, Young R: Complementation and characterization of the nested
 </p>
 <p>
  <i>Rz </i>and
  <i> Rz1 </i>reading
  <i> </i>frames
  <i> </i>in
  <i> </i>the
  <i> </i>genome
  <i> </i>of
  <i> </i>bacteriophage
  <i> </i>lambda.
  <i> Mol</i>
 </p>
 <p>
  <i>Gen Genet </i>1999,
  <i> </i>262:659-667. 34. Berry J, Summer EJ, Struck DK, Young R: The final step in the phage
 </p>
 <p>infection cycle: the Rz and Rz1 lysis proteins link the inner and outer</p>
 <p>membranes. 
  <i>Mol Microbiol</i> 2008, 70:341-351. 35. Young R, Way J, Way S, Yin J, Syvanen M: Transposition mutagenesis of
 </p>
 <p>bacteriophage lambda: a new gene affecting cell lysis. 
  <i>J Mol Biol</i> 1979,
 </p>
 <p>132:307-322. 36. Friedman DI, Gottesman M: Lytic mode of lambda development. In</p>
 <p>
  <i>Lambda II. </i>Edited
  <i> </i>by:
  <i> </i>Hendrix
  <i> </i>RW,Roberts
  <i> </i>JW,Stahl
  <i> </i>FW,Weisberg
  <i> </i>RA.
  <i> </i>Cold
 </p>
 <p>Spring Harbor, New York: Cold Spring Harbor Laboratory; 1983:21-51. 37. Gründling A, Bläsi U, Young R: Genetic and biochemical analysis of dimer</p>
 <p>and oligomer interactions of the lambda S holin. 
  <i>J Bacteriol</i> 2000,
 </p>
 <p>182:6082-6090. 38. Dewey JS, Savva CG, White RL, Vitha S, Holzenburg A, Young R: Micron-</p>
 <p>scale holes terminate the phage infection cycle. 
  <i>Proc Natl Acad Sci USA</i>
 </p>
 <p>2010, 107:2219-2223. 39. Ryan GL, Rutenberg AD: Clocking out: modeling phage-induced lysis of</p>
 <p>
  <i>Escherichia coli</i>.
  <i> J Bacteriol </i>2007,
  <i> </i>189:4749-4755. 40. White R, Chiba S, Pang T, Dewey JS, Savva CG, Holzenburg A, Pogliano K,
 </p>
 <p>Young R: Holin triggering in real time. 
  <i>Proc Natl Acad Sci USA</i> 2010,
 </p>
 <p>108:798-803. 41. Ellis EL, Delbrück M: The growth of bacteriophage. 
  <i>J Gen Physiol</i> 1939,
 </p>
 <p>22:365-384. 42. Delbrück M: The growth of bacteriophage and lysis of the host. 
  <i>J Gen</i>
 </p>
 <p>
  <i>Physiol </i>1940,
  <i> </i>23:643-660. 43. Doermann AH: The intracellular growth of bacteriophages. I. Liberation
 </p>
 <p>of intracellular bacteriophage T4 by premature lysis with another phage</p>
 <p>or with cyanide. 
  <i>J Gen Physiol</i> 1952, 35:645-656. 44. Young R: Bacteriophage lysis: mechanism and regulation. 
  <i>Microbiol Rev</i>
 </p>
 <p>1992, 56:430-481. 45. Gründling A, Manson MD, Young R: Holins kill without warning. 
  <i>Proc Natl</i>
 </p>
 <p>
  <i>Acad Sci USA </i>2001,
  <i> </i>98:9348-9352. 46. Wang IN: Lysis timing and bacteriophage fitness. 
  <i>Genetics</i> 2006, 172:17-26. 47. Raab R, Neal G, Garrett J, Grimaila R, Fusselman R, Young R: Mutational
 </p>
 <p>analysis of bacteriophage lambda lysis gene S. 
  <i>J Bacteriol</i> 1986,
 </p>
 <p>167:1035-1042. 48. Swain PS, Elowitz MB, Siggia ED: Intrinsic and extrinsic contributions to</p>
 <p>stochasticity in gene expression. 
  <i>Proc Natl Acad Sci USA</i> 2002,
 </p>
 <p>99:12795-12800. 49. Raj A, Peskin CS, Tranchina D, Vargas DY, Tyagi S: Stochastic mRNA</p>
 <p>synthesis in mammalian cells. 
  <i>PLoS Biol</i> 2006, 4:1707-1719. 50. Shao Y, Wang IN: Effect of late promoter activity on bacteriophage λ
 </p>
 <p>fitness. 
  <i>Genetics</i> 2009, 181:1467-1475.
 </p>
 <span>
  <b> [g.12.2] </b>
 </span>
 <div id="g.12.2" last="g.12.1" next="g.12.3">
  <p>51. Gillespie DT: Exact stochastic simulation of coupled chemical reactions. 
   <i>J</i>
  </p>
  <p>
   <i>Phys Chem </i>1977,
   <i> </i>81:2340-2361. 52. McAdams HH, Arkin A: Stochastic mechanisms in gene expression. 
   <i>Proc</i>
  </p>
  <p>
   <i>Natl Acad Sci USA </i>1997,
   <i> </i>94:814-819. 53. Bremer H, Dennis PP: Modulation of chemical composition and other
  </p>
  <p>parameters of the cell by growth rate. In 
   <i>Escherichia coli and Salmonella</i>
  </p>
  <p>
   <i>typhimurium Cellular and Molecular Biology. Volume 2. </i>Edited
   <i> </i>by:
   <i> </i>Ingraham
  </p>
  <p>JL,Low KB,Magasanik B,Schaechter M,Umbarger HE. Washington, D.C.:</p>
  <p>American Society for Microbiology; 1987:1527-1542. 54. Hadas H, Einav M, Fishov I, Zaritsky A: Bacteriophage T4 development</p>
  <p>depends on the physiology of its host 
   <i>Escherichia coli</i>. 
   <i>Microbiology</i> 1997,
  </p>
  <p>143:179-185. 55. Bertani G: Lysogeny at mid-twentieth century: P1, P2, and other</p>
  <p>experimental systems. 
   <i>J Bacteriol</i> 2004, 186:595-600. 56. Sokal RR, Rohlf FJ: Biometry. New York, New York: W. H. Freeman and
  </p>
  <p>Company;, 3 1995. 57. Abedon ST: Selection for bacteriophage latent period length by bacterial</p>
  <p>density: A theoretical examination. 
   <i>Microb Ecol</i> 1989, 18:79-88. 58. Abedon ST, Herschler TD, Stopar D: Bacteriophage latent-period evolution
  </p>
  <p>as a response to resource availability. 
   <i>Appl Environ Microbiol</i> 2001,
  </p>
  <p>67:4233-4241. 59. Heineman RH, Bull JJ: Testing optimality with experimental evolution:</p>
  <p>lysis time in a bacteriophage. 
   <i>Evolution</i> 2007, 61:1695-1709. 60. Shao Y, Wang IN: Bacteriophage adsorption rate and optimal lysis time.
  </p>
  <p>
   <i>Genetics </i>2008,
   <i> </i>180:471-482. 61. Wang IN, Dykhuizen DE, Slobodkin LB: The evolution of phage lysis
  </p>
  <p>timing. 
   <i>Evol Ecol</i> 1996, 10:545-558. 62. Gillespie JH: Nautural selection for within-generation variance in
  </p>
  <p>offspring number. 
   <i>Genetics</i> 1974, 76:601-606. 63. Gillespie JH: Natural selection for variances in offspring numbers: a new
  </p>
  <p>evolutionary principle. 
   <i>Am Nat</i> 1977, 111:1010-1014. 64. Powell BS, Rivas MP, Court DL, Nakamura Y, Turnbough CL Jr: Rapid
  </p>
  <p>confirmation of single copy lambda prophage integration by PCR.</p>
  <p>
   <i>Nucleic Acids Res </i>1994,
   <i> </i>22:5765-5766.
  </p>
 </div>
 <span>
  <b> [g.12.3] </b>
 </span>
 <div id="g.12.3" last="g.12.2" next="g.12.4">
  <div>
   <div>
    <p>doi:10.1186/1471-2180-11-174 Cite this article as: Dennehy and Wang: Factors influencing lysis time stochasticity in bacteriophage λ . 
     <i>BMC Microbiology</i> 2011 11:174.
    </p>
   </div>
  </div>
  <div>
   <div>
    <svg width="1000.0" height="800.0" xmlns="http://www.w3.org/2000/svg">
     <g transform="matrix(0.5,0.0,0.0,0.5,0.0,0.0)">
      <path stroke="black" fill="#1f6fb2" d="M56.693 398.814 L56.75 398.304 C56.964 396.941 57.913 396.21 59.187 395.866 L59.698 395.809 L59.698 396.32 L59.244 396.376 C58.229 396.55 57.376 397.406 57.203 398.417 L57.203 398.814 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M56.693 398.814 L56.75 398.304 C56.964 396.941 57.913 396.21 59.187 395.866 L59.698 395.809 L59.698 396.32 L59.244 396.376 C58.229 396.55 57.376 397.406 57.203 398.417 L57.203 398.814 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M59.698 395.81 L287.547 395.81 L287.547 396.32 L59.698 396.32 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M59.698 395.81 L287.547 395.81 L287.547 396.32 L59.698 396.32 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M287.546 395.81 L288.056 395.867 C289.27 396.085 290.334 397.05 290.494 398.304 L290.551 398.815 L290.04 398.815 L290.04 398.418 C289.78 397.313 289.139 396.605 288.0 396.377 L287.546 396.32 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M287.546 395.81 L288.056 395.867 C289.27 396.085 290.334 397.05 290.494 398.304 L290.551 398.815 L290.04 398.815 L290.04 398.418 C289.78 397.313 289.139 396.605 288.0 396.377 L287.546 396.32 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M56.693 425.743 L56.693 398.814 L57.203 398.814 L57.203 425.743 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M56.693 425.743 L56.693 398.814 L57.203 398.814 L57.203 425.743 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M290.551 398.814 L290.551 425.743 L290.041 425.743 L290.041 398.814 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M290.551 398.814 L290.551 425.743 L290.041 425.743 L290.041 398.814 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M59.698 428.691 L59.188 428.691 C57.914 428.347 56.964 427.616 56.75 426.253 L56.693 425.743 L57.204 425.743 L57.204 426.14 C57.377 427.151 58.229 428.007 59.244 428.181 L59.698 428.237 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M59.698 428.691 L59.188 428.691 C57.914 428.347 56.964 427.616 56.75 426.253 L56.693 425.743 L57.204 425.743 L57.204 426.14 C57.377 427.151 58.229 428.007 59.244 428.181 L59.698 428.237 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M290.551 425.743 L290.494 426.253 C290.328 427.508 289.261 428.415 288.056 428.691 L287.546 428.691 L287.546 428.237 L288.0 428.181 C289.015 428.007 289.868 427.151 290.041 426.14 L290.041 425.743 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M290.551 425.743 L290.494 426.253 C290.328 427.508 289.261 428.415 288.056 428.691 L287.546 428.691 L287.546 428.237 L288.0 428.181 C289.015 428.007 289.868 427.151 290.041 426.14 L290.041 425.743 " stroke-width="1.0"/>
      <path stroke="black" fill="#1f6fb2" d="M287.546 428.691 L59.697 428.691 L59.697 428.237 L287.546 428.237 " stroke-width="1.0"/>
      <path fill="none" stroke="#1f6fb2" d="M287.546 428.691 L59.697 428.691 L59.697 428.237 L287.546 428.237 " stroke-width="1.0"/>
     </g>
    </svg>
   </div>
  </div>
 </div>
 <span>
  <b> [g.12.4] </b>
 </span>
 <div id="g.12.4" last="g.12.3">
  <div>
   <div>
    <p>
     <b>Submit your next manuscript to BioMed Central and take full advantage of: • </b>
     <b> Convenient online submission • </b>
     <b> Thorough peer review • </b>
     <b> No space constraints or color figure charges • </b>
     <b> Immediate publication on acceptance • </b>
     <b> Inclusion in PubMed, CAS, Scopus and Google Scholar • </b>
     <b>Research which is freely available for redistribution Submit your manuscript at www.biomedcentral.com/submit</b>
    </p>
   </div>
  </div>
  <div>
   <div>
    <svg width="1000.0" height="800.0" xmlns="http://www.w3.org/2000/svg">
     <g transform="matrix(0.5,0.0,0.0,0.5,0.0,0.0)">
      <path fill="none" stroke="#5e5e5e" d="M535.492 740.315 C535.492 743.472 532.963 746.032 529.844 746.032 L312.467 746.032 C309.346 746.032 306.817 743.472 306.817 740.315 L306.817 599.183 C306.817 596.025 309.346 593.464 312.467 593.464 L529.844 593.464 C532.963 593.464 535.492 596.025 535.492 599.183 L535.492 740.315 Z" stroke-width="1.0"/>
      <path stroke="black" fill="#aef9ff" d="M454.043 738.043 C451.786 736.939 450.236 734.65 450.236 732.007 C450.236 728.282 453.293 725.335 457.089 725.335 C458.464 725.335 459.954 726.122 459.954 726.122 C459.186 725.76 458.326 725.556 457.417 725.556 C454.182 725.556 451.56 728.129 451.56 731.301 C451.56 733.577 452.909 735.543 454.866 736.474 L454.043 738.043 " stroke-width="1.0"/>
      <path stroke="black" fill="#15ffea" d="M460.782 724.554 C463.039 725.659 464.588 727.947 464.588 730.59 C464.588 734.315 461.532 737.262 457.735 737.262 C456.36 737.262 454.871 736.474 454.871 736.474 C455.638 736.837 456.5 737.039 457.408 737.039 C460.642 737.039 463.264 734.469 463.264 731.296 C463.264 729.02 461.916 727.053 459.96 726.123 L460.782 724.554 " stroke-width="1.0"/>
      <path stroke="black" fill="#aef9ff" d="M472.88 730.137 L474.842 730.137 L474.842 734.117 L472.88 734.117 L472.88 730.137 M474.842 729.716 L472.88 729.716 L472.88 728.388 L474.842 728.388 L474.842 729.716 " stroke-width="1.0"/>
      <path stroke="black" fill="#15ffea" d="M495.495 731.236 C495.082 731.236 495.012 731.649 495.012 732.062 C495.012 732.481 495.082 732.925 495.511 732.925 C495.931 732.925 495.993 732.481 495.993 732.107 C495.993 731.649 495.931 731.236 495.495 731.236 M496.079 734.117 L496.079 733.391 L496.063 733.391 C495.791 733.986 495.394 734.208 494.754 734.208 C493.455 734.208 492.972 733.215 492.972 732.062 C492.972 730.984 493.478 730.046 494.701 730.046 C495.223 730.046 495.62 730.221 495.924 730.649 L495.939 730.649 L495.939 728.388 L497.901 728.388 L497.901 734.117 L496.079 734.117 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M520.384 733.97 L520.749 733.97 L520.749 728.456 L520.384 728.456 L520.384 733.97 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M520.384 728.456 L520.749 728.456 L520.749 733.97 L520.384 733.97 L520.384 728.456 " stroke-width="1.0"/>
      <path stroke="black" fill="#aef9ff" d="M469.174 733.017 L469.719 733.017 C470.101 733.017 470.513 732.878 470.513 732.466 C470.513 732.008 470.125 731.924 469.734 731.924 L469.174 731.924 L469.174 733.017 ZM469.174 730.824 L469.696 730.824 C470.062 730.824 470.46 730.717 470.46 730.313 C470.46 729.915 470.116 729.793 469.742 729.793 L469.174 729.793 L469.174 730.824 ZM467.212 728.785 L470.63 728.785 C471.572 728.785 472.5 729.082 472.5 730.099 C472.5 730.832 472.102 731.222 471.378 731.404 L471.378 731.42 C472.133 731.52 472.6 732.1 472.6 732.634 C472.6 733.979 471.494 734.116 470.365 734.116 L467.212 734.116 L467.212 728.785 " stroke-width="1.0"/>
      <path stroke="black" fill="#15ffea" d="M480.572 728.785 L483.655 728.785 L484.364 731.702 L484.379 731.702 L485.08 728.785 L488.156 728.785 L488.156 734.116 L486.287 734.116 L486.255 729.931 L486.24 729.931 L485.173 734.116 L483.437 734.116 L482.402 729.931 L482.371 729.931 L482.346 734.116 L480.572 734.116 L480.572 728.785 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M503.969 733.784 C503.706 733.924 503.201 734.056 502.563 734.056 C501.297 734.056 500.086 733.24 500.086 731.384 C500.086 729.831 501.088 728.65 502.735 728.65 C503.403 728.65 503.791 728.798 503.954 728.883 L503.83 729.193 C503.565 729.062 503.184 728.969 502.743 728.969 C501.329 728.969 500.466 729.885 500.466 731.392 C500.466 732.813 501.283 733.73 502.68 733.73 C503.13 733.73 503.565 733.636 503.861 733.488 L503.969 733.784 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M503.969 733.784 C503.706 733.924 503.201 734.056 502.563 734.056 C501.297 734.056 500.086 733.24 500.086 731.384 C500.086 729.831 501.088 728.65 502.735 728.65 C503.403 728.65 503.791 728.798 503.954 728.883 L503.83 729.193 C503.565 729.062 503.184 728.969 502.743 728.969 C501.329 728.969 500.466 729.885 500.466 731.392 C500.466 732.813 501.283 733.73 502.68 733.73 C503.13 733.73 503.565 733.636 503.861 733.488 L503.969 733.784 Z" stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M512.984 729.419 L512.984 730.242 L514.047 730.242 L514.047 730.537 L512.984 730.537 L512.984 732.961 C512.984 733.434 513.131 733.737 513.528 733.737 C513.721 733.737 513.861 733.706 513.955 733.675 L514.001 733.962 C513.876 734.017 513.707 734.055 513.481 734.055 C513.208 734.055 512.984 733.962 512.842 733.784 C512.673 733.59 512.611 733.279 512.611 732.9 L512.611 730.537 L511.982 730.537 L511.982 730.242 L512.611 730.242 L512.611 729.559 L512.984 729.419 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M512.984 729.419 L512.984 730.242 L514.047 730.242 L514.047 730.537 L512.984 730.537 L512.984 732.961 C512.984 733.434 513.131 733.737 513.528 733.737 C513.721 733.737 513.861 733.706 513.955 733.675 L514.001 733.962 C513.876 734.017 513.707 734.055 513.481 734.055 C513.208 734.055 512.984 733.962 512.842 733.784 C512.673 733.59 512.611 733.279 512.611 732.9 L512.611 730.537 L511.982 730.537 L511.982 730.242 L512.611 730.242 L512.611 729.559 L512.984 729.419 Z" stroke-width="1.0"/>
      <path stroke="black" fill="#aef9ff" d="M477.505 733.063 C477.887 733.063 477.941 732.634 477.941 732.054 C477.941 731.58 477.887 731.191 477.505 731.191 C477.125 731.191 477.077 731.58 477.077 732.054 C477.077 732.634 477.125 733.063 477.505 733.063 M477.505 730.045 C479.14 730.045 480.028 730.717 480.028 732.054 C480.028 733.429 479.14 734.207 477.505 734.207 C475.862 734.207 474.99 733.429 474.99 732.054 C474.99 730.717 475.862 730.045 477.505 730.045 " stroke-width="1.0"/>
      <path stroke="black" fill="#15ffea" d="M491.231 731.748 C491.231 731.442 491.184 730.962 490.741 730.962 C490.289 730.962 490.188 731.437 490.211 731.748 L491.231 731.748 ZM490.218 732.588 C490.25 733.085 490.725 733.199 491.3 733.199 C491.838 733.199 492.322 733.109 492.632 732.947 L492.632 733.979 C492.111 734.138 491.488 734.207 490.802 734.207 C489.16 734.207 488.288 733.429 488.288 732.054 C488.288 730.717 489.16 730.045 490.802 730.045 C492.376 730.045 492.928 731.007 492.928 732.351 L492.928 732.588 L490.218 732.588 " stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M507.176 731.773 C507.184 731.252 506.967 730.459 506.065 730.459 C505.242 730.459 504.885 731.197 504.823 731.773 L507.176 731.773 ZM504.8 732.067 C504.8 733.264 505.452 733.73 506.221 733.73 C506.764 733.73 507.044 733.62 507.254 733.52 L507.347 733.816 C507.208 733.893 506.827 734.056 506.174 734.056 C505.102 734.056 504.435 733.272 504.435 732.176 C504.435 730.934 505.149 730.157 506.112 730.157 C507.331 730.157 507.564 731.307 507.564 731.842 C507.564 731.943 507.564 732.005 507.549 732.067 L504.8 732.067 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M504.8 732.067 C504.8 733.264 505.452 733.73 506.221 733.73 C506.764 733.73 507.044 733.62 507.254 733.52 L507.347 733.816 C507.208 733.893 506.827 734.056 506.174 734.056 C505.102 734.056 504.435 733.272 504.435 732.176 C504.435 730.934 505.149 730.157 506.112 730.157 C507.331 730.157 507.564 731.307 507.564 731.842 C507.564 731.943 507.564 732.005 507.549 732.067 L504.8 732.067 Z" stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M507.176 731.773 C507.184 731.252 506.967 730.459 506.065 730.459 C505.242 730.459 504.885 731.197 504.823 731.773 L507.176 731.773 Z" stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M508.41 731.136 C508.41 730.793 508.394 730.537 508.379 730.242 L508.721 730.242 L508.753 730.917 L508.767 730.917 C508.969 730.498 509.435 730.156 510.049 730.156 C510.406 730.156 511.346 730.335 511.346 731.734 L511.346 733.971 L510.981 733.971 L510.981 731.765 C510.981 731.089 510.717 730.476 509.948 730.476 C509.42 730.476 508.977 730.849 508.83 731.329 C508.798 731.423 508.775 731.547 508.775 731.663 L508.775 733.971 L508.41 733.971 L508.41 731.136 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M508.41 731.136 C508.41 730.793 508.394 730.537 508.379 730.242 L508.721 730.242 L508.753 730.917 L508.767 730.917 C508.969 730.498 509.435 730.156 510.049 730.156 C510.406 730.156 511.346 730.335 511.346 731.734 L511.346 733.971 L510.981 733.971 L510.981 731.765 C510.981 731.089 510.717 730.476 509.948 730.476 C509.42 730.476 508.977 730.849 508.83 731.329 C508.798 731.423 508.775 731.547 508.775 731.663 L508.775 733.971 L508.41 733.971 L508.41 731.136 Z" stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M514.77 731.36 C514.77 730.987 514.753 730.6 514.739 730.242 L515.081 730.242 L515.095 730.996 L515.111 730.996 C515.275 730.529 515.663 730.156 516.174 730.156 C516.221 730.156 516.277 730.165 516.323 730.173 L516.323 730.529 C516.269 730.515 516.212 730.515 516.145 730.515 C515.631 730.515 515.267 730.957 515.166 731.555 C515.15 731.664 515.134 731.788 515.134 731.912 L515.134 733.971 L514.77 733.971 L514.77 731.36 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M514.77 731.36 C514.77 730.987 514.753 730.6 514.739 730.242 L515.081 730.242 L515.095 730.996 L515.111 730.996 C515.275 730.529 515.663 730.156 516.174 730.156 C516.221 730.156 516.277 730.165 516.323 730.173 L516.323 730.529 C516.269 730.515 516.212 730.515 516.145 730.515 C515.631 730.515 515.267 730.957 515.166 731.555 C515.15 731.664 515.134 731.788 515.134 731.912 L515.134 733.971 L514.77 733.971 L514.77 731.36 Z" stroke-width="1.0"/>
      <path stroke="black" fill="#000000" d="M518.909 731.975 C518.092 731.951 517.022 732.075 517.022 732.968 C517.022 733.504 517.378 733.745 517.765 733.745 C518.388 733.745 518.738 733.364 518.87 732.999 C518.892 732.922 518.909 732.844 518.909 732.775 L518.909 731.975 ZM519.274 733.085 C519.274 733.38 519.289 733.684 519.328 733.97 L518.994 733.97 L518.94 733.435 L518.917 733.435 C518.738 733.722 518.327 734.055 517.729 734.055 C516.982 734.055 516.633 733.527 516.633 733.03 C516.633 732.168 517.395 731.649 518.909 731.663 L518.909 731.57 C518.909 731.197 518.837 730.46 517.953 730.467 C517.628 730.467 517.285 730.553 517.022 730.747 L516.905 730.476 C517.239 730.25 517.659 730.157 517.993 730.157 C519.072 730.157 519.274 730.964 519.274 731.632 L519.274 733.085 " stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M519.274 733.085 C519.274 733.38 519.289 733.684 519.328 733.97 L518.994 733.97 L518.94 733.435 L518.917 733.435 C518.738 733.722 518.327 734.055 517.729 734.055 C516.982 734.055 516.633 733.527 516.633 733.03 C516.633 732.168 517.395 731.649 518.909 731.663 L518.909 731.57 C518.909 731.197 518.837 730.46 517.953 730.467 C517.628 730.467 517.285 730.553 517.022 730.747 L516.905 730.476 C517.239 730.25 517.659 730.157 517.993 730.157 C519.072 730.157 519.274 730.964 519.274 731.632 L519.274 733.085 Z" stroke-width="1.0"/>
      <path fill="none" stroke="#000000" d="M518.909 731.975 C518.092 731.951 517.022 732.075 517.022 732.968 C517.022 733.504 517.378 733.745 517.765 733.745 C518.388 733.745 518.738 733.364 518.87 732.999 C518.892 732.922 518.909 732.844 518.909 732.775 L518.909 731.975 Z" stroke-width="1.0"/>
     </g>
    </svg>
   </div>
  </div>
 </div>
</div>
