<?xml version="1.0" encoding="UTF-8"?>
<div id="g.1.1" last="g.1.0" next="g.1.2" class="concatenated" xmlns="http://www.w3.org/1999/xhtml">
 <div id="g.1.3" last="g.1.2" next="g.1.4" class="omit">
  <p>Factors influencing lysis time stochasticity in bacteriophage ?
  <span>
   <b> [g.1.3]</b>
  </span>
  </p>
 </div>
 <p id="g.1.4" last="g.1.3" next="g.1.5" class="omit">John J Dennehy
  <sup>1,2*</sup>and Ing-Nang Wang
  <sup>1</sup>
  <span>
   <b> [g.1.4]</b>
  </span>
 </p>
 <div id="g.1.6" last="g.1.5" next="g.1.7" class="omit">
  <p>
   <b>Background Some phenotypic variation arises from randomness in cellular processes despite identical environments and genotypes [1-9]. Population heterogeneity, resulting from such molecular stochasticity, has been documented in many microbial organisms including bacteriophage (phage) ? [10-13], </b>
   <i>Escherichia coli</i> [14-16],
   <i>Bacillus subtilis </i>[17,18]
   <i> </i>and
   <i> Saccharomyces cerevisiae </i>[19-24].
   <i> </i>This within-population variation can have far reaching life history consequences. For example, experimentally reducing noise in the expression of ComK decreased the number of competent
   <i>B. subtilis</i> cells in one study [18]. In another study, mutants of
   <i>S. cerevisiae</i> showing greater heterogeneity in survival had higher rates of occasional-cell survival during high stress conditions than did wild-type cells [25].
  <span>
   <b> [g.1.6]</b>
  </span>
  </p>
 </div>
 <div id="g.1.7" last="g.1.6" next="g.1.8" class="omit">
  <div>
   <div>
    <p>* Correspondence: john.dennehy@qc.cuny.edu</p>
    <p>
     <sup>1</sup>Department of Biological Sciences, University at Albany, 1400 Washington Avenue, Albany, NY 12222, USA Full list of author information is available at the end of the article
  <span>
   <b> [g.1.7]</b>
  </span>
    </p>
   </div>
  </div>
 </div>
 <div id="g.1.8" last="g.1.7" next="g.1.9" class="omit">
  <p>Because of their simplicity and ease of manipulation, phages are excellent models to explore the life history consequences of molecular stochasticity. Many phages use a “ holin-endolysin” system to compromise two physical barriers, the cell membrane and the peptidoglycan layer, in order to lyse an infected host cell [26,27]. Although there are some variations on the theme, holin usually forms a hole(s) in the inner membrane, thus either allowing soluble endolysin into the periplasmic space [28,29] or activating the membrane-tethered endolysin already translocated to the periplasm [30-32]. Endolysin then digests the peptidoglycan, causing host cell lysis.</p>
  <p>The most extensively studied lysis system is that of phage l, which consists of four genes:
   <i>S</i> (encodes holin and antiholin),
   <i>R</i> (encodes endolysin),
   <i>Rz</i>, and
   <i>Rz1 (encode an integral inner membrane protein and an outer membrane lipoprotein, respectively). All genes are co-transcribed from the late promoter </i>
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>during
   <i> </i>the late phase of the lytic cycle [26,27,33,34]. Under typical
  <span>
   <b> [g.1.8]</b>
  </span>
 laboratory conditions, only
   <i>S</i> and
   <i>R</i> are needed for host lysis, though both
   <i>Rz</i> and
   <i>Rz1</i> are essential in the presence of high concentrations of divalent cations [33-35].
  </p>
  <p>The lytic pathway of phage ? is commonly divided into the early, delayed early, and late phases. Transitions between stages are triggered by well-characterized molecular actions involving gene transcription and translation [36]. Consequently, the timing of when individual cells enter each phase greatly influences the length of individual lysis times. A recent study by Amir
   <i>et al.</i> [10] showed that 69% of the total lysis time variance is due to variation in the time interval between the onset of the
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter
   <i> </i>and
   <i> </i>the
   <i> </i>eventual
   <i> </i>lysis
   <i> </i>(see
   <i> </i>APPENDIX A). This observation suggests that a large portion of the observed lysis time stochasticity is a
   <i>de novo</i> phenomenon, confined to the production and accumulation of holin proteins in the cell membrane, rather than a direct carryover from the various upstream stochastic events.
  </p>
  <p>The formation of the ? holin hole in the membrane is hypothesized to be a multi-step process that starts with the transcription of the late mRNA and the translation of the S holin protein. The resulting holin monomers are then inserted into the cell membrane, where they dimerize, then oligomerize [37], eventually leading to the formation of higher-order holin aggregates, or rafts, in the cell membrane. At a time that is specific to the holin protein sequence, the holin rafts are transformed into a membrane lesion(s) &gt; 300 nm across [38], which is large enough for the passage of a 500 KDa protein [28,29]. Lysis ensues after endolysin digests the peptidoglycan. Thus, by
  <span>
   <b> [g.2.2]</b>
  </span>
regulating endolysin’ s access to the peptidoglycan, holin controls the timing of lysis [26,27].</p>
  <p>To formalize the heuristic model of holin hole formation described by Wang
   <i>et al.</i> [28], Ryan and Rutenberg [39] proposed a two-stage nucleation model, in which the production rate of the holin monomers and holin self-affinity contribute to the aggregation of holin rafts. Raft aggregation is opposed by thermal Brownian motion which tends to disintegrate rafts into their holin constituents. As the rafts grow and then exceed a certain critical size (the first stage of nucleation), the probability of a second stage nucleation (triggering to hole formation) increases (Figure 1). According to this model, lysis time stochasticity is the inevitable outcome of each infected cell in the population following its own time course of growth in holin raft size. However, a recent study [40] using C-terminus GFP-fused ? S holin protein showed that, for most of the latent period, holin proteins are distributed uniformly in a relatively mobile state in the cell membrane. At a time that coincided with the triggering time, large immobile holin rafts suddenly appeared in the membrane. The transition from uniformly distributed holin to holin rafts occurred in less than a minute. Although it is not clear whether these large rafts correspond to the membrane holes observed by cryoelectron microscopy [38], this study nevertheless casts doubt on the previously hypothesized importance of holin raft size growth as the determining factor in lysis timing [28,39]. Rather, it is proposed that the lysis time is determined by when a critical holin concentration is reached in the cell membrane (Figure 1). According to this model, lysis time stochasticity is mainly the result of variation in
  <span>
   <b> [g.2.3]</b>
  </span>
the timing of reaching the critical holin concentration in the membrane.</p>
  <p>Typically, the lysis time of a phage is estimated using a one-step growth curve [41-43]. In the case of phage ? , however, the availability of thermally-inducible
   <i>E. coli</i> ? lysogens allows a more precise determination of the lysis time by following the decline of culture turbidity [26,44]. Direct observation of the lysis of individual ? lysogenic cells [45] confirmed that the precipitous decline of culture turbidity, commonly observed among thermallyinduced ? lysogen cultures, is a reflection of the saltatory nature of individual lysis events at the microscopic level. However, it is not clear to what extent the seemingly high synchronicity of lysis is influenced by various aspects of phage biology and host growth conditions. In this study, we used a simple experimental setup to assess how lysis time stochasticity is affected by allelic variation in the S protein, late promoter
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity,
   <i> </i>host
   <i> </i>growth rate, and the timing of energy poison KCN addition. Our
  <span>
   <b> [g.3.2]</b>
  </span>
 results establish the ranges and limits of lysis time stochasticity under various conditions.
  <span>
   <b> [g.3.3]</b>
  </span>
  </p>
 </div>
 <div id="g.3.4" last="g.3.3" next="g.3.5" class="omit">
  <p>
   <b>Results Using a microscope-mounted, temperature-controlled perfusion chamber, we observed and recorded individual lysis events of thermally-induced </b>
   <i>Escherichia coli</i> l lysogens (Figure 2A). These observations revealed a considerable amount of variation in lysis time for the wildtype (WT) ? phage (Table 1; Figure 2B). Although the mean lysis time for the WT ? phage was 65.1 min, lysis times for individual lysogenic cells ranged from 45.4 to 74.5 min. Given that phage progeny accumulate linearly at ~7.7 phage per minute beginning ~28 min after lysis induction [46], the ~30 min range of lysis times could result in a three-fold difference in burst size between phages that lyse early and those that lyse late. This result motivated further exploration of variation in lysis time among other ? strains.
  <span>
   <b> [g.3.4]</b>
  </span>
  </p>
 </div>
 <div id="g.4.3" last="g.4.2" next="g.4.4" class="omit">
  <p>Effect of allelic variation in holin sequence It has long been known that different holin alleles show different lysis times [37,46,47]. However, it is not clear to what extent allelic differences in holin protein would affect the lysis timing of individual cells. To gain further insight, we determined the MLTs (mean lysis times) and SDs (standard deviations) of lysis time for 14 isogenic l lysogens differing in their
   <i>S</i> holin sequences (see APPENDIX B for our rationale for using SD as the measure for lysis time stochasticity). The directly observed MLTs (Table 1) were longer than those reported previously [46]. This discrepancy was mainly due to the fact that, in previous work, lysis time was defined by the time point when the turbidity of the lysogen culture began to decline, whereas in our current measurement, it was the mean of all individual lysis times observed for a particular phage strain. Figure 3A revealed a significant positive relationship
 between MLT and SD (
   <i>F = 8.42, </i>
   <i>p</i> = 0.0133). How[1,12] ever, we did not observe a significant relationship
  <span>
   <b> [g.4.3]</b>
  </span>
between MLT and another commonly used measure of stochasticity, the coefficient of variation (CV, defined as SD/MLT; [15,25,48,49]) (
   <i>F</i>
   <sub>[1,12]</sub>= 1.50,
   <i>p</i> = 0.2445), indicating a proportional increase of the SD with the MLT. Figure 3A also reveals a relatively scattered relationship between the MLTs and the SDs (adjusted
   <i>R</i>
   <sup>2</sup>= 0.363), with several instances in which strains with similar MLTs are accompanied by very different SDs. For example, the mean lysis times for IN56 and IN71 were 65.1 and 68.8 min, but the SDs were 3.2 and 7.7 min, respectively. Apparently the observed positive relationship is only a general trend, not an absolute. The scattering of the plot also suggests that different missense mutations in the holin sequence can influence MLT and SD somewhat independently.
  <span>
   <b> [g.5.2]</b>
  </span>
 </p>
  </div>
 <div id="g.5.3" last="g.5.2" next="g.5.4" class="omit">
  <p>Effect of late promoter
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity Transcription of the late genes, including the holin gene S,
   <i> </i>from
   <i> </i>the
   <i> p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter
   <i> </i>marks
   <i> </i>the
   <i> </i>beginning
   <i> </i>of
   <i> </i>the late stage of the ? lytic development [36]. Since the major determinant of lysis time is thought to be when a critical holin concentration is reached in the cell membrane [40], reduced promoter activity should not only lengthen the lysis time, as shown in a previous study [50], but should also increase the lysis time stochasticity
  </p>
  <p>[51,52]. As shown in Figure 3B, our data showed a negative relationship between the
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity,
   <i> </i>and
   <i> </i>the
   <i> </i>MLTs, SDs, and CVs. However, the increase of the
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity had a diminishing influence on both the MLTs, as has been shown previously [50], and the associated SDs and CVs (see Table 2). Interestingly, linear regressions (Figure 3C) showed a much tighter, positive relationship between the MLTs and the SDs (
   <i>F</i>
   <sub>[1,3]</sub>= 81.04,
   <i>p</i> = 0.0029; adjusted
   <i>R</i>
   <sup>
    <i>2</i>
   </sup>= 0.952;
   <i>y</i> = -15.7 + 0.3
   <i>x</i>) and a significant positive relationship between the MLTs and CVs (
   <i>F</i>
   <sub>[1,3]</sub>= 14.51,
   <i>p</i> = 0.0318, result not shown in the figure). That is, for the WT
   <i>S</i> gene, every 1 minute increase in the MLT corresponds to 0.3 minute increase in lysis time stochasticity.
  <span>
   <b> [g.5.3]</b>
  </span>
  </p>
 </div>
 <div id="g.5.4" last="g.5.3" next="g.5.5" class="omit">
  <p>Effect of Host Growth Rates In general, cells growing at a faster rate have higher concentrations of various biosynthesis machineries [53]. Since the expression of the phage holin gene is entirely dependent on the host, we hypothesized that a lower host growth rate would lead to a lower rate of holin protein synthesis, thus resulting in a longer lysis time and increased lysis time stochasticity. In the phage T4, it was shown that lysis time was negatively correlated</p>
  <p>with host growth rate [54]. We determined the MLTs and SDs for wild-type l lysogen grown in four different growth media: standard LB (lysogeny broth [55]), 20% LB, Davis minimal salts
  <span>
   <b> [g.5.4]</b>
  </span>
  </p>
medium (DM) with 20 mM glucose, and DM with 40 mM glycerol, resulting in growth rates of 1.01 ± 0.07, 0.93 ± 0.05, 0.49 ± 0.04, and 0.35 ± 0.01 h
   <sup>-1</sup>(mean ± 95% confidence limits), respectively (see Table 2). As shown in Figure 3D, lower growth rates led to increased lysis time SDs (
   <i>F</i>
   <sub>[1,2]</sub>= 24.50,
   <i>p</i> = 0.0385) and CVs (
   <i>F</i>
   <sub>[1,2]</sub>
  </p>
  <p>= 46.24,
   <i>p</i> = 0.0209). A similar negative relationship was
  </p>
  <p>also apparent for the MLTs. However, because of the</p>
  <p>case of the LB medium, in which the higher growth rate</p>
  <p>actually resulted in a slightly longer MLT, the observed</p>
  <p>negative relationship was not significant (
   <i>F</i>
   <sub>[1,2]</sub>= 6.44,
   <i>p</i>
  </p>
  <p>= 0.1265). Interestingly, neither the SDs (
   <i>F</i>
   <sub>[1,2]</sub>= 16.11,
   <i>p = 0.0568) nor the CVs (</i>
   <i>F</i>
   <sub>[1,2]</sub>= 6.04,
   <i>p</i> = 0.133) was sig-
  </p>
  <p>nificantly associated with the MLTs.
  <span>
   <b> [g.5.6]</b>
  </span>
  </p>
 </div>
 <div id="g.5.7" last="g.5.6" next="g.6.0" class="omit">
  <p>Effects of KCN Addition The energy poison potassium cyanide, KCN, has long been used in phage research to trigger premature lysis [43]. Typically, after KCN addition, culture turbidity declines precipitously [44], indicating that individual lysis events are relatively synchronous. The KCNinduced premature lysis is thought to be mediated through a collapsed proton motive force (PMF) resulting from a inhibition of the bacterial respiratory chain. As has been shown with ? S holin, a 40% drop in the PMF
  <span>
   <b> [g.5.7]</b>
  </span>
triggers lysis [45]. Without a constant supply of ATP, the production of holin protein would also be terminated. If KCN is added soon after thermal induction of the lysogen culture, few holin proteins would have been made before the termination of holin production. Consequently, it should take a longer time for the holin proteins in the membrane to transition from a diffused state to aggregated rafts. Therefore, after the cessation of holin production by KCN addition, it may take a longer time, on average, before any lysis events are observed. On the other hand, if KCN is added late, a larger proportion of the thermally-induced lysogenic cells should have accumulated enough holin proteins in the cell membrane such that they could be triggered to form holin holes quickly. That is, the addition of KCN should prompt the rapid formation of holin holes, thus resulting in an almost immediate and synchronous lysis of most of the cells in the population. Based on the aforementioned scenarios, we expected that (1) the time delay between the time of KCN addition (
   <i>t</i>
  </p>
  <p>) and the KCN eventual mean lysis time (
   <i>t</i>
   <sub>L</sub>) (
   <i>i.e.</i>,
   <i>t</i>
   <sub>L</sub>-
   <i>t</i>
   <sub>KCN</sub>) would be negatively correlated with the timing of KCN addition, and (2)
   <i>t</i>
   <sub>KCN</sub>would be negatively correlated with lysis
  </p>
  <p>time stochasticity. Figure 4A shows a significant negative relationship between
   <i>t</i>
   <sub>L</sub>-
   <i>t</i>
   <sub>KCN</sub>and
   <i>t</i>
   <sub>KCN</sub>. As KCN was added later in time (
   <i>i. e.</i>, closer to the normal lysis time of 65.1 min), the time delay between addition of KCN and the MLT was reduced (a quadratic fit,
   <i>F</i>
   <sub>[2,4]</sub>= 12.87,
   <i>p</i> = 0.0181, adjusted
   <i>R</i>
   <sup>2</sup>= 0.798). In fact, when added 55 min after induction (
   <i>i.e.</i>, 10 min before the normal MLT), the time delay was only 2.6 min, almost instantaneous when compared to the 2 min sampling rate of the sipperequipped spectrophotometer method of lysis time determination [46]. Interestingly, a theoretical study of lysis time stochasticity by Ryan and Rutenberg also showed a
  <span>
   <b> [g.6.2]</b>
  </span>
seemingly convex relationship between
   <i>t</i>
   <sub>L</sub>-
   <i>t</i>
   <sub>KCN</sub>and t
   <sub>KCN</sub>[[39], their figure five].
  </p>
  <p>The effects of
   <i>t</i>
   <sub>KCN</sub>on lysis time SDs and CVs are shown in Figure 4B. Again, we witnessed the expected pattern of a significant negative relationship between t
   <sub>KCN</sub>and the SDs (a quadratic fit,
   <i>F</i>
   <sub>[2,4]</sub>= 9.91,
   <i>p</i> = CVs (a quadratic fit,
   <i>F</i>
   <sub>[2,4]</sub>= 16.03,
   <i>p</i> = 0.0282, adjusted R
   <sup>2</sup>= 0.834). These results showed that the later in time KCN was added, the less variation there was in individual lysis times. In fact, the lowest SD (1.45 min) and lowest CV (2.53%) were observed when KCN was added 55 min after induction. This was a significant two-fold reduction in the SD when compared normal lysis conditions (see Table 1 for strain IN56 with the SD = 3.24 min; Student’ s
   <i>t</i> = 15.45,
   <i>p</i> &lt; 0.0001, using the standard deviation for the SD in Box 7.1 of [56]). This observation indicated that individual triggering for hole formation during the normal progression of cell lysis was relatively asynchronous when compared to the artificial method of acute triggering by KCN addition. Similar to the effect of growth rate, a linear regression
  </p>
  <p>of the SDs (
   <i>F</i>
   <sub>[1,5]</sub>= 0.60,
   <i>p</i> = 0.4726) or CVs (
   <i>F</i>
   <sub>[1,5]</sub>= 0.328,
   <i>p</i> = 0.5917) against the MLTs did not yield significant result. Another interesting aspect of the relationship between
   <i>t</i>
   <sub>KCN</sub>and the lysis time SDs is that the SDs drop precipitously when KCN is added about 35 min after induction. This observation suggests that, approximately 35 min after thermal induction, the majority of the lysogenic cells have accumulated enough holin proteins in the cell membrane to form holes immediately if triggered.
  <span>
   <b> [g.6.3]</b>
  </span>
  </p>
 </div>
 <div id="g.6.4" last="g.6.3" next="g.6.5" class="omit">
  <p>
   <b>Discussion The current model of holin hole formation hypothesizes that ? phage lysis timing is mainly determined by when</b>
  <span>
   <b> [g.6.4]</b>
  </span>
a critical concentration of holin proteins is reached in the cell membrane [40] (Figure 1, dark arrows). According to this model, any factor that influences the holin protein production should also affect the timing of lysis. Furthermore, the realized rate of holin production in each cell should also be subjected to stochastic influences impacting the various upstream biochemical reactions, such as gene transcription and translation, that lead to holin production. As has been shown by others, the lower the average rates of the biochemical reactions, the more prominent the cell-to-cell variation is [51,52].
  <span>
   <b> [g.7.2]</b>
  </span>
  </p>
 </div>
 <div id="g.7.3" last="g.7.2" next="g.7.4" class="omit">
  <p>Manipulation of holin production rate In our study, we manipulated the holin production rate</p>
  <p>by manipulating the ?
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity
   <i> </i>and
   <i> </i>the
   <i> </i>lysogen growth rate. We observed that, in general, treatments expected to result in higher holin production rates (
   <i>e.g.</i>, high
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity
   <i> </i>or
   <i> </i>high
   <i> </i>lysogen
   <i> </i>growth
   <i> </i>rate)
   <i> </i>also resulted in shorter MLTs and smaller SDs (Figure 3B and 3D). Furthermore, it was surprising that the combined MLTs and SDs, despite being from two different experimental treatments, namely
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activity
   <i> </i>and
   <i> </i>lysogen growth rate, showed almost identical positive correlations, even after excluding the far-flung data point with the longest MLT and largest SD (obtained with strain SYP028, see Table 2) from the analysis (Figure 3C). This result suggests that, irrespective of how the MLT was achieved, as long as the MLTs are the same, we should expect to observe similar SDs. For the wild-type ? S holin sequence, any factor that results in 1.0 min increase in MLT would be accompanied by a concomitant 0.3 min increase in the SD. It would be interesting to conduct a similar experiment with different holin sequences to see if the rate of SD increase is sequence-
  </p>
  <p>specific. Regarding the effects of host growth rate on lysis time stochasticity, it is interesting to note the following. Amir et al. [10]
   <i> </i>found
   <i> </i>that
   <i> </i>the
   <i> </i>MLTs,
   <i> </i>SDs,
   <i> </i>and
   <i> </i>CVs,
   <i> </i>following UV induction, ranged from 72 min, 9 min, and 12.5% respectively for ? lysogens alone to 99 min, 14 min, and 14.1% respectively for ? lysogens carrying pRGFP reporter plasmid and 117 min, 19 min, and 15.8% respectively for ? lysogens carrying pR’ -tR’ -GFP reporter plasmid (all values are extracted from their figures six A and B). Since their ? lysogens were grown in M9 minimal salts medium plus various growth factors and 0.4% glucose at 37°C, it is similar to our Davis minimal salts medium with glucose, from which we obtained the comparable values of 70.3 min, 6.3 min, and 8.96% respectively (see Table 2). It is not clear whether the difference between these two SDs is the result of different methods used for lysogen induction (thermal vs. UV induction) or different growth media, but the MLTs are virtually identical. Their result also indirectly confirmed
  <span>
   <b> [g.7.3]</b>
  </span>
our current result that host physiology (which is presumably somewhat perturbed in their lysogen strains carrying the medium-copy reporter plasmids) would affect the overall MLTs and SDs of lysis time.
  <span>
   <b> [g.7.4]</b>
  </span>
  </p>
 </div>
 <div id="g.7.5" last="g.7.4" next="g.7.6" class="omit">
  <p>Manipulation of holin protein sequence Barring potential post-translational modifications due to differences in holin protein sequence (
   <i>e.g.</i>, differential rate in proteolysis), isogenic ? strains expressing different holin sequences would have a similar average rate of holin accumulation in the membrane and consequently the same distribution of holin proteins among the cells across different lysogen populations. That is, at any given moment, we would expect a certain proportion of cells to accumulate a certain number of holin molecules in the membrane, irrespective of the holin sequences. The observed differences in MLTs, as the result of differences in holin sequence, can be seen as a reflection of different set-points for critical concentrations in an increasingly crowded cell membrane [40]. Presumably, a sequence with a lower set-point would not only result in a shorter MLT, but also a smaller SD as well. However, the existence of similar MLTs, but very different SDs, suggests that missense mutations in the holin sequence not only affect the set-point for spontaneous triggering, but also impact the robustness of the setpoint. For example, some mutations may be relatively insensitive to the critical holin concentration, thus resulting in proportionally more cells that are triggered earlier and later than expected, hence greater lysis time stochasticity.
  <span>
   <b> [g.7.5]</b>
  </span>
  </p>
 </div>
 <div id="g.7.6" last="g.7.5" next="g.8.0" class="omit">
  <p>Effect of energy poison KCN It is well known that addition of the energy poison, KCN, to induced lysogen cultures will accelerate the onset of lysis [44]. Our results also confirmed this observation (see Table 2). However, it is not clear how this accelerated lysis would affect the lysis time stochasticity. From anecdotal observations, the addition of KCN seems to synchronize lysis, thus resulting in a precipitous decline of lysogen culture turbidity. Our study showed that the timing of KCN addition was inversely related to lysis time stochasticity (see Figure 4B). In fact, the smallest SD (1.45 min) was achieved by adding KCN at 55 min after thermal induction (see Table 2), a time where normally only about 1% of the cells have lysed. The almost synchronous lysis when KCN was added 55 min post thermal induction suggests that most cells would have already accumulated enough holin proteins</p>
  <p>in the cell membrane to form a hole. Besides collapsing the PMF, the addition of KCN should also halt the production of holin protein, thus “ fixes” the amount of holin proteins on the cell membrane at the time of addition. The progressive decline in lysis time
  <span>
   <b> [g.7.6]</b>
  </span>
stochasticity as KCN was added later in time (see Figure 4B) strongly suggests that a larger supply of holin protein is a key factor in ensuring synchronous lysis. As more holin proteins are inserted into the cell membrane, the kinetics of raft formation gradually shifts from stochastic to deterministic and synchronous. In fact, there was a nearly five-fold decrease in lysis time stochasticity when the PMF was collapsed at 55 min after lysogen induction when compared to collapse at 25 min (see Table 2). It is also noted that the properties of the normally triggered and the prematurely triggered holin holes are quite distinct, with the prematurely triggered holes being much smaller than the normally triggered holes [28].
  <span>
   <b> [g.8.2]</b>
  </span>
  </p>
 </div>
 <div id="g.8.3" last="g.8.2" next="g.8.4" class="omit">
  <p>Evolutionary implication of lysis time stochasticity Both theoretical and experimental studies have demonstrated the importance of lysis timing on phage fitness [46,57-61]. However, it is not clear if lysis time stochasticity would have any impact on phage fitness. All else being equal, genotypes with reduced variances in offspring number would, in the long run, have higher fitness than genotypes that have the same mean offspring number, but larger variances [62,63]. Overall, this suggests that natural selection would tend to minimize stochasticity in phenotypes that are closely linked to Darwinian fitness. If the phage burst size is positively linked with the lysis time, as has been shown previously [46], then selection for reduced burst size stochasticity should lead to reduced lysis time stochasticity as well. Presumably, this hypothesis can be tested by competing two isogenic phage strains that have the same MLTs but very different lysis time SDs. Interestingly, inspection of Table 1 revealed that mutations introduced into WT ? holin sequence usually result in increased stochasticity, except in one case. It is not clear if this observation implies that the WT holin sequences have already been selected for reduced stochasticity in the wild as well. Experiments with more phage holins should provide some hints in this respect.
  <span>
   <b> [g.8.3]</b>
  </span>
  </p>
 </div>
 <div id="g.8.4" last="g.8.3" next="g.8.5" class="omit">
  <p>
   <b>Conclusions Even in a seemingly uniform environment, the lysis time</b>
  </p>
  <p>can vary greatly among individual ? lysogenic cells (lysis</p>
  <p>time stochasticity). The extent of stochasticity, as quan-</p>
  <p>tified by the standard deviation, depends on the quality</p>
  <p>(due to isogenic ? lysogens expressing different S pro-</p>
  <p>tein alleles) and quantity (manipulated by having differ-</p>
  <p>ent
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>activities
   <i> </i>and
   <i> </i>lysogen
   <i> </i>growth
   <i> </i>rates)
   <i> </i>of
   <i> </i>the
   <i> </i>holin
  </p>
  <p>protein, the major determinant of lysis timing in large-</p>
  <p>genome phages. There is a general positive trend</p>
  <p>between the mean lysis time and the degree of stochasti-</p>
  <p>city. However, this positive relationship is much tighter</p>
  <p>when difference in mean lysis time is due to holin quantity rather than quality. The pattern of lysis time
  <span>
   <b> [g.8.4]</b>
  </span>
stochasticity obtained by addition of KCN at various time points after lysogen induction showed a negative relationship between the timing of KCN addition and the level of lysis time stochasticity.
  <span>
   <b> [g.8.5]</b>
  </span>
  </p>
 </div>
 <div id="g.8.6" last="g.8.5" next="g.8.7" class="omit">
  <p>
   <b>Appendix A This section provides the rationale for partitioning lysis</b>
  </p>
  <p>time variance found in the study by Amir
   <i>et al.</i> [10]. For each UV-induced ? lysogenic cell, the lysis time
   <i>T can be divided into three time intervals: (1) </i>
   <i>t</i>
   <sub>1</sub>, the time interval between lysogen induction and the onset of
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>promoter, (2)
   <i>t</i>
   <sub>2</sub>, the time interval between the onset of the
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>promoter and the onset of the
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter,
   <i> </i>and (3)
   <i>t</i>
   <sub>3</sub>, the time interval between the onset of the
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ promoter and the eventual lysis. The following relationships describe the above time intervals and the empirically determined time intervals by Amir </i>
   <i>et al.</i> [10]:
   <i>t</i>
   <sub>1</sub>= t
   <sub>pR</sub>,
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>=
   <i>t</i>
   <sub>pR’ -tR’</sub>,
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>+
   <i>t</i>
   <sub>3</sub>=
   <i>t</i>
   <sub>lysis</sub>, and
   <i>t</i>
   <sub>3</sub>= ?
   <i>t</i> =
   <i>t</i>
   <sub>ly-</sub>
  </p>
  <p>
   <sub>sis</sub>-
   <i>t</i>
   <sub>pR’ -tR’</sub>. For,
   <i>T</i> =
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>+
   <i>t</i>
   <sub>3</sub>, the variance for the lysis time can be expressed as
   <i>VAR</i>(
   <i>T</i>) =
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>) +
   <i>VAR</i>(
   <i>t</i>
   <sub>2</sub>) + VAR(
   <i>t</i>
   <sub>3</sub>) + 2
   <i>COV</i> (
   <i>t</i>
   <sub>1</sub>,
   <i>t</i>
   <sub>2</sub>) + 2
   <i>COV</i> (
   <i>t</i>
   <sub>2</sub>,
   <i>t</i>
   <sub>3</sub>) + 2
   <i>COV</i> (
   <i>t</i>
   <sub>1</sub>,
   <i>t</i>
   <sub>3</sub>). While the authors did not provide all possible combinations of covariance, it is empirically determined that COV(
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>,
   <i>t</i>
   <sub>3</sub>) = 0, as shown in their figure seven E (
   <i>i. e.</i>,
   <i> </i>no
   <i> </i>correlation
   <i> </i>between
   <i> t</i>
   <sub>pR’ -tR’</sub>and ?
   <i>t</i>). That is,
   <i>COV (</i>
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>,
   <i>t</i>
   <sub>3</sub>) =
   <i>COV</i>(
   <i>t</i>
   <sub>1</sub>,
   <i>t</i>
   <sub>3</sub>) +
   <i>COV</i>(
   <i>t</i>
   <sub>2</sub>,
   <i>t</i>
   <sub>3</sub>) = 0. Although not empirically demonstrated, it seems unlikely that the timing of turning on either the
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>or
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>promoter would have a positive or negative effect on the assembly of lysis apparatus such that their effects would cancel each other out, resulting in the observed
   <i>COV</i>(
   <i>t</i>
   <sub>1</sub>,
   <i>t</i>
   <sub>3</sub>) + COV(
   <i>t</i>
   <sub>2</sub>,
   <i>t</i>
   <sub>3</sub>) = 0. Most likely, time intervals are mutually
  </p>
  <p>independent,
   <i>i.e.</i>,
   <i>COV</i>(
   <i>t</i>
   <sub>1</sub>,
   <i>t</i>
   <sub>3</sub>) =
   <i>COV</i>(
   <i>t</i>
   <sub>2</sub>,
   <i>t</i>
   <sub>3</sub>) = 0. The standard deviations ("absolute noise” in their terminology) for
   <i>t</i>
   <sub>pR’ -tR’</sub>and
   <i>t</i>
   <sub>lysis</sub>can be extracted from their figure six A using data determined from cells carrying the pR’ -tR’ -GFP plasmid. The estimated SDs for t
   <sub>pR’ -tR’</sub>and
   <i>t</i>
   <sub>lysis</sub>are ~10 min and ~18 min, respectively; therefore,
   <i>VAR</i>(
   <i>t</i>
   <sub>pR’ -tR’</sub>) = ~100 and
   <i>VAR</i>(
   <i>t</i>
   <sub>lysis</sub>) = ~324. The SD for
   <i>t</i>
   <sub>pR</sub>can be estimated by extrapolating the line connecting between lysis and
   <i>p</i>
   <sub>
    <i>R</i>
   </sub>
   <i>’ </i>onset
   <i> </i>to
   <i> </i>the
   <i> </i>20 min mean time at the
   <i>x</i>-axis (based on the result from cells carrying the pR-GFP plasmid in their figure six A). The corresponding SD for
   <i>t</i>
   <sub>pR</sub>is ~7 min, thus
   <i>VAR</i>(
   <i>t</i>
   <sub>pR</sub>) = ~49. Taken together,
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>) = 49,
   <i>VAR</i>(
   <i>t</i>
   <sub>2</sub>) = 51 (= VAR(
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>) -
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>) = 100 - 49 ), and
   <i>VAR</i>(
   <i>t</i>
   <sub>3</sub>) = 224 (=
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>+
   <i>t</i>
   <sub>3</sub>) -
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>+
   <i>t</i>
   <sub>2</sub>) = 324 - 100). That is,
   <i>VAR</i>(
   <i>t</i>
   <sub>1</sub>),
   <i>VAR</i>(
   <i>t</i>
   <sub>2</sub>), and
   <i>VAR</i>(
   <i>t</i>
   <sub>3</sub>) contributed to 15%, 16%, and 69% of total lysis time variance, respectively.
  <span>
   <b> [g.8.6]</b>
  </span>
  </p>
 </div>
 <div id="g.8.7" last="g.8.6" next="g.9.0" class="omit">
  <p>
   <b>Appendix B Studies of molecular stochasticity typically use the coefficient of variation (CV) as the measurement for the</b>
  <span>
   <b> [g.8.7]</b>
  </span>
degree of stochasticity [15,25,48,49]. Since CV is a composite statistic (defined as standard deviation/mean), it is sometimes difficult to discern whether an increase in the observed stochasticity (as quantified by CV) is due to decrease in mean or increase in SD. In some cases, a different metric, such as phenotypic noise strength (defined as variance/mean) [17,20], or a slight variant of it (defined as variance/squared mean) [19], has been used as well. Many times, it is not clear why a particular metric is used, except in the instance where the phenotypic noise strength is used to test against an
   <i>a priori expectation of a Poisson distribution, for which variance/mean = 1.</i>
  </p>
  <p>It is understandable why the CV, or a variant, is used in certain situations. For example, if the means are drastically different from each other or a comparison is made between measurements using different units [[56], pp. 57-59.]. In our study, however, the means were not very different and the same measuring unit (
   <i>i.e.</i>, min) was used. Therefore, we presented our means and SDs separately and then jointly as CVs. Except in one instance where presenting stochasticity as SD or CV makes a difference (
   <i>i.e.</i>, effect of genotype on SD or CV vs. MLT), all the other results showed that SD and CV followed the same trend. Since CV can be derived from SD and mean, no information is lost by presenting them separately. Furthermore, when the stochasticity (or noise distribution) is not apparent, it may be advisable to present results as means and SDs (and if necessary, higher moments like skewness and kurtosis) separately, so that the impact of various experimental treatments on the noise distribution can be fully appreciated.
  <span>
   <b> [g.9.2]</b>
  </span>
  </p>
 </div>
 <div id="g.9.3" last="g.9.2" next="g.9.4" class="omit">
  <p>
   <b>Methods Bacterial strains All bacteria and phage strains used in this study are listed in Table 3. The copy number of ? genome was checked by PCR following the method of Powell </b>
   <i>et al.</i> [64].
  <span>
   <b> [g.9.3]</b>
  </span>
  </p>
 </div>
 <div id="g.9.4" last="g.9.3" next="g.9.5" class="omit">
  <p>Experimental instrumentation</p>
  <p>
   <i>E. coli </i>cells
   <i> </i>lysogenic
   <i> </i>for
   <i> </i>?
   <i> </i>phage
   <i> </i>were
   <i> </i>induced
   <i> </i>and
  </p>
  <p>observed to lyse in a temperature-controlled perfusion</p>
  <p>chamber. The experimental apparatus consisted of a 250</p>
  <p>mL side-arm (on bottom) medium bottle clamped to an</p>
  <p>elevated support with tubing leading to an inline heater</p>
  <p>(SH-27B, Warner Instruments, New Haven, CT) that</p>
  <p>was controlled by a dual channel heater controller (TC-</p>
  <p>344B, Warner Instruments, New Haven, CT). The</p>
  <p>growth medium, flowing at a rate of ~1 mL/min (driven</p>
  <p>by gravity) and heated by the inline heater to the desired</p>
  <p>temperature, was introduced to a 358 ? L perfusion</p>
  <p>chamber (RC-21B, Warner Instruments, New Haven,</p>
  <p>CT) mounted on a heating platform (PM2, Warner Instruments, New Haven, CT) that was controlled by
  <span>
   <b> [g.9.4]</b>
  </span>
the same dual channel heater controller to maintain the desired temperature. The internal temperature of the perfusion chamber was independently monitored by a thermistor. Waste flowed out of the perfusion chamber, pooled in a reservoir, and was siphoned into a 2 L bottle by a vacuum source. Both the perfusion chamber and the heating platform were placed on the stage of an inverted microscope (TS100, Nikon) for observation at 400× magnification. One of the microscope’ s ocular lenses was replaced with a 10X MiniVID™ microscope camera (LW Scientific, Norcross, GA) to record individual lysis events onto a laptop computer at the rate of 1 frame per second. All data were collected in unit of seconds, though the results were presented in minutes.
  <span>
   <b> [g.9.6]</b>
  </span>
  </p>
 </div>
 <div id="g.9.7" last="g.9.6" next="g.10.0" class="omit">
  <p>Sample preparation and lysis time determination Lysogens were cultured overnight in LB or minimal salts media (see below) at 30°C on a rolling drum. Stationary phase cultures were diluted 100-fold in LB or minimal</p>
  <p>salts media, then grown to A
   <sub>550</sub>~ 0.2. 200 ? L of expo-
  </p>
  <p>nentially growing cells were immobilized on a 22 mm square glass coverslip that has been pretreated with
  <span>
   <b> [g.9.7]</b>
  </span>
0.01% tissue-culture tested poly-L-lysine (mol. wt. 150 K - 300 K, Sigma, St. Louis, MO) at room temperature for 30 min. After assembling the perfusion chamber, the device was immediately placed on the heating platform and infused with heated medium to maintain the chamber temperature at 30°C for 30 min to stabilize the cells. To induce lysis, the chamber temperature was raised to 42°C for 15 min, and then dropped to 37°C for the duration of the observation period (
   <i>i.e.</i>, until ~95% of cells are lysed). Video recording was initiated at the time when the temperature was raised to 42°C. Under these conditions, it usually takes less than 5 min for the temperature to rise from 30°C to 42°C, a transition comparable to shifting culture flasks from a 30°C to 42°C waterbath shaker. Some experiments were performed by adding KCN to the growth medium in the sidearm feeder bottle to a final concentration of 20 mM.
  </p>
  <p>Videos were subsequently analyzed using Windows Media Player™ playback. The times of individual lysis events were then noted visually and recorded manually. The lysis time was defined as the time from the initiation of the first temperature shift to when the image of the cell disappeared from view. In general, it takes about a few seconds (frames) for lysing cells to fully disappear from view (Figure 1A).
  <span>
   <b> [g.10.2]</b>
  </span>
  </p>
 </div>
 <div id="g.10.3" last="g.10.2" next="g.10.4" class="omit">
  <p>Determination of lysogen growth rate Lysogen growth rate was manipulated by using different growth medium formulations: (i) full-strength LB (10 g tryptone, 5 g yeast extract, 10 g NaCl per L dH
   <sub>2</sub>O), (ii) one-fifth-strength LB (2 g tryptone, 1 g yeast extract, 10 g NaCl per L dH
   <sub>2</sub>O), (iii) 20 mM glucose in Davis minimal salts (7 g K
   <sub>2</sub>HPO
   <sub>4</sub>, 2 g KH
   <sub>2</sub>PO
   <sub>4</sub>, 1 g (NH
   <sub>4</sub>)
   <sub>2</sub>SO
   <sub>4</sub>, 0.5 g sodium citrate• 2H
   <sub>2</sub>O, and 0.2 g MgSO
   <sub>4</sub>• 7H
   <sub>2</sub>O), and (iv) 40 mM glycerol in Davis minimal salts. We assessed the growth of the lysogen strain IN56 by culturing it overnight at 30°C in each growth media. The next day, 90 ? L of the overnight culture was used to inoculate 25 mL growth medium and the culture was placed in a 30°C waterbath shaker at 220 rpm. Culture growth was followed
  </p>
  <p>with a sipper-equipped spectrophotometer at A . 550 The growth rate was calculated as the slope of the linear regression of natural-logarithm transformed A 550 values over time.
  <span>
   <b> [g.10.3]</b>
  </span>
  </p>
 </div>
 <div id="g.10.4" last="g.10.3" next="g.10.5" class="omit">
  <p>Statistical analysis In most cases, data collection for a given strain or treatment spanned several days. Therefore, even for the same lysogen strain or experimental treatment the means and/or variances may be significantly different among data collected from different dates. We conducted a preliminary exploration of our data set using the software package JMP version 7.0.2, as
  <span>
   <b> [g.10.4]</b>
  </span>
implemented in MacOS operating system. For each lysogen strain or experimental treatment, the means and standard deviations (SDs) were extracted from the data set according to the date the data were collected and were treated as replicates. Pairwise comparisons of the means (using the Tukey-Kramer HSD test) showed that, for more than half of the cases, at least one mean was significantly different from the others. Since we were mainly interested in the variation, we subsequently converted all values into their corresponding residuals (centered by their corresponding means). We also tested the homogeneity of variance from each date replicate, using O’ Brien’ s test, Brown-Forsythe test, Levene’ s test, and Bartlett’ s test, all implemented in</p>
  <p>JMP. Not surprisingly, more than half of the cases</p>
  <p>showed that at least one replicate variance was signifi-</p>
  <p>cantly different from the others. Although we did not</p>
  <p>have an
   <i>a priori</i> expectation of lysis time distribution,
  </p>
  <p>we nonetheless tested to see if the lysis time in each</p>
  <p>replicate is normally distributed or not, using the Sha-</p>
  <p>piro-Wilk W test. Again, in many cases, the replicates</p>
  <p>do not show a normal distribution. Despite variability</p>
  <p>in our data set, none of our conclusions were funda-</p>
  <p>mentally changed. Therefore, for the presented results,</p>
  <p>the mean and standard deviation for each lysogen</p>
  <p>strain or experimental treatment were calculated based</p>
  <p>on the following criteria: (
   <i>i</i>) if the means and variances
  </p>
  <p>were the same among all blocks, then all the data</p>
  <p>would be pooled together to estimate the combined</p>
  <p>means and SDs, (
   <i>ii</i>) if the means were significantly different, but the variances were the same among all blocks, then the mean would be estimated by averaging the block means while the SDs would be estimated by pooled residuals, and (
   <i>iii</i>) if the means and variances were significantly different among all blocks, then the means and SDs would be estimated by averaging block means and SDs. For details of our data set, see additional file 1.
  <span>
   <b> [g.10.5]</b>
  </span>
  </p>
 </div>
 <p id="g.10.6" last="g.10.5" next="g.10.7" class="omit">
  <b>Additional material</b>
  <span>
   <b> [g.10.6]</b>
  </span>
 </p>
 <div id="g.10.7" last="g.10.6" next="g.10.8" class="omit">
  <div>
   <div>
    <p>Additional file 1: Sample sizes and standard deviations. More detailed data sets for both Table 1 and Table 2.</p>
   </div>
  </div>
  <span>
   <b> [g.10.7]</b>
  </span>
 </div>
 <div id="g.11.5" last="g.11.4" next="g.11.6" class="omit">
  <p>Received: 1 December 2010 Accepted: 2 August 2011 Published: 2 August 2011
  <span>
   <b> [g.11.5]</b>
  </span>
  </p>
 </div>
 <div id="g.11.7" last="g.11.6" next="g.12.0" class="omit">
  <p>24. Blake WJ, Balazsi G, Kohanski MA, Isaacs FJ, Murphy KF, Kuang Y, Cantor CR,</p>
  <p>Walt DR, Collins JJ: Phenotypic consequences of promoter-mediated</p>
  <p>transcriptional noise.
   <i>Mol Cell</i> 2006, 24:853-865. 25. Bishop AL, Rab FA, Sumner ER, Avery SV: Phenotypic heterogeneity can
  </p>
  <p>enhance rare-cell survival in ‘ stress-sensitive’ yeast populations.
   <i>Mol</i>
  </p>
  <p>
   <i>Microbiol </i>2007,
   <i> </i>63:507-520. 26. Wang IN, Smith DL, Young R: HOLINS: The Protein Clocks of
  </p>
  <p>Bacteriophage Infections.
   <i>Annu Rev Microbiol</i> 2000, 54:799-825. 27. Young R, Wang IN, Roof WD: Phages will out: strategies of host cell lysis.
  </p>
  <p>
   <i>Trends Microbiol </i>2000,
   <i> </i>8:120-128. 28. Wang IN, Deaton J, Young R: Sizing the holin lesion with an endolysin-? -
  </p>
  <p>galactosidase fusion.
   <i>J Bacteriol</i> 2003, 185:779-787. 29. Savva CG, Dewey JS, Deaton J, White RL, Struck DK, Holzenburg A,
  </p>
  <p>Young R: The holin of bacteriophage lambda forms rings with large</p>
  <p>diameter.
   <i>Mol Microbiol</i> 2008, 69:784-793. 30. Park T, Struck DK, Dankenbring CA, Young R: The pinholin of lambdoid
  </p>
  <p>phage 21: control of lysis by membrane depolarization.
   <i>J Bacteriol</i> 2007,
  </p>
  <p>189:9135-9139. 31. Xu M, Arulandu A, Struck DK, Swanson S, Sacchettini JC, Young R: Disulfide</p>
  <p>isomerization after membrane release of its SAR domain activates P1</p>
  <p>lysozyme.
   <i>Science</i> 2005, 307:113-117. 32. Xu M, Struck DK, Deaton J, Wang IN, Young R: A signal-arrest-release
  </p>
  <p>sequence mediates export and control of the phage P1 endolysin.
   <i>Proc</i>
  </p>
  <p>
   <i>Natl Acad Sci USA </i>2004,
   <i> </i>101:6415-6420. 33. Zhang N, Young R: Complementation and characterization of the nested
  </p>
  <p>
   <i>Rz </i>and
   <i> Rz1 </i>reading
   <i> </i>frames
   <i> </i>in
   <i> </i>the
   <i> </i>genome
   <i> </i>of
   <i> </i>bacteriophage
   <i> </i>lambda.
   <i> Mol</i>
  </p>
  <p>
   <i>Gen Genet </i>1999,
   <i> </i>262:659-667. 34. Berry J, Summer EJ, Struck DK, Young R: The final step in the phage
  </p>
  <p>infection cycle: the Rz and Rz1 lysis proteins link the inner and outer</p>
  <p>membranes.
   <i>Mol Microbiol</i> 2008, 70:341-351. 35. Young R, Way J, Way S, Yin J, Syvanen M: Transposition mutagenesis of
  </p>
  <p>bacteriophage lambda: a new gene affecting cell lysis.
   <i>J Mol Biol</i> 1979,
  </p>
  <p>132:307-322. 36. Friedman DI, Gottesman M: Lytic mode of lambda development. In</p>
  <p>
   <i>Lambda II. </i>Edited
   <i> </i>by:
   <i> </i>Hendrix
   <i> </i>RW,Roberts
   <i> </i>JW,Stahl
   <i> </i>FW,Weisberg
   <i> </i>RA.
   <i> </i>Cold
  </p>
  <p>Spring Harbor, New York: Cold Spring Harbor Laboratory; 1983:21-51. 37. Gründling A, Bläsi U, Young R: Genetic and biochemical analysis of dimer</p>
  <p>and oligomer interactions of the lambda S holin.
   <i>J Bacteriol</i> 2000,
  </p>
  <p>182:6082-6090. 38. Dewey JS, Savva CG, White RL, Vitha S, Holzenburg A, Young R: Micron-</p>
  <p>scale holes terminate the phage infection cycle.
   <i>Proc Natl Acad Sci USA</i>
  </p>
  <p>2010, 107:2219-2223. 39. Ryan GL, Rutenberg AD: Clocking out: modeling phage-induced lysis of</p>
  <p>
   <i>Escherichia coli</i>.
   <i> J Bacteriol </i>2007,
   <i> </i>189:4749-4755. 40. White R, Chiba S, Pang T, Dewey JS, Savva CG, Holzenburg A, Pogliano K,
  </p>
  <p>Young R: Holin triggering in real time.
   <i>Proc Natl Acad Sci USA</i> 2010,
  </p>
  <p>108:798-803. 41. Ellis EL, Delbrück M: The growth of bacteriophage.
   <i>J Gen Physiol</i> 1939,
  </p>
  <p>22:365-384. 42. Delbrück M: The growth of bacteriophage and lysis of the host.
   <i>J Gen</i>
  </p>
  <p>
   <i>Physiol </i>1940,
   <i> </i>23:643-660. 43. Doermann AH: The intracellular growth of bacteriophages. I. Liberation
  </p>
  <p>of intracellular bacteriophage T4 by premature lysis with another phage</p>
  <p>or with cyanide.
   <i>J Gen Physiol</i> 1952, 35:645-656. 44. Young R: Bacteriophage lysis: mechanism and regulation.
   <i>Microbiol Rev</i>
  </p>
  <p>1992, 56:430-481. 45. Gründling A, Manson MD, Young R: Holins kill without warning.
   <i>Proc Natl</i>
  </p>
  <p>
   <i>Acad Sci USA </i>2001,
   <i> </i>98:9348-9352. 46. Wang IN: Lysis timing and bacteriophage fitness.
   <i>Genetics</i> 2006, 172:17-26. 47. Raab R, Neal G, Garrett J, Grimaila R, Fusselman R, Young R: Mutational
  </p>
  <p>analysis of bacteriophage lambda lysis gene S.
   <i>J Bacteriol</i> 1986,
  </p>
  <p>167:1035-1042. 48. Swain PS, Elowitz MB, Siggia ED: Intrinsic and extrinsic contributions to</p>
  <p>stochasticity in gene expression.
   <i>Proc Natl Acad Sci USA</i> 2002,
  </p>
  <p>99:12795-12800. 49. Raj A, Peskin CS, Tranchina D, Vargas DY, Tyagi S: Stochastic mRNA</p>
  <p>synthesis in mammalian cells.
   <i>PLoS Biol</i> 2006, 4:1707-1719. 50. Shao Y, Wang IN: Effect of late promoter activity on bacteriophage ?
  </p>
  <p>fitness.
   <i>Genetics</i> 2009, 181:1467-1475.
  </p>
  <span>
   <b> [g.11.7]</b>
  </span>
 </div>
 <div id="g.12.2" last="g.12.1" next="g.12.3" class="omit">
  <p>51. Gillespie DT: Exact stochastic simulation of coupled chemical reactions.
   <i>J</i>
  </p>
  <p>
   <i>Phys Chem </i>1977,
   <i> </i>81:2340-2361. 52. McAdams HH, Arkin A: Stochastic mechanisms in gene expression.
   <i>Proc</i>
  </p>
  <p>
   <i>Natl Acad Sci USA </i>1997,
   <i> </i>94:814-819. 53. Bremer H, Dennis PP: Modulation of chemical composition and other
  </p>
  <p>parameters of the cell by growth rate. In
   <i>Escherichia coli and Salmonella</i>
  </p>
  <p>
   <i>typhimurium Cellular and Molecular Biology. Volume 2. </i>Edited
   <i> </i>by:
   <i> </i>Ingraham
  </p>
  <p>JL,Low KB,Magasanik B,Schaechter M,Umbarger HE. Washington, D.C.:</p>
  <p>American Society for Microbiology; 1987:1527-1542. 54. Hadas H, Einav M, Fishov I, Zaritsky A: Bacteriophage T4 development</p>
  <p>depends on the physiology of its host
   <i>Escherichia coli</i>.
   <i>Microbiology</i> 1997,
  </p>
  <p>143:179-185. 55. Bertani G: Lysogeny at mid-twentieth century: P1, P2, and other</p>
  <p>experimental systems.
   <i>J Bacteriol</i> 2004, 186:595-600. 56. Sokal RR, Rohlf FJ: Biometry. New York, New York: W. H. Freeman and
  </p>
  <p>Company;, 3 1995. 57. Abedon ST: Selection for bacteriophage latent period length by bacterial</p>
  <p>density: A theoretical examination.
   <i>Microb Ecol</i> 1989, 18:79-88. 58. Abedon ST, Herschler TD, Stopar D: Bacteriophage latent-period evolution
  </p>
  <p>as a response to resource availability.
   <i>Appl Environ Microbiol</i> 2001,
  </p>
  <p>67:4233-4241. 59. Heineman RH, Bull JJ: Testing optimality with experimental evolution:</p>
  <p>lysis time in a bacteriophage.
   <i>Evolution</i> 2007, 61:1695-1709. 60. Shao Y, Wang IN: Bacteriophage adsorption rate and optimal lysis time.
  </p>
  <p>
   <i>Genetics </i>2008,
   <i> </i>180:471-482. 61. Wang IN, Dykhuizen DE, Slobodkin LB: The evolution of phage lysis
  </p>
  <p>timing.
   <i>Evol Ecol</i> 1996, 10:545-558. 62. Gillespie JH: Nautural selection for within-generation variance in
  </p>
  <p>offspring number.
   <i>Genetics</i> 1974, 76:601-606. 63. Gillespie JH: Natural selection for variances in offspring numbers: a new
  </p>
  <p>evolutionary principle.
   <i>Am Nat</i> 1977, 111:1010-1014. 64. Powell BS, Rivas MP, Court DL, Nakamura Y, Turnbough CL Jr: Rapid
  </p>
  <p>confirmation of single copy lambda prophage integration by PCR.</p>
  <p>
   <i>Nucleic Acids Res </i>1994,
   <i> </i>22:5765-5766.
  <span>
   <b> [g.12.2]</b>
  </span>
  </p>
 </div>
 <div id="g.12.3" last="g.12.2" next="g.12.4" class="omit">
  <div>
    <p>doi:10.1186/1471-2180-11-174 Cite this article as: Dennehy and Wang: Factors influencing lysis time stochasticity in bacteriophage ? .
     <i>BMC Microbiology</i> 2011 11:174.
    </p>
  <span>
   <b> [g.12.3]</b>
  </span>
  </div>
 </div>
</div>
