<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html id="nojs" xmlns="http://www.w3.org/1999/xhtml" xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns:og="http://ogp.me/ns#" xml:lang="en-GB" lang="en-GB">

<head>

 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
        <title>BMC Microbiology | Full text | Factors influencing lysis time stochasticity in bacteriophage lambda</title>
<link rel="shortcut icon" type="image/x-icon" href="/images/icon/1010/favicon.ico" />
<meta http-equiv="X-UA-Compatible" content="IE=7" />


                
    
		
		
		
<!-- <?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/">
<cc:Work rdf:about="http://www.biomedcentral.com/1471-2180/11/174">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.biomedcentral.com/1471-2180/11/174">
<title>Factors influencing lysis time stochasticity in bacteriophage λ</title>
<dc:title>Factors influencing lysis time stochasticity in bacteriophage λ</dc:title>
<dc:creator>Dennehy, John J</dc:creator>
<dc:creator>Wang, Ing-Nang</dc:creator>
<dc:identifier>info:doi/10.1186/1471-2180-11-174</dc:identifier>
<dc:identifier>info:pmid/21810267</dc:identifier>
<dc:source>BMC Microbiology 2011, 11:174</dc:source>
<dc:date>2011-08-02</dc:date>

<prism:publicationName>BMC Microbiology</prism:publicationName>
<prism:publicationDate>2011-08-02</prism:publicationDate>
<prism:volume>11</prism:volume>
<prism:number>1</prism:number>
<prism:section>Research article</prism:section>
<prism:startingPage>174</prism:startingPage>
<prism:copyright>2011 Dennehy and Wang; licensee BioMed Central Ltd.</prism:copyright>
</item>
</rdf:RDF>
 -->
 
 
<meta name="citation_journal_title" content="BMC Microbiology" />
<meta name="citation_publisher" content="BioMed Central Ltd" />


<meta name="citation_author" content="John J Dennehy" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University at Albany, 1400 Washington Avenue, Albany, NY 12222, USA" />
<meta name="citation_author_institution" content="Biology Department, Queens College, and the Graduate Center of the City University of New York, Flushing, NY 11367, USA" />
<meta name="citation_author" content="Ing-Nang Wang" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University at Albany, 1400 Washington Avenue, Albany, NY 12222, USA" />


<meta name="citation_issn" content="1471-2180" />
<meta name="citation_title" content="Factors influencing lysis time stochasticity in bacteriophage λ" />
<meta name="citation_volume" content="11" />
<meta name="citation_issue" content="1" />
<meta name="citation_date" content="2011-08-02" />
<meta name="citation_firstpage" content="174" />
<meta name="citation_doi" content="10.1186/1471-2180-11-174" />
<meta name="citation_pdf_url" content="http://www.biomedcentral.com/content/pdf/1471-2180-11-174.pdf" />
<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/content/pdf/1471-2180-11-174.pdf" />
<meta name="title" content="Factors influencing lysis time stochasticity in bacteriophage λ" />
<meta name="description" content="Despite identical genotypes and seemingly uniform environments, stochastic gene expression and other dynamic intracellular processes can produce considerable phenotypic diversity within clonal microbes. One trait that provides a good model to explore the molecular basis of stochastic variation is the timing of host lysis by bacteriophage (phage)." />

<meta name="dc.title" content="Factors influencing lysis time stochasticity in bacteriophage λ" />
<meta name="dc.creator" content="Dennehy, John J" />
<meta name="dc.creator" content="Wang, Ing-Nang" />
<meta name="dc.description" content="Despite identical genotypes and seemingly uniform environments, stochastic gene expression and other dynamic intracellular processes can produce considerable phenotypic diversity within clonal microbes. One trait that provides a good model to explore the molecular basis of stochastic variation is the timing of host lysis by bacteriophage (phage)." />
<meta name="dc.source" content="BMC Microbiology 2011 11:174" />
<meta name="dc.format" content="text/html" />
<meta name="dc.publisher" content="BioMed Central Ltd" />
<meta name="dc.date" content="2011-08-02" />
<meta name="dc.type" content="Research article" />
<meta name="dc.identifier" content="10.1186/1471-2180-11-174" />
<meta name="dc.language" content="en" />
<meta name="dc.copyright" content="2011 Dennehy and Wang; licensee BioMed Central Ltd." />
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/" />
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com" />

<meta name="prism.issn" content="1471-2180" />
<meta name="prism.publicationName" content="BMC Microbiology" />
<meta name="prism.publicationDate" content="2011-08-02" />
<meta name="prism.volume" content="11" />
<meta name="prism.number" content="1" />
<meta name="prism.section" content="Research article" />
<meta name="prism.startingPage" content="174" />
<meta name="prism.copyright" content="2011 Dennehy and Wang; licensee BioMed Central Ltd." />
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com" />


        <meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2180/11/174/abstract" />
            <meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2180/11/174" />
            <link rel='alternate' type='text/xml' title='XML version' href='http://www.biomedcentral.com/content/download/xml/1471-2180-11-174.xml' />
            <link rel="image_src" href="http://www.biomedcentral.com/content/figures/1471-2180-11-174-toc.gif" />
    
    <link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2180-11-174-s1.doc" title="Additional file 1" />

    
    
                    
    
<script type="text/javascript">
// must run before css
document.documentElement.id = "js";
</script>






	                <link rel="stylesheet" type="text/css" href="/css/main-0.css" media="screen, print"/>
    
                <link rel="stylesheet" type="text/css" href="/css/plugins-0.css" media="screen, print"/>
    



    
			                    <link rel="stylesheet" type="text/css" href="/css/articles-0.css" media="screen, print"/>
    			


    

    <link rel="stylesheet" type="text/css" href="/bmcmicrobiol/css/themes/1010.css?1330428864857" media="screen, print"/>


                <link rel="stylesheet" type="text/css" href="/css/media/print-0.css" media="print"/>
    


<!--[if IE]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie-0.css"/>
    <![endif]-->

<!--[if IE 7]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie7-0.css"/>
    <![endif]-->

			
<script type="text/javascript">
    //configuration
    OAS_url = 'http://oas.biomedcentral.com/RealMedia/ads/';

	// control of advert's position on page [AMC]
    
    	
	
    OAS_sitepage = "biomedcentral.com/bmcmicrobiol/article/10.1186/1471/2180/11/174";
    	
		OAS_listpos = 'Top,x96,Bottom';
	
	
			
		OAS_query = '';
		
	

    OAS_target = '_top';
    //end of configuration
    OAS_version = 10;
    OAS_rn = '001234567890'; OAS_rns = '1234567890';
    OAS_rn = new String (Math.random()); OAS_rns = OAS_rn.substring (2, 11);
    function OAS_NORMAL(pos) {
		document.write('<a href="' + OAS_url + 'click_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" target="' + OAS_target + '">');
		document.write('<img src="' + OAS_url + 'adstream_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" border="0"/></a>');
    }
    
	function refererTerms(referer) {

	if(referer.match("q=") != null) {
		referer = referer.split("q=");
		referer = referer[1].split("&");
		referer = referer[0];
	} else {
		referer = referer;
	}

		if(referer.match("%20") != null && referer.match("%20").length >= 1) {
			referer =  referer.replace(/%20/gi,"+");
			return referer;
		} else { 
			return referer; 
		}
	}
 </script>

 
 <div id="oas-campaign" style="display:none">biomedcentral.com/bmcmicrobiol/article/10.1186/1471/2180/11/174</div>
 <div id="oas-positions" style="display:none">Bottom,Top</div> 


 <script type="text/javascript">
	// Set the version of JavaScript to 10 for a Browser 'Mozilla/3' on webTV 
    OAS_version = 11;
    if ((navigator.userAgent.indexOf('Mozilla/3') != -1) || (navigator.userAgent.indexOf('Mozilla/4.0 WebTV') != -1)) {
     OAS_version = 10; }
    if (OAS_version >= 11) {
    document.write('<scr' + 'ipt type="text/javascript" src="' + OAS_url + 'adstream_mjx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '?' + OAS_query + '"><\/script>'); }
</script>
	
<script type="text/javascript">
    document.write('');
	
    function OAS_AD(pos) {
        if (OAS_version >= 11)
         OAS_RICH(pos);
        else
         OAS_NORMAL(pos);
    }
 </script>
	




<script type="text/javascript">
    jsonLoaders = function(){
        var _loaders = [];
        return {
            register: function(config){
                config.prefix = config.prefix || '/webapi/1.0/blogs/';
                config.feedLength = config.feedLength || 3;

                _loaders.push(config);
            },
            list: function(){return _loaders}
        };
    }();

    function totext() {
	    document.write('<a href="javascript:returnToText();">Return to text</a>');
    }

    function retrieve_doi(){
        var metas = document.getElementsByTagName('meta');
        var i;
        for (i = 0; i < metas.length; i++)
            if (metas[i].getAttribute('name') == "citation_doi")
                break;
        var citation_doi = metas[i].getAttribute('content');
        return citation_doi;
    }

</script>




        <link title="Latest articles" rel="alternate" href="http://www.biomedcentral.com/bmcmicrobiol/latest/rss" type="application/rss+xml" exposeToHeader="true"/>
    <link title="Most viewed" rel="alternate" href="http://www.biomedcentral.com/bmcmicrobiol/mostviewed/rss/" type="application/rss+xml" exposeToHeader="true"/>
    <link title="Latest comments" rel="alternate" href="http://www.biomedcentral.com/bmcmicrobiol/latestcomments/rss" type="application/rss+xml" exposeToHeader="true"/>
    


	<script type="text/javascript">
	var _gaq = _gaq || [];
	_gaq.push(['_setAccount', 'UA-72326-1']);
	_gaq.push(['_gat._anonymizeIp']);
	_gaq.push(['_setLocalRemoteServerMode']);

		_gaq.push(['_trackPageview']); 

	(function() {
		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	})();
	</script>

	
        </head>


<body class=" theme    ">





 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<script type="text/javascript">

function createCookie(name,value,days,path) {
	if (days) {
		var date = new Date();
		date.setTime(date.getTime()+(days*24*60*60*1000));
		var expires = "; expires="+date.toGMTString();
	}
	else{ var expires = "";}
	if (path!=null)
	{
	    document.cookie = name+"="+value+expires+"; path=/";
	}
	else
	{
	    document.cookie = name+"="+value+expires+";";
	}
}
function createCookie1(name,value,days) {
	if (days) {
		var date = new Date();
		date.setTime(date.getTime()+(days*24*60*60*1000));
		var expires = "; expires="+date.toGMTString();
	}
	else{ var expires = "";}
		    document.cookie = name+"="+value+expires;
	}

function eraseCookie(name) {
    // path describes the location of the cookie
    var path= "/";
	createCookie("database_statistics","",-1,path);
	// null value as a parameter tells that cookie existing in current application context like cookie in  http://biomedcentral.com.local/about/tandc
	createCookie("database_statistics_request_id","",-1,null);
	hidediv();
}
function hidediv() 
{ 
	if (document.getElementById) 
	{
	    document.getElementById('dbstats_main').style.visibility = 'hidden';
	}
}

function getCookieExists(c_name)
{
    var i,x,y,ARRcookies=document.cookie.split(";");
    for (i=0;i<ARRcookies.length;i++)
    {
          x=ARRcookies[i].substr(0,ARRcookies[i].indexOf("="));
          y=ARRcookies[i].substr(ARRcookies[i].indexOf("=")+1);
          x=x.replace(/^\s+|\s+$/g,"");
          if (x==c_name)
            {
                return unescape(y);
            }
     }
}

var request_Id;
var search_cookie = "database_statistics_request_id" + "="
var more_dbstats_link;
//readingDBStatscookie() will be called from JQuery function jQuery(document).ready(function() in behaviour.js which is called on "onload()" event
// in order to read the data from the lates cookie

function readingDBStatsCookie()
{
 if ( getCookieExists("database_statistics")!=null)
   {	
	if ( getCookieExists("database_statistics_request_id") != null)
   {
        if ( getCookieExists("database_statistics_request_id") != null)
        {
            if (document.cookie.length > 0)
            {
                // Search for a cookie.
                offset = document.cookie.indexOf(search_cookie)
                if (offset != -1)
                {
                        offset += search_cookie.length
                        // set index of beginning of value
                        end = document.cookie.indexOf(";",offset)
                        if (end == -1)
                        {
                            end = document.cookie.length
                        }
                       request_Id=document.cookie.substring(offset, end);
                       more_dbstats_link = "http://www.biomedcentral.com:80/newjournal/dbstats?content_type=html&request_id="+request_Id;
			  
			  createCookie("database_statistics_request_id","",-1);
				
                       var link = document.createElement('a');
                       link.setAttribute('href', more_dbstats_link);
                       link.innerHTML="<b>More DBStats</b>";
                       link.title="More dbstats";
                       //var text = "<a href=\""+more_dbstats_link+"\" >More DB Stats</a>";
                    document.getElementById('more_dbstats_link').appendChild(link);
                }
            }
        }
	}
 }
}
  if (getCookieExists("database_statistics")!=null)
     {
           document.write( "<div id =\"dbstats_main\" style=\"background-color:white\">"   );
             document.write("<div id =\"dbstats\" style=\"background-color:lightgray;margin-left:300px;margin-right:300px;margin-top:-10px\">");
             document.write("<br>");
                    document.write("<b>DB Stats</b>");
                    document.write("- consistent gets:");
                    document.write("<span id = \"consistent_gets\">NA</span> ");
                    document.write(", physical reads:");
                    document.write("<span id = \"physical_reads\">NA</span>");
                    document.write(", bytes sent via SQL*Net to client: ");
                    document.write("<span id = \"SQL_NET\">NA</span>");
                    document.write("&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp");
                    document.write("<span id=\"more_dbstats_link\">");
                    document.write("</span>	");
                    document.write("<span id=\"close_dbstats_link\">");
                    document.write("[<a href='javascript:eraseCookie()' >x</a>]");
                    document.write("</span>	");
             document.write("</div>");
           document.write("</div> ");
     }
 </script>
<div id="branding" role="banner">
            			
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Top' begin -->
		<script type="text/javascript">
		OAS_AD('Top');

		</script>
		<!-- OAS AD 'Top' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad wide noscript">  
    <dt class="hide"></dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcmicrobiol/article/10.1186/1471/2180/11/174/19356019873@Top?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcmicrobiol/article/10.1186/1471/2180/11/174/19843600288@Top?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
			        <div class="sup-panel-outer">
        <div class="sup-panel-inner">
            <ul id="login" class="login">
            <li id="loginLink"><a href="/bmcmicrobiol/logon">Log on</a></li>
    </ul>            <ul class="nav-sup">
    
        <li id="BMC" class="current">
            <a href="http://www.biomedcentral.com/"><span class="img-title">&nbsp;</span>
            <span class="text-title">BioMed Central</span></a>
        </li>
		
        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>
		
								<li>
				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>
			</li>
					    	
</ul>
        </div>
    </div>
    <div class="branding-inner">
					<div class="logo">
			<a href="/bmcmicrobiol/"><img alt="BMC Microbiology" title="BMC Microbiology Logo" src="/sites/1010/images/logo.gif" /></a>			
			</div>		
				
                    

        <div id="impact-factor" class="official">
    <img src="/images/branding/official.gif" alt="official impact factor" title="3.04"/>
    <span id="impact-factor-value">3.04</span>
</div>
    
                            

<div class="module search gray small  " role="search">
    <div class="module-inner padded">
        <form method="get" class="searchForm"
              action=" /quickSearchRedirectURL "
              onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">
            <fieldset class="search">
			                <span>
                    Search <select name="url">
                                <option value="http://www.biomedcentral.com/bmcmicrobiol/search/results" selected>BMC Microbiology</option>
                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>
                           </select> for
                </span>
                            <input id="searchTerms" name="terms" type="text" alt="Search terms" class="text " />
                 <button class="w37" value="Go" type="submit">Go</button>
            </fieldset>
        </form>
    </div>
	<a class="advanced-search" href="/bmcmicrobiol/search">Advanced search</a>
</div> 

                        <div id="mobile-nav-list" >

	<ul class="primary-nav" role="navigation">
		<li     ><a href="/bmcmicrobiol/"><span id="home-tab">Home</span></a></li>
														<li             class="current"
    ><a href="/bmcmicrobiol/content"><span id="articles-tab">Articles</span></a></li>
										
	

	
            <li     ><a href="/bmcmicrobiol/authors/instructions"><span id="authors-tab">Authors</span></a></li>
        <li     ><a href="/bmcmicrobiol/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>
    		<li     ><a href="/bmcmicrobiol/about"><span id='about-tab'>About this journal</span></a></li>
	<li     ><a href="/bmcmicrobiol/my"><span id="my-tab">My BMC Microbiology</span></a></li>
		
    		</ul>
</div>
            </div>
</div>
<hr class="hide" />
<div class="container">
	<div id="content"  class="overflow-visible"  role="main">
			    
 	

<!--div class="branding-mobile">
		<h1 class="logo">
	    <a href="/bmcmicrobiol/">
		    <img alt="BMC Microbiology" title="BMC Microbiology Logo" src="/sites/1010/images/logo.gif" />
		</a>
		</h1>
	</div><br/-->



			
	



<div class="outline-wrapper block mobile-hidden" id="left-article-box">
<ul id="box-outline" class="box">
<ul id="outline" class="outline">
   <li>
      <a href="#">Top</a>
   </li>
   <li>
      <a href="#abs">Abstract</a>
   </li>
   <li>
      <a href="#sec1">Background</a>
   </li>
   <li>
      <a href="#sec2">Results</a>
   </li>
   <li>
      <a href="#sec3">Discussion</a>
   </li>
   <li>
      <a href="#sec4">Conclusions</a>
   </li>
   <li>
      <a href="#sec5">Appendix A</a>
   </li>
   <li>
      <a href="#sec6">Appendix B</a>
   </li>
   <li>
      <a href="#sec7">Methods</a>
   </li>
   <li>
      <a href="#sec8">Competing interests</a>
   </li>
   <li>
      <a href="#sec9">Authors' contributions</a>
   </li>
   <li>
      <a href="#ack">Acknowledgements</a>
   </li>
   <li class="lowest">
      <a href="#refs">References</a>
   </li>
</ul>
</ul>
<dl id="box-outline1">
</dl>
</div>

<div class="mobile-hidden" id="right-panel">
  
<div id="mobile-sidebar">
	<div id="mobile-sidebar-tab">
		<span>
			<a href="#" id="mobile-sidebar-tab-link"><img src="/images/sidebar-hidden.png"><img style="display:none;" src="/images/sidebar-shown.png"></a>
		</span>
	</div>	<div id="mobile-sidebar-list">

<div id="article-navigation-bar">
    <div class="issue-information block">
        <h5><a href="http://www.biomedcentral.com/bmcmicrobiol"><strong>BMC Microbiology</strong></a></h5>
        <ul class="square normal">
            <li><a href="http://www.biomedcentral.com/bmcmicrobiol/content/11">Volume 11</a></li>
				
                    </ul>
    </div>

    <div class="article-information block" id="article-info">
        


<div id="viewing-options-links">
	<h5>Viewing options</h5>
	<ul class="square normal">
        
            <li>
                             <a href="/1471-2180/11/174/abstract">Abstract</a>
                        </li>
        				<li>
							<strong>Full text</strong>
					</li>
				
		
		
		<li class="pdfFileSize">			<a onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/1471-2180/11/174', 1, true]);" href="/content/pdf/1471-2180-11-174.pdf">PDF</a>
			<span>(1.2MB)</span>		</li>

		<li>
                    </li>
		
				<li>
						<a href="/1471-2180/11/174/additional">Additional files</a>
					</li>
			</ul>
</div>
		
<div id="associated-material-links">
    <h5>Associated material</h5>
    <ul class="square normal">
                        <li>
            <a href="/pubmed/21810267">PubMed record</a>
        </li>
                            <li><a href="/1471-2180/11/174/about">About this article</a></li>
                <li>
                        <a href="/1471-2180/11/174/comments">Readers' comments </a>
                    </li>
        <li>
                    </li>
    </ul>
</div>
		




<div id="related-literature-links">
    <h5>Related literature</h5>
    <ul class="normal">
                    <li> <a href="http://www.biomedcentral.com/1471-2180/11/174/about#citations"><h6>Cited by</h6></a></li>
                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=Factors+influencing+lysis+time+stochasticity+in+bacteriophage+lambda&amp;btnG=Search">Google blog search</a></li>
        <h6>Other articles by authors</h6>
        <li>
            <a class="collapser"><i>&nbsp;</i>on Google Scholar</a>
<ul id="authg" class="hidebeforeload">                         <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22JJ+Dennehy%22">Dennehy JJ</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22IN+Wang%22">Wang IN</a>
            </li>
            </ul>          </li>
                <li>
            		<a class="collapser"><i>&nbsp;</i>on PubMed</a>
	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Dennehy_JJ%20[Author]">Dennehy JJ</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Wang_IN%20[Author]">Wang IN</a>
				</li>
						</ul > 	        </li>
    </ul>     <h6>Related articles/pages</h6>
    <ul class="square normal">
        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2180/11/174">on Google</a></li>
        <li><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2180/11/174">on Google Scholar</a></li>
                    <li><a href="/pubmed/related/21810267">on PubMed</a></li>
            </ul>

</div>		
        
    
    
        

    	





<h5>Tools</h5>
	<ul class="square normal">
									<li>
						<a href="/1471-2180/11/174/citation">Download references</a>
					</li>
				<li><a href="/content/download/xml/1471-2180-11-174.xml">Download XML</a></li>
				<li>
						<a href="/1471-2180/11/174/email">Email to a friend</a>
					</li>
		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186/1471-2180-11-174&volume=11&issue=1&title=Factors+influencing+lysis+time+stochasticity+in+bacteriophage+%26%23955%3B&author_name=John J Dennehy&start_page=1&end_page=11">Order reprints</a></li>
		<li>
						<a href="/1471-2180/11/174/postcomment">Post a comment</a>
					</li>
	</ul>
		



<h5>Share this article</h5>
<ul id="social-networking-links">
	<li>
	<div class="fb-like" data-href="http://www.biomedcentral.com/1471-2180/11/174" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"></div>
	</li>
	
	<li>
		<a href="http://twitter.com/share" data-url="http://www.biomedcentral.com/1471-2180/11/174" data-text="#bmcmicrobiology
Factors influencing lysis time stochasticity in bacteriophage lambda" class="twitter-share-button">Tweet</a>
	</li>
	<li>
			<g:plusone size="medium" width="100"></g:plusone>
	</li>
	<li>
		<a class="sharethis_title btn" href="#"><span class="icon">&nbsp;</span>More options...</a>
	<ul class="iconlist hidebeforeload" id="sharethis">
			<li>
				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2180/11/174"><span class="share-icons citeulike"></span>Citeulike</a>
			</li>
	<li>
				<a onclick="largepopup(this.href,'linkedin',800,600);return false" href="http://www.linkedin.com/shareArticle?mini=true&url=%23bmcmicrobiology%0D%0AFactors%20influencing%20lysis%20time%20stochasticity%20in%20bacteriophage%20lambda%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2180%2F11%2F174&title=Factors influencing lysis time stochasticity in bacteriophage lambda"><span class="share-icons linkedin"></span>LinkedIn</a>  
			</li>
			<li>
				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2180/11/174&title=Factors influencing lysis time stochasticity in bacteriophage lambda"><span class="share-icons delicious"></span>Del.icio.us</a>
	</li>
			<li>
				<a href="/1471-2180/11/174/email"><span class="share-icons email"></span>Email</a>
	</li>
			<li>
				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2180/11/174"><span class="share-icons facebook"></span>Facebook</a>
	</li>
	<li class="gp">
			<div class="googlehider mobile-hide"> 
				<div class="g-plusone" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus mobile-hide"></span>
				<div class="googlehider2 mobile-hide"> 
				<div class="g-plusone mobile-hide" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus"></span>
				<span class="gp-link"><a href="#">Google+</a></span>
	</li>
			<li>
				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2180/11/174"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
			<li>
				<a onclick="largepopup(this.href,'twitter',800,600);return false" href="http://twitter.com/?status=%23bmcmicrobiology%0D%0AFactors%20influencing%20lysis%20time%20stochasticity%20in%20bacteriophage%20lambda%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2180%2F11%2F174"><span class="share-icons twitter"></span>Twitter</a>  
			</li>
			
		</ul>
</ul>

    </div>

    	
	
    		









		
    
        
        
    
        
    
        
        
    
		
	
	
</div>
		<div class="mobile-sidebar-gradient top"></div>
		<div class="mobile-sidebar-gradient bottom"></div>
	</div>

    

</div>








</div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
    <div class="padded-inner">                


    
 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="topmatter">
	
	   








				<a href="/about/access"><img alt="Open Access" src="/images/articles/openaccess-large.png" class="access mr15"/></a>
			<a href="http://www.biomedcentral.com/about/mostviewed/"><img alt="Highly Accessed" src="/images/articles/highlyaccessed-large.png" class="access mr15"/></a>
				<span class="articletype">Research article</span>
<h1>Factors influencing lysis time stochasticity in bacteriophage &#955;</h1>

        
			<div class="multipleins">
		
	<p class="authors">
        																																																																									        <strong>John J Dennehy</strong><sup><a href="/1471-2180/11/174/#ins1">1</a>,</sup><sup><a href="/1471-2180/11/174/#ins2">2</a></sup><sup>&#42;</sup> and <strong>Ing-Nang  Wang</strong><sup><a href="/1471-2180/11/174/#ins1">1</a></sup>	</p>

			<div id="affiliations">
		<div class="module gray  inner">
<div class="module-inner padded-inner">
		<ul>
			<li>                        
                                        		<p class="authors" >
							<span>*</span>
									 Corresponding author:										            John J Dennehy <a href='mailto:john.dennehy@qc.cuny.edu'>john.dennehy@qc.cuny.edu</a>    					</p>
      
</li>
			
                                                                        		</ul>

		
		
            <section> 
						<div class="collapsible-content">
							 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow"></i>Author Affiliations</a></p>				<div id="ins_container" style="display: block;">
										    						    <p >
						        <sup id="ins1">1</sup>                                Department of Biological Sciences, University at Albany, 1400 Washington Avenue, Albany, NY 12222, USA
                            </p>
																    						    <p >
						        <sup id="ins2">2</sup>                                Biology Department, Queens College, and the Graduate Center of the City University of New York, Flushing, NY 11367, USA
                            </p>
																
				</div>
										<p id="authoremails">            For all author emails, please <a href="/bmcmicrobiol/logon">log on</a>.
    </p>
						</div>
			</section>
				</div>
</div>
		</div>
	  </div>

            <section class="cit">
                        <div class="collapsible-content">
        <p><em>BMC Microbiology</em> 2011, <strong>11</strong>:174&nbsp;<span class="pseudotab">doi:10.1186/1471-2180-11-174</span></p>
    	
    
    
        		<br/>
		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2180/11/174">http://www.biomedcentral.com/1471-2180/11/174</a></p>
		<br/>
                    <table cellspacing="0" cellpadding="0">
        <tbody>
        <tr><td>Received:</td><td>1 December 2010</td></tr>                <tr><td>Accepted:</td><td>2 August 2011</td></tr>        <tr><td>Published:</td><td>2 August 2011</td></tr>        </tbody>
        </table>
    		<!--<br/>-->
		<div  style="line-height:140%">
        <p>
            &copy; 2011 Dennehy and Wang; licensee BioMed Central Ltd. <br />
        </p>
					<p>
				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href='http://creativecommons.org/licenses/by/2.0'>http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
			</p>
		    	</div>
	</section>

</div>
    	<div id="article-body">		
    <section><a name="abs"></a><h3>Abstract</h3>
   <div class="collapsible-content">
      <h4>Background</h4>
      <p style="line-height:160%">Despite identical genotypes and seemingly uniform environments, stochastic gene expression
         and other dynamic intracellular processes can produce considerable phenotypic diversity
         within clonal microbes. One trait that provides a good model to explore the molecular
         basis of stochastic variation is the timing of host lysis by bacteriophage (phage).
      </p>
      <h4>Results</h4>
      <p style="line-height:160%">Individual lysis events of thermally-inducible λ lysogens were observed using a temperature-controlled
         perfusion chamber mounted on an inverted microscope. Both mean lysis time (MLT) and
         its associated standard deviation (SD) were estimated. Using the SD as a measure of
         lysis time stochasticity, we showed that lysogenic cells in controlled environments
         varied widely in lysis times, and that the level of lysis time stochasticity depended
         on allelic variation in the holin sequence, late promoter (<em>p</em><sub><em>R</em></sub><em>'</em>) activity, and host growth rate. In general, the MLT was positively correlated with
         the SD. Both lower <em>p</em><sub><em>R</em></sub><em>' </em>activities and lower host growth rates resulted in larger SDs. Results from premature
         lysis, induced by adding KCN at different time points after lysogen induction, showed
         a negative correlation between the timing of KCN addition and lysis time stochasticity.
      </p>
      <h4>Conclusions</h4>
      <p style="line-height:160%">Taken together with results published by others, we conclude that a large fraction
         of λ lysis time stochasticity is the result of random events following the expression
         and diffusion of the holin protein. Consequently, factors influencing the timing of
         reaching critical holin concentrations in the cell membrane, such as holin production
         rate, strongly influence the mean lysis time and the lysis time stochasticity.
      </p>
   </div>
</section>
<section><a name="sec1"></a><h3>Background</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Some phenotypic variation arises from randomness in cellular processes despite identical
         environments and genotypes <a name="d8574e142"></a><a name="d8574e144"></a><a name="d8574e146"></a><a name="d8574e148"></a><a name="d8574e150"></a><a name="d8574e152"></a><a name="d8574e154"></a><a name="d8574e156"></a><a name="d8574e158"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>-<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>]. Population heterogeneity, resulting from such molecular stochasticity, has been
         documented in many microbial organisms including bacteriophage (phage) λ <a name="d8574e162"></a><a name="d8574e164"></a><a name="d8574e166"></a><a name="d8574e168"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>-<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>], <em>Escherichia coli </em><a name="d8574e174"></a><a name="d8574e176"></a><a name="d8574e178"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>-<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>], <em>Bacillus subtilis </em><a name="d8574e184"></a><a name="d8574e186"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a></a>,<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>] and <em>Saccharomyces cerevisiae </em><a name="d8574e193"></a><a name="d8574e195"></a><a name="d8574e197"></a><a name="d8574e199"></a><a name="d8574e201"></a><a name="d8574e203"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>-<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>]. This within-population variation can have far reaching life history consequences.
         For example, experimentally reducing noise in the expression of ComK decreased the
         number of competent <em>B. subtilis </em>cells in one study <a name="d8574e210"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. In another study, mutants of <em>S. cerevisiae </em>showing greater heterogeneity in survival had higher rates of occasional-cell survival
         during high stress conditions than did wild-type cells <a name="d8574e217"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>].
      </p>
      <p style="line-height:160%">Because of their simplicity and ease of manipulation, phages are excellent models
         to explore the life history consequences of molecular stochasticity. Many phages use
         a "holin-endolysin" system to compromise two physical barriers, the cell membrane
         and the peptidoglycan layer, in order to lyse an infected host cell <a name="d8574e223"></a><a name="d8574e225"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a></a>,<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>]. Although there are some variations on the theme, holin usually forms a hole(s) in
         the inner membrane, thus either allowing soluble endolysin into the periplasmic space
         <a name="d8574e229"></a><a name="d8574e231"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a></a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>] or activating the membrane-tethered endolysin already translocated to the periplasm
         <a name="d8574e235"></a><a name="d8574e237"></a><a name="d8574e239"></a>[<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>-<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>]. Endolysin then digests the peptidoglycan, causing host cell lysis.
      </p>
      <p style="line-height:160%">The most extensively studied lysis system is that of phage l, which consists of four
         genes: <em>S </em>(encodes holin and antiholin), <em>R </em>(encodes endolysin), <em>Rz</em>, and <em>Rz1 </em>(encode an integral inner membrane protein and an outer membrane lipoprotein, respectively).
         All genes are co-transcribed from the late promoter <em>p</em><sub><em>R</em></sub><em>' </em>during the late phase of the lytic cycle <a name="d8574e266"></a><a name="d8574e268"></a><a name="d8574e270"></a><a name="d8574e272"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a></a>,<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>,<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a></a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]. Under typical laboratory conditions, only <em>S </em>and <em>R </em>are needed for host lysis, though both <em>Rz </em>and <em>Rz1 </em>are essential in the presence of high concentrations of divalent cations <a name="d8574e289"></a><a name="d8574e291"></a><a name="d8574e293"></a>[<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>-<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>].
      </p>
      <p style="line-height:160%">The lytic pathway of phage λ is commonly divided into the early, delayed early, and
         late phases. Transitions between stages are triggered by well-characterized molecular
         actions involving gene transcription and translation <a name="d8574e299"></a>[<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>]. Consequently, the timing of when individual cells enter each phase greatly influences
         the length of individual lysis times. A recent study by Amir <em>et al. </em><a name="d8574e305"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>] showed that 69% of the total lysis time variance is due to variation in the time
         interval between the onset of the <em>p</em><sub><em>R</em></sub><em>' </em>promoter and the eventual lysis (see APPENDIX A). This observation suggests that a
         large portion of the observed lysis time stochasticity is a <em>de novo </em>phenomenon, confined to the production and accumulation of holin proteins in the cell
         membrane, rather than a direct carryover from the various upstream stochastic events.
      </p>
      <p style="line-height:160%">The formation of the λ holin hole in the membrane is hypothesized to be a multi-step
         process that starts with the transcription of the late mRNA and the translation of
         the S holin protein. The resulting holin monomers are then inserted into the cell
         membrane, where they dimerize, then oligomerize <a name="d8574e322"></a>[<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>], eventually leading to the formation of higher-order holin aggregates, or rafts,
         in the cell membrane. At a time that is specific to the holin protein sequence, the
         holin rafts are transformed into a membrane lesion(s) &gt; 300 nm across <a name="d8574e326"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>], which is large enough for the passage of a 500 KDa protein <a name="d8574e330"></a><a name="d8574e332"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a></a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. Lysis ensues after endolysin digests the peptidoglycan. Thus, by regulating endolysin's
         access to the peptidoglycan, holin controls the timing of lysis <a name="d8574e336"></a><a name="d8574e338"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a></a>,<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>].
      </p>
      <p style="line-height:160%">To formalize the heuristic model of holin hole formation described by Wang <em>et al. </em><a name="d8574e346"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>], Ryan and Rutenberg <a name="d8574e350"></a>[<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>] proposed a two-stage nucleation model, in which the production rate of the holin
         monomers and holin self-affinity contribute to the aggregation of holin rafts. Raft
         aggregation is opposed by thermal Brownian motion which tends to disintegrate rafts
         into their holin constituents. As the rafts grow and then exceed a certain critical
         size (the first stage of nucleation), the probability of a second stage nucleation
         (triggering to hole formation) increases (Figure <a name="d8574e353"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F1">1</a>). According to this model, lysis time stochasticity is the inevitable outcome of
         each infected cell in the population following its own time course of growth in holin
         raft size. However, a recent study <a name="d8574e357"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>] using C-terminus GFP-fused λ S holin protein showed that, for most of the latent
         period, holin proteins are distributed uniformly in a relatively mobile state in the
         cell membrane. At a time that coincided with the triggering time, large immobile holin
         rafts suddenly appeared in the membrane. The transition from uniformly distributed
         holin to holin rafts occurred in less than a minute. Although it is not clear whether
         these large rafts correspond to the membrane holes observed by cryoelectron microscopy
         <a name="d8574e361"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>], this study nevertheless casts doubt on the previously hypothesized importance of
         holin raft size growth as the determining factor in lysis timing <a name="d8574e366"></a><a name="d8574e368"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>]. Rather, it is proposed that the lysis time is determined by when a critical holin
         concentration is reached in the cell membrane (Figure <a name="d8574e371"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F1">1</a>). According to this model, lysis time stochasticity is mainly the result of variation
         in the timing of reaching the critical holin concentration in the membrane.
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F1"><img align="top" src="/content/figures/1471-2180-11-174-1.gif" alt="thumbnail" class="thumbnail"><strong>Figure 1.</strong></a> <strong>Schematic presentation of two models of holin hole formation</strong>. Holin monomers (shaded circles) are produced in the cytoplasm, and then transported
         to the cell membrane (a top-down view of the cell membrane thereafter) where they
         dimerize. A previous model (open arrows) <a name="d8574e387"></a><a name="d8574e389"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>] hypothesized that the growth of the holin aggregates ("rafts") to a critical size
         that is responsible for the collapse of the proton motive force (pmf), thus resulting
         in hole formation. The current model <a name="d8574e393"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>] (filled arrows) suggests that the holin proteins are mostly in a mobile state, then
         quite suddenly, aggregates are formed, leading to the formation of holin holes. Addition
         of the energy poison KCN halts further holin production and abolishes the pmf. This
         figure is adapted from Wang <em>et al. </em><a name="d8574e399"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>] and White <em>et al. </em><a name="d8574e405"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>].
      </div></div>
      <p style="line-height:160%">Typically, the lysis time of a phage is estimated using a one-step growth curve <a name="d8574e412"></a><a name="d8574e414"></a><a name="d8574e416"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>-<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>]. In the case of phage λ, however, the availability of thermally-inducible <em>E. coli </em>λ lysogens allows a more precise determination of the lysis time by following the
         decline of culture turbidity <a name="d8574e423"></a><a name="d8574e425"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]. Direct observation of the lysis of individual λ lysogenic cells <a name="d8574e429"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>] confirmed that the precipitous decline of culture turbidity, commonly observed among
         thermally-induced λ lysogen cultures, is a reflection of the saltatory nature of individual
         lysis events at the microscopic level. However, it is not clear to what extent the
         seemingly high synchronicity of lysis is influenced by various aspects of phage biology
         and host growth conditions. In this study, we used a simple experimental setup to
         assess how lysis time stochasticity is affected by allelic variation in the S protein,
         late promoter <em>p</em><sub><em>R</em></sub><em>' </em>activity, host growth rate, and the timing of energy poison KCN addition. Our results
         establish the ranges and limits of lysis time stochasticity under various conditions.
      </p>
   </div>
</section>
<section><a name="sec2"></a><h3>Results</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Using a microscope-mounted, temperature-controlled perfusion chamber, we observed
         and recorded individual lysis events of thermally-induced <em>Escherichia coli </em>l lysogens (Figure <a name="d8574e450"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F2">2A</a>). These observations revealed a considerable amount of variation in lysis time for
         the wild-type (WT) λ phage (Table <a name="d8574e453"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T1">1</a>; Figure <a name="d8574e456"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F2">2B</a>). Although the mean lysis time for the WT λ phage was 65.1 min, lysis times for individual
         lysogenic cells ranged from 45.4 to 74.5 min. Given that phage progeny accumulate
         linearly at ~7.7 phage per minute beginning ~28 min after lysis induction <a name="d8574e460"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>], the ~30 min range of lysis times could result in a three-fold difference in burst
         size between phages that lyse early and those that lyse late. This result motivated
         further exploration of variation in lysis time among other λ strains.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T1','T1',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T1"><strong>Table 1.</strong></a> Effects of holin allelic sequences on the stochasticity of lysis time.
      </div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F2"><img align="top" src="/content/figures/1471-2180-11-174-2.gif" alt="thumbnail" class="thumbnail"><strong>Figure 2.</strong></a> <strong>Samples of a lysis recording and frequency distributions of various experimental treatments</strong>. (A) Sample recordings from strain IN63. It takes about 5 sec for the upper left
         cell to disappear from view. (B) Sample frequency distributions of lysis times from
         strains IN56, IN67, IN68, SYP028, IN56 with KCN added at 55 min after thermal induction,
         and IN56 grown in glycerol minimal salts medium. The bin size was 2 min. Additional
         data are shown in Tables 1 and 2.
      </div></div>
      <h4>Effect of allelic variation in holin sequence</h4>
      <p style="line-height:160%">It has long been known that different holin alleles show different lysis times <a name="d8574e711"></a><a name="d8574e713"></a><a name="d8574e715"></a>[<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>,<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a></a>,<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>]. However, it is not clear to what extent allelic differences in holin protein would
         affect the lysis timing of individual cells. To gain further insight, we determined
         the MLTs (mean lysis times) and SDs (standard deviations) of lysis time for 14 isogenic
         l lysogens differing in their <em>S </em>holin sequences (see APPENDIX B for our rationale for using SD as the measure for
         lysis time stochasticity). The directly observed MLTs (Table <a name="d8574e721"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T1">1</a>) were longer than those reported previously <a name="d8574e725"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>]. This discrepancy was mainly due to the fact that, in previous work, lysis time was
         defined by the time point when the turbidity of the lysogen culture began to decline,
         whereas in our current measurement, it was the mean of all individual lysis times
         observed for a particular phage strain.
      </p>
      <p style="line-height:160%">Figure <a name="d8574e730"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3A</a> revealed a significant positive relationship between MLT and SD (<em>F</em><sub>[1,12]</sub> = 8.42, <em>p </em>= 0.0133). However, we did not observe a significant relationship between MLT and
         another commonly used measure of stochasticity, the coefficient of variation (CV,
         defined as SD/MLT; <a name="d8574e742"></a><a name="d8574e744"></a><a name="d8574e746"></a><a name="d8574e748"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a></a>,<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>]) (<em>F</em><sub>[1,12]</sub> = 1.50, <em>p </em>= 0.2445), indicating a proportional increase of the SD with the MLT. Figure <a name="d8574e760"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3A</a> also reveals a relatively scattered relationship between the MLTs and the SDs (adjusted
         <em>R</em><sup>2 </sup>= 0.363), with several instances in which strains with similar MLTs are accompanied
         by very different SDs. For example, the mean lysis times for IN56 and IN71 were 65.1
         and 68.8 min, but the SDs were 3.2 and 7.7 min, respectively. Apparently the observed
         positive relationship is only a general trend, not an absolute. The scattering of
         the plot also suggests that different missense mutations in the holin sequence can
         influence MLT and SD somewhat independently.
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3"><img align="top" src="/content/figures/1471-2180-11-174-3.gif" alt="thumbnail" class="thumbnail"><strong>Figure 3.</strong></a> <strong>Factors influencing λ lysis time stochasticity</strong>. (A) Effect of allelic variation in holin proteins on mean lysis times (MLTs) and
         standard deviations (SDs). (B) Effect of λ's late promoter <em>p</em><sub><em>R</em></sub><em>' </em>activity <a name="d8574e789"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>] on MLTs, SDs and CVs (coefficients of variation). Solid curve is <em>SD </em>= 3.05 (72.73 + <em>P</em>)/<em>P</em>, where <em>P </em>was the <em>p</em><sub><em>R</em></sub><em>' </em>activity. (C) Effects of <em>p</em><sub><em>R</em></sub><em>' </em>activity and host growth rate on lysis time stochasticity. The regression line was
         obtained by fitting all data points from the late promoter activity (filled diamonds)
         and lysogen growth rate (open squares) treatments, except for the datum with the longest
         MLT and largest SD (from SYP028 in Table 2). (D) Effect of lysogen growth rate on
         MLT, SD, and CV. The fitted solid line shows the relationship between the growth rate
         and SD. All data are from Tables 1 and 2. Symbols: open circles, MLT; close circles,
         SD; closed triangles, CV.
      </div></div>
      <h4>Effect of late promoter <em>p</em><sub><em>R</em></sub><em>' </em>activity
      </h4>
      <p style="line-height:160%">Transcription of the late genes, including the holin gene <em>S</em>, from the <em>p</em><sub><em>R</em></sub><em>' </em>promoter marks the beginning of the late stage of the λ lytic development <a name="d8574e848"></a>[<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>]. Since the major determinant of lysis time is thought to be when a critical holin
         concentration is reached in the cell membrane <a name="d8574e852"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>], reduced promoter activity should not only lengthen the lysis time, as shown in a
         previous study <a name="d8574e857"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>], but should also increase the lysis time stochasticity <a name="d8574e861"></a><a name="d8574e863"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a></a>,<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>].
      </p>
      <p style="line-height:160%">As shown in Figure <a name="d8574e868"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3B</a>, our data showed a negative relationship between the <em>p</em><sub><em>R</em></sub><em>' </em>activity, and the MLTs, SDs, and CVs. However, the increase of the <em>p</em><sub><em>R</em></sub><em>' </em>activity had a diminishing influence on both the MLTs, as has been shown previously
         <a name="d8574e889"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>], and the associated SDs and CVs (see Table <a name="d8574e892"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>). Interestingly, linear regressions (Figure <a name="d8574e895"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3C</a>) showed a much tighter, positive relationship between the MLTs and the SDs (<em>F</em><sub>[1,3]</sub> = 81.04, <em>p </em>= 0.0029; adjusted <em>R</em><sup><em>2 </em></sup>= 0.952; <em>y </em>= -15.7 + 0.3<em>x</em>) and a significant positive relationship between the MLTs and CVs (<em>F</em><sub>[1,3]</sub> = 14.51, <em>p </em>= 0.0318, result not shown in the figure). That is, for the WT <em>S </em>gene, every 1 minute increase in the MLT corresponds to 0.3 minute increase in lysis
         time stochasticity.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','T2',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2"><strong>Table 2.</strong></a> Effect of late promoter activity, lysogen growth rate and KCN addition on the stochasticity
         of lysis time.
      </div></div>
      <h4>Effect of Host Growth Rates</h4>
      <p style="line-height:160%">In general, cells growing at a faster rate have higher concentrations of various biosynthesis
         machineries <a name="d8574e1270"></a>[<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>]. Since the expression of the phage holin gene is entirely dependent on the host,
         we hypothesized that a lower host growth rate would lead to a lower rate of holin
         protein synthesis, thus resulting in a longer lysis time and increased lysis time
         stochasticity. In the phage T4, it was shown that lysis time was negatively correlated
         with host growth rate <a name="d8574e1274"></a>[<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a>].
      </p>
      <p style="line-height:160%">We determined the MLTs and SDs for wild-type l lysogen grown in four different growth
         media: standard LB (lysogeny broth <a name="d8574e1280"></a>[<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>]), 20% LB, Davis minimal salts medium (DM) with 20 mM glucose, and DM with 40 mM glycerol,
         resulting in growth rates of 1.01 ± 0.07, 0.93 ± 0.05, 0.49 ± 0.04, and 0.35 ± 0.01
         h<sup>-1 </sup>(mean ± 95% confidence limits), respectively (see Table <a name="d8574e1286"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>). As shown in Figure <a name="d8574e1289"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3D</a>, lower growth rates led to increased lysis time SDs (<em>F</em><sub>[1,2]</sub> = 24.50, <em>p </em>= 0.0385) and CVs (<em>F</em><sub>[1,2]</sub> = 46.24, <em>p </em>= 0.0209). A similar negative relationship was also apparent for the MLTs. However,
         because of the case of the LB medium, in which the higher growth rate actually resulted
         in a slightly longer MLT, the observed negative relationship was not significant (<em>F</em><sub>[1,2]</sub> = 6.44, <em>p </em>= 0.1265). Interestingly, neither the SDs (<em>F</em><sub>[1,2]</sub> = 16.11, <em>p </em>= 0.0568) nor the CVs (<em>F</em><sub>[1,2]</sub> = 6.04, <em>p </em>= 0.133) was significantly associated with the MLTs.
      </p>
      <h4>Effects of KCN Addition</h4>
      <p style="line-height:160%">The energy poison potassium cyanide, KCN, has long been used in phage research to
         trigger premature lysis <a name="d8574e1341"></a>[<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>]. Typically, after KCN addition, culture turbidity declines precipitously <a name="d8574e1345"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>], indicating that individual lysis events are relatively synchronous. The KCN-induced
         premature lysis is thought to be mediated through a collapsed proton motive force
         (PMF) resulting from a inhibition of the bacterial respiratory chain. As has been
         shown with λ S holin, a 40% drop in the PMF triggers lysis <a name="d8574e1349"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>]. Without a constant supply of ATP, the production of holin protein would also be
         terminated. If KCN is added soon after thermal induction of the lysogen culture, few
         holin proteins would have been made before the termination of holin production. Consequently,
         it should take a longer time for the holin proteins in the membrane to transition
         from a diffused state to aggregated rafts. Therefore, after the cessation of holin
         production by KCN addition, it may take a longer time, on average, before any lysis
         events are observed. On the other hand, if KCN is added late, a larger proportion
         of the thermally-induced lysogenic cells should have accumulated enough holin proteins
         in the cell membrane such that they could be triggered to form holin holes quickly.
         That is, the addition of KCN should prompt the rapid formation of holin holes, thus
         resulting in an almost immediate and synchronous lysis of most of the cells in the
         population. Based on the aforementioned scenarios, we expected that (1) the time delay
         between the time of KCN addition (<em>t</em><sub>KCN</sub>) and the eventual mean lysis time (<em>t</em><sub>L</sub>) (<em>i.e.</em>, <em>t</em><sub>L </sub>- <em>t</em><sub>KCN</sub>) would be negatively correlated with the timing of KCN addition, and (2) <em>t</em><sub>KCN </sub>would be negatively correlated with lysis time stochasticity.
      </p>
      <p style="line-height:160%">Figure <a name="d8574e1384"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F4">4A</a> shows a significant negative relationship between <em>t</em><sub>L </sub>- <em>t</em><sub>KCN </sub>and <em>t</em><sub>KCN</sub>. As KCN was added later in time (<em>i. e.</em>, closer to the normal lysis time of 65.1 min), the time delay between addition of
         KCN and the MLT was reduced (a quadratic fit, <em>F</em><sub>[2,4]</sub> = 12.87, <em>p </em>= 0.0181, adjusted <em>R</em><sup>2 </sup>= 0.798). In fact, when added 55 min after induction (<em>i.e.</em>, 10 min before the normal MLT), the time delay was only 2.6 min, almost instantaneous
         when compared to the 2 min sampling rate of the sipper-equipped spectrophotometer
         method of lysis time determination <a name="d8574e1424"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>]. Interestingly, a theoretical study of lysis time stochasticity by Ryan and Rutenberg
         also showed a seemingly convex relationship between <em>t</em><sub>L </sub>- <em>t</em><sub>KCN </sub>and <em>t</em><sub>KCN </sub>[<a name="d8574e1444"></a>[<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>], their figure five].
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F4"><img align="top" src="/content/figures/1471-2180-11-174-4.gif" alt="thumbnail" class="thumbnail"><strong>Figure 4.</strong></a> <strong>Effects of <em>t</em><sub>KCN </sub>(timing of KCN addition)</strong>. (A) On time delay <em>t</em><sub>L </sub>- <em>t</em><sub>KCN</sub>. The solid curve shows the quadratic fit of <em>y </em>= 54.52 - 1.09<em>x </em>+ 0.02(<em>x </em>- 36.57)<sup>2</sup>. Error bars indicate the associated SDs. As an example, when <em>t</em><sub>KCN </sub>= 45 min, the observed <em>t</em><sub>L </sub>is 50.11 min, thus the time delay is <em>t</em><sub>L </sub>- <em>t</em><sub>KCN </sub>= 5.11 min. (B) On lysis time SD (closed circles) and CV (closed triangles). Solid
         curve shows the quadratic fit of SD against <em>t</em><sub>KCN </sub>(<em>y </em>= 13.24 - 0.28<em>x </em>+ 0.01(<em>x </em>- 36.57)<sup>2</sup>).
      </div></div>
      <p style="line-height:160%">The effects of <em>t</em><sub>KCN </sub>on lysis time SDs and CVs are shown in Figure <a name="d8574e1539"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F4">4B</a>. Again, we witnessed the expected pattern of a significant negative relationship
         between <em>t</em><sub>KCN </sub>and the SDs (a quadratic fit, <em>F</em><sub>[2,4]</sub> = 9.91, <em>p </em>= 0.0123, adjusted <em>R</em><sup>2 </sup>= 0.748) and between <em>t</em><sub>KCN </sub>and the CVs (a quadratic fit, <em>F</em><sub>[2,4]</sub> = 16.03, <em>p </em>= 0.0282, adjusted <em>R</em><sup>2 </sup>= 0.834). These results showed that the later in time KCN was added, the less variation
         there was in individual lysis times. In fact, the lowest SD (1.45 min) and lowest
         CV (2.53%) were observed when KCN was added 55 min after induction. This was a significant
         two-fold reduction in the SD when compared normal lysis conditions (see Table <a name="d8574e1580"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T1">1</a> for strain IN56 with the SD = 3.24 min; Student's <em>t </em>= 15.45, <em>p </em>&lt; 0.0001, using the standard deviation for the SD in Box 7.1 of <a name="d8574e1591"></a>[<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>]). This observation indicated that individual triggering for hole formation during
         the normal progression of cell lysis was relatively asynchronous when compared to
         the artificial method of acute triggering by KCN addition.
      </p>
      <p style="line-height:160%">Similar to the effect of growth rate, a linear regression of the SDs (<em>F</em><sub>[1,5]</sub> = 0.60, <em>p </em>= 0.4726) or CVs (<em>F</em><sub>[1,5]</sub> = 0.328, <em>p </em>= 0.5917) against the MLTs did not yield significant result. Another interesting aspect
         of the relationship between <em>t</em><sub>KCN </sub>and the lysis time SDs is that the SDs drop precipitously when KCN is added about
         35 min after induction. This observation suggests that, approximately 35 min after
         thermal induction, the majority of the lysogenic cells have accumulated enough holin
         proteins in the cell membrane to form holes immediately if triggered.
      </p>
   </div>
</section>
<section><a name="sec3"></a><h3>Discussion</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">The current model of holin hole formation hypothesizes that λ phage lysis timing is
         mainly determined by when a critical concentration of holin proteins is reached in
         the cell membrane <a name="d8574e1625"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>] (Figure <a name="d8574e1628"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F1">1</a>, dark arrows). According to this model, any factor that influences the holin protein
         production should also affect the timing of lysis. Furthermore, the realized rate
         of holin production in each cell should also be subjected to stochastic influences
         impacting the various upstream biochemical reactions, such as gene transcription and
         translation, that lead to holin production. As has been shown by others, the lower
         the average rates of the biochemical reactions, the more prominent the cell-to-cell
         variation is <a name="d8574e1632"></a><a name="d8574e1634"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a></a>,<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>].
      </p>
      <h4>Manipulation of holin production rate</h4>
      <p style="line-height:160%">In our study, we manipulated the holin production rate by manipulating the λ <em>p</em><sub><em>R</em></sub><em>' </em>activity and the lysogen growth rate. We observed that, in general, treatments expected
         to result in higher holin production rates (<em>e.g.</em>, high <em>p</em><sub><em>R</em></sub><em>' </em>activity or high lysogen growth rate) also resulted in shorter MLTs and smaller SDs
         (Figure <a name="d8574e1663"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3B</a> and <a name="d8574e1666"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3D</a>). Furthermore, it was surprising that the combined MLTs and SDs, despite being from
         two different experimental treatments, namely <em>p</em><sub><em>R</em></sub><em>' </em>activity and lysogen growth rate, showed almost identical positive correlations, even
         after excluding the far-flung data point with the longest MLT and largest SD (obtained
         with strain SYP028, see Table <a name="d8574e1677"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>) from the analysis (Figure <a name="d8574e1680"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F3">3C</a>). This result suggests that, irrespective of how the MLT was achieved, as long as
         the MLTs are the same, we should expect to observe similar SDs. For the wild-type
         λ S holin sequence, any factor that results in 1.0 min increase in MLT would be accompanied
         by a concomitant 0.3 min increase in the SD. It would be interesting to conduct a
         similar experiment with different holin sequences to see if the rate of SD increase
         is sequence-specific.
      </p>
      <p style="line-height:160%">Regarding the effects of host growth rate on lysis time stochasticity, it is interesting
         to note the following. Amir <em>et al. </em><a name="d8574e1688"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>] found that the MLTs, SDs, and CVs, following UV induction, ranged from 72 min, 9
         min, and 12.5% respectively for λ lysogens alone to 99 min, 14 min, and 14.1% respectively
         for λ lysogens carrying pR-GFP reporter plasmid and 117 min, 19 min, and 15.8% respectively
         for λ lysogens carrying pR'-tR'-GFP reporter plasmid (all values are extracted from
         their figures six A and B). Since their λ lysogens were grown in M9 minimal salts
         medium plus various growth factors and 0.4% glucose at 37°C, it is similar to our
         Davis minimal salts medium with glucose, from which we obtained the comparable values
         of 70.3 min, 6.3 min, and 8.96% respectively (see Table <a name="d8574e1691"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>). It is not clear whether the difference between these two SDs is the result of different
         methods used for lysogen induction (thermal vs. UV induction) or different growth
         media, but the MLTs are virtually identical. Their result also indirectly confirmed
         our current result that host physiology (which is presumably somewhat perturbed in
         their lysogen strains carrying the medium-copy reporter plasmids) would affect the
         overall MLTs and SDs of lysis time.
      </p>
      <h4>Manipulation of holin protein sequence</h4>
      <p style="line-height:160%">Barring potential post-translational modifications due to differences in holin protein
         sequence (<em>e.g.</em>, differential rate in proteolysis), isogenic λ strains expressing different holin
         sequences would have a similar average rate of holin accumulation in the membrane
         and consequently the same distribution of holin proteins among the cells across different
         lysogen populations. That is, at any given moment, we would expect a certain proportion
         of cells to accumulate a certain number of holin molecules in the membrane, irrespective
         of the holin sequences. The observed differences in MLTs, as the result of differences
         in holin sequence, can be seen as a reflection of different set-points for critical
         concentrations in an increasingly crowded cell membrane <a name="d8574e1704"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>]. Presumably, a sequence with a lower set-point would not only result in a shorter
         MLT, but also a smaller SD as well. However, the existence of similar MLTs, but very
         different SDs, suggests that missense mutations in the holin sequence not only affect
         the set-point for spontaneous triggering, but also impact the robustness of the set-point.
         For example, some mutations may be relatively insensitive to the critical holin concentration,
         thus resulting in proportionally more cells that are triggered earlier and later than
         expected, hence greater lysis time stochasticity.
      </p>
      <h4>Effect of energy poison KCN</h4>
      <p style="line-height:160%">It is well known that addition of the energy poison, KCN, to induced lysogen cultures
         will accelerate the onset of lysis <a name="d8574e1714"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]. Our results also confirmed this observation (see Table <a name="d8574e1717"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>). However, it is not clear how this accelerated lysis would affect the lysis time
         stochasticity. From anecdotal observations, the addition of KCN seems to synchronize
         lysis, thus resulting in a precipitous decline of lysogen culture turbidity. Our study
         showed that the timing of KCN addition was inversely related to lysis time stochasticity
         (see Figure <a name="d8574e1720"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F4">4B</a>). In fact, the smallest SD (1.45 min) was achieved by adding KCN at 55 min after
         thermal induction (see Table <a name="d8574e1723"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>), a time where normally only about 1% of the cells have lysed. The almost synchronous
         lysis when KCN was added 55 min post thermal induction suggests that most cells would
         have already accumulated enough holin proteins in the cell membrane to form a hole.
      </p>
      <p style="line-height:160%">Besides collapsing the PMF, the addition of KCN should also halt the production of
         holin protein, thus "fixes" the amount of holin proteins on the cell membrane at the
         time of addition. The progressive decline in lysis time stochasticity as KCN was added
         later in time (see Figure <a name="d8574e1728"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F4">4B</a>) strongly suggests that a larger supply of holin protein is a key factor in ensuring
         synchronous lysis. As more holin proteins are inserted into the cell membrane, the
         kinetics of raft formation gradually shifts from stochastic to deterministic and synchronous.
         In fact, there was a nearly five-fold decrease in lysis time stochasticity when the
         PMF was collapsed at 55 min after lysogen induction when compared to collapse at 25
         min (see Table <a name="d8574e1731"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>). It is also noted that the properties of the normally triggered and the prematurely
         triggered holin holes are quite distinct, with the prematurely triggered holes being
         much smaller than the normally triggered holes <a name="d8574e1735"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>].
      </p>
      <h4>Evolutionary implication of lysis time stochasticity</h4>
      <p style="line-height:160%">Both theoretical and experimental studies have demonstrated the importance of lysis
         timing on phage fitness <a name="d8574e1745"></a><a name="d8574e1747"></a><a name="d8574e1749"></a><a name="d8574e1751"></a><a name="d8574e1753"></a><a name="d8574e1755"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>,<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>-<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>]. However, it is not clear if lysis time stochasticity would have any impact on phage
         fitness. All else being equal, genotypes with reduced variances in offspring number
         would, in the long run, have higher fitness than genotypes that have the same mean
         offspring number, but larger variances <a name="d8574e1759"></a><a name="d8574e1761"></a>[<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a></a>,<a onclick="LoadInParent('#B63'); return false;" href="#B63">63</a>]. Overall, this suggests that natural selection would tend to minimize stochasticity
         in phenotypes that are closely linked to Darwinian fitness. If the phage burst size
         is positively linked with the lysis time, as has been shown previously <a name="d8574e1765"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>], then selection for reduced burst size stochasticity should lead to reduced lysis
         time stochasticity as well. Presumably, this hypothesis can be tested by competing
         two isogenic phage strains that have the same MLTs but very different lysis time SDs.
         Interestingly, inspection of Table <a name="d8574e1768"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T1">1</a> revealed that mutations introduced into WT λ holin sequence usually result in increased
         stochasticity, except in one case. It is not clear if this observation implies that
         the WT holin sequences have already been selected for reduced stochasticity in the
         wild as well. Experiments with more phage holins should provide some hints in this
         respect.
      </p>
   </div>
</section>
<section><a name="sec4"></a><h3>Conclusions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Even in a seemingly uniform environment, the lysis time can vary greatly among individual
         λ lysogenic cells (lysis time stochasticity). The extent of stochasticity, as quantified
         by the standard deviation, depends on the quality (due to isogenic λ lysogens expressing
         different S protein alleles) and quantity (manipulated by having different <em>p</em><sub><em>R</em></sub><em>' </em>activities and lysogen growth rates) of the holin protein, the major determinant of
         lysis timing in large-genome phages. There is a general positive trend between the
         mean lysis time and the degree of stochasticity. However, this positive relationship
         is much tighter when difference in mean lysis time is due to holin quantity rather
         than quality. The pattern of lysis time stochasticity obtained by addition of KCN
         at various time points after lysogen induction showed a negative relationship between
         the timing of KCN addition and the level of lysis time stochasticity.
      </p>
   </div>
</section>
<section><a name="sec5"></a><h3>Appendix A</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">This section provides the rationale for partitioning lysis time variance found in
         the study by Amir <em>et al. </em><a name="d8574e1794"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>].
      </p>
      <p style="line-height:160%">For each UV-induced λ lysogenic cell, the lysis time <em>T </em>can be divided into three time intervals: (1) <em>t</em><sub>1</sub>, the time interval between lysogen induction and the onset of <em>p</em><sub><em>R </em></sub>promoter, (2) <em>t</em><sub>2</sub>, the time interval between the onset of the <em>p</em><sub><em>R </em></sub>promoter and the onset of the <em>p</em><sub><em>R</em></sub><em>' </em>promoter, and (3) <em>t</em><sub>3</sub>, the time interval between the onset of the <em>p</em><sub><em>R</em></sub><em>' </em>promoter and the eventual lysis. The following relationships describe the above time
         intervals and the empirically determined time intervals by Amir <em>et al. </em><a name="d8574e1850"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>]: <em>t</em><sub>1 </sub>= <em>t</em><sub>pR</sub>, <em>t</em><sub>1 </sub>+ <em>t</em><sub>2 </sub>= <em>t</em><sub>pR'-tR'</sub>, <em>t</em><sub>1 </sub>+ <em>t</em><sub>2 </sub>+ <em>t</em><sub>3 </sub>= <em>t</em><sub>lysis</sub>, and <em>t</em><sub>3 </sub>= Δ<em>t </em>= <em>t</em><sub>lysis </sub>- <em>t</em><sub>pR'-tR'</sub>.
      </p>
      <p style="line-height:160%">For, <em>T </em>= <em>t</em><sub>1 </sub>+ <em>t</em><sub>2 </sub>+ <em>t</em><sub>3</sub>, the variance for the lysis time can be expressed as <em>VAR</em>(<em>T</em>) = <em>VAR</em>(<em>t</em><sub>1</sub>) + <em>VAR</em>(<em>t</em><sub>2</sub>) + <em>VAR</em>(<em>t</em><sub>3</sub>) + 2<em>COV </em>(<em>t</em><sub>1</sub>, <em>t</em><sub>2</sub>) + 2<em>COV </em>(<em>t</em><sub>2</sub>, <em>t</em><sub>3</sub>) + 2<em>COV </em>(<em>t</em><sub>1</sub>, <em>t</em><sub>3</sub>). While the authors did not provide all possible combinations of covariance, it is
         empirically determined that <em>COV</em>(<em>t</em><sub>1 </sub>+ <em>t</em><sub>2</sub>, <em>t</em><sub>3</sub>) = 0, as shown in their figure seven E (<em>i.e.</em>, no correlation between <em>t</em><sub>pR'-tR' </sub>and Δ<em>t</em>). That is, <em>COV</em>(<em>t</em><sub>1 </sub>+ <em>t</em><sub>2</sub>, <em>t</em><sub>3</sub>) = <em>COV</em>(<em>t</em><sub>1</sub>, <em>t</em><sub>3</sub>) + <em>COV</em>(<em>t</em><sub>2</sub>, <em>t</em><sub>3</sub>) = 0. Although not empirically demonstrated, it seems unlikely that the timing of
         turning on either the <em>p</em><sub><em>R </em></sub>or <em>p</em><sub><em>R</em></sub><em>' </em>promoter would have a positive or negative effect on the assembly of lysis apparatus
         such that their effects would cancel each other out, resulting in the observed <em>COV</em>(<em>t</em><sub>1</sub>, <em>t</em><sub>3</sub>) + <em>COV</em>(<em>t</em><sub>2</sub>, <em>t</em><sub>3</sub>) = 0. Most likely, time intervals are mutually independent, <em>i.e.</em>, <em>COV</em>(<em>t</em><sub>1</sub>, <em>t</em><sub>3</sub>) = <em>COV</em>(<em>t</em><sub>2</sub>, <em>t</em><sub>3</sub>) = 0.
      </p>
      <p style="line-height:160%">The standard deviations ("absolute noise" in their terminology) for <em>t</em><sub>pR'-tR' </sub>and <em>t</em><sub>lysis </sub>can be extracted from their figure six A using data determined from cells carrying
         the pR'-tR'-GFP plasmid. The estimated SDs for <em>t</em><sub>pR'-tR' </sub>and <em>t</em><sub>lysis </sub>are ~10 min and ~18 min, respectively; therefore, <em>VAR</em>(<em>t</em><sub>pR'-tR'</sub>) = ~100 and <em>VAR</em>(<em>t</em><sub>lysis</sub>) = ~324. The SD for <em>t</em><sub>pR </sub>can be estimated by extrapolating the line connecting between lysis and <em>p</em><sub><em>R</em></sub><em>' </em>onset to the 20 min mean time at the <em>x</em>-axis (based on the result from cells carrying the pR-GFP plasmid in their figure
         six A). The corresponding SD for <em>t</em><sub>pR </sub>is ~7 min, thus <em>VAR</em>(<em>t</em><sub>pR</sub>) = ~49. Taken together, <em>VAR</em>(<em>t</em><sub>1</sub>) = 49, <em>VAR</em>(<em>t</em><sub>2</sub>) = 51 (= <em>VAR</em>(<em>t</em><sub>1 </sub>+ <em>t</em><sub>2</sub>) - <em>VAR</em>(<em>t</em><sub>1</sub>) = 100 - 49 ), and <em>VAR</em>(<em>t</em><sub>3</sub>) = 224 (= <em>VAR</em>(<em>t</em><sub>1 </sub>+ <em>t</em><sub>2 </sub>+ <em>t</em><sub>3</sub>) - <em>VAR</em>(<em>t</em><sub>1 </sub>+ <em>t</em><sub>2</sub>) = 324 - 100). That is, <em>VAR</em>(<em>t</em><sub>1</sub>), <em>VAR</em>(<em>t</em><sub>2</sub>), and <em>VAR</em>(<em>t</em><sub>3</sub>) contributed to 15%, 16%, and 69% of total lysis time variance, respectively.
      </p>
   </div>
</section>
<section><a name="sec6"></a><h3>Appendix B</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Studies of molecular stochasticity typically use the coefficient of variation (CV)
         as the measurement for the degree of stochasticity <a name="d8574e2344"></a><a name="d8574e2346"></a><a name="d8574e2348"></a><a name="d8574e2350"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a></a>,<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>]. Since CV is a composite statistic (defined as standard deviation/mean), it is sometimes
         difficult to discern whether an increase in the observed stochasticity (as quantified
         by CV) is due to decrease in mean or increase in SD. In some cases, a different metric,
         such as phenotypic noise strength (defined as variance/mean) <a name="d8574e2354"></a><a name="d8574e2356"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>], or a slight variant of it (defined as variance/squared mean) <a name="d8574e2360"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>], has been used as well. Many times, it is not clear why a particular metric is used,
         except in the instance where the phenotypic noise strength is used to test against
         an <em>a priori </em>expectation of a Poisson distribution, for which variance/mean = 1.
      </p>
      <p style="line-height:160%">It is understandable why the CV, or a variant, is used in certain situations. For
         example, if the means are drastically different from each other or a comparison is
         made between measurements using different units [<a name="d8574e2369"></a>[<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>], pp. 57-59.]. In our study, however, the means were not very different and the same
         measuring unit (<em>i.e.</em>, min) was used. Therefore, we presented our means and SDs separately and then jointly
         as CVs. Except in one instance where presenting stochasticity as SD or CV makes a
         difference (<em>i.e.</em>, effect of genotype on SD or CV vs. MLT), all the other results showed that SD and
         CV followed the same trend. Since CV can be derived from SD and mean, no information
         is lost by presenting them separately. Furthermore, when the stochasticity (or noise
         distribution) is not apparent, it may be advisable to present results as means and
         SDs (and if necessary, higher moments like skewness and kurtosis) separately, so that
         the impact of various experimental treatments on the noise distribution can be fully
         appreciated.
      </p>
   </div>
</section>
<section><a name="sec7"></a><h3>Methods</h3>
   <div class="collapsible-content">
      <h4>Bacterial strains</h4>
      <p style="line-height:160%">All bacteria and phage strains used in this study are listed in Table <a name="d8574e2388"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T3">3</a>. The copy number of λ genome was checked by PCR following the method of Powell <em>et al. </em><a name="d8574e2394"></a>[<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>].
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T3','T3',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T3"><strong>Table 3.</strong></a> Bacterial strains used in this study.
      </div></div>
      <h4>Experimental instrumentation</h4>
      <p style="line-height:160%"><em>E. coli </em>cells lysogenic for λ phage were induced and observed to lyse in a temperature-controlled
         perfusion chamber. The experimental apparatus consisted of a 250 mL side-arm (on bottom)
         medium bottle clamped to an elevated support with tubing leading to an inline heater
         (SH-27B, Warner Instruments, New Haven, CT) that was controlled by a dual channel
         heater controller (TC-344B, Warner Instruments, New Haven, CT). The growth medium,
         flowing at a rate of ~1 mL/min (driven by gravity) and heated by the inline heater
         to the desired temperature, was introduced to a 358 μL perfusion chamber (RC-21B,
         Warner Instruments, New Haven, CT) mounted on a heating platform (PM2, Warner Instruments,
         New Haven, CT) that was controlled by the same dual channel heater controller to maintain
         the desired temperature. The internal temperature of the perfusion chamber was independently
         monitored by a thermistor. Waste flowed out of the perfusion chamber, pooled in a
         reservoir, and was siphoned into a 2 L bottle by a vacuum source. Both the perfusion
         chamber and the heating platform were placed on the stage of an inverted microscope
         (TS100, Nikon) for observation at 400× magnification. One of the microscope's ocular
         lenses was replaced with a 10X MiniVID™ microscope camera (LW Scientific, Norcross,
         GA) to record individual lysis events onto a laptop computer at the rate of 1 frame
         per second. All data were collected in unit of seconds, though the results were presented
         in minutes.
      </p>
      <h4>Sample preparation and lysis time determination</h4>
      <p style="line-height:160%">Lysogens were cultured overnight in LB or minimal salts media (see below) at 30°C
         on a rolling drum. Stationary phase cultures were diluted 100-fold in LB or minimal
         salts media, then grown to A<sub>550 </sub>~ 0.2. 200 μL of exponentially growing cells were immobilized on a 22 mm square glass
         coverslip that has been pretreated with 0.01% tissue-culture tested poly-L-lysine
         (mol. wt. 150 K - 300 K, Sigma, St. Louis, MO) at room temperature for 30 min. After
         assembling the perfusion chamber, the device was immediately placed on the heating
         platform and infused with heated medium to maintain the chamber temperature at 30°C
         for 30 min to stabilize the cells. To induce lysis, the chamber temperature was raised
         to 42°C for 15 min, and then dropped to 37°C for the duration of the observation period
         (<em>i.e.</em>, until ~95% of cells are lysed). Video recording was initiated at the time when the
         temperature was raised to 42°C. Under these conditions, it usually takes less than
         5 min for the temperature to rise from 30°C to 42°C, a transition comparable to shifting
         culture flasks from a 30°C to 42°C waterbath shaker. Some experiments were performed
         by adding KCN to the growth medium in the sidearm feeder bottle to a final concentration
         of 20 mM.
      </p>
      <p style="line-height:160%">Videos were subsequently analyzed using Windows Media Player™ playback. The times
         of individual lysis events were then noted visually and recorded manually. The lysis
         time was defined as the time from the initiation of the first temperature shift to
         when the image of the cell disappeared from view. In general, it takes about a few
         seconds (frames) for lysing cells to fully disappear from view (Figure <a name="d8574e2817"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/figure/F1">1A</a>).
      </p>
      <h4>Determination of lysogen growth rate</h4>
      <p style="line-height:160%">Lysogen growth rate was manipulated by using different growth medium formulations:
         (i) full-strength LB (10 g tryptone, 5 g yeast extract, 10 g NaCl per L dH<sub>2</sub>O), (ii) one-fifth-strength LB (2 g tryptone, 1 g yeast extract, 10 g NaCl per L dH<sub>2</sub>O), (iii) 20 mM glucose in Davis minimal salts (7 g K<sub>2</sub>HPO<sub>4</sub>, 2 g KH<sub>2</sub>PO<sub>4</sub>, 1 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.5 g sodium citrate•2H<sub>2</sub>O, and 0.2 g MgSO<sub>4</sub>•7H<sub>2</sub>O), and (iv) 40 mM glycerol in Davis minimal salts. We assessed the growth of the
         lysogen strain IN56 by culturing it overnight at 30°C in each growth media. The next
         day, 90 μL of the overnight culture was used to inoculate 25 mL growth medium and
         the culture was placed in a 30°C waterbath shaker at 220 rpm. Culture growth was followed
         with a sipper-equipped spectrophotometer at A<sub>550</sub>. The growth rate was calculated as the slope of the linear regression of natural-logarithm
         transformed A<sub>550 </sub>values over time.
      </p>
      <h4>Statistical analysis</h4>
      <p style="line-height:160%">In most cases, data collection for a given strain or treatment spanned several days.
         Therefore, even for the same lysogen strain or experimental treatment the means and/or
         variances may be significantly different among data collected from different dates.
         We conducted a preliminary exploration of our data set using the software package
         JMP version 7.0.2, as implemented in MacOS operating system. For each lysogen strain
         or experimental treatment, the means and standard deviations (SDs) were extracted
         from the data set according to the date the data were collected and were treated as
         replicates. Pairwise comparisons of the means (using the Tukey-Kramer HSD test) showed
         that, for more than half of the cases, at least one mean was significantly different
         from the others. Since we were mainly interested in the variation, we subsequently
         converted all values into their corresponding residuals (centered by their corresponding
         means). We also tested the homogeneity of variance from each date replicate, using
         O'Brien's test, Brown-Forsythe test, Levene's test, and Bartlett's test, all implemented
         in JMP. Not surprisingly, more than half of the cases showed that at least one replicate
         variance was significantly different from the others. Although we did not have an
         <em>a priori </em>expectation of lysis time distribution, we nonetheless tested to see if the lysis
         time in each replicate is normally distributed or not, using the Shapiro-Wilk W test.
         Again, in many cases, the replicates do not show a normal distribution. Despite variability
         in our data set, none of our conclusions were fundamentally changed. Therefore, for
         the presented results, the mean and standard deviation for each lysogen strain or
         experimental treatment were calculated based on the following criteria: (<em>i</em>) if the means and variances were the same among all blocks, then all the data would
         be pooled together to estimate the combined means and SDs, (<em>ii</em>) if the means were significantly different, but the variances were the same among
         all blocks, then the mean would be estimated by averaging the block means while the
         SDs would be estimated by pooled residuals, and (<em>iii</em>) if the means and variances were significantly different among all blocks, then the
         means and SDs would be estimated by averaging block means and SDs. For details of
         our data set, see additional file <a name="d8574e2888"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/suppl/S1">1</a>.
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/suppl/S1','S1',630,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/suppl/S1"></a><strong>Additional file 1.</strong> <strong>Sample sizes and standard deviations</strong>. More detailed data sets for both Table <a name="d8574e2900"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T1">1</a> and Table <a name="d8574e2903"></a><a onclick="popup('http://www.biomedcentral.com/1471-2180/11/174/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2180/11/174/table/T2">2</a>.</p><p>
         	Format: DOC 
         	Size: 86KB <a href="/content/supplementary/1471-2180-11-174-s1.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div></div>
   </div>
</section>
<section><a name="sec8"></a><h3>Competing interests</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">The authors declare that they have no competing interests.</p>
   </div>
</section>
<section><a name="sec9"></a><h3>Authors' contributions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">JJD was responsible for conducting all the relevant experiments, data analyses, and
         the preparation of the manuscript. INW was responsible for the supervision, data analyses,
         and preparation of the manuscript. Both authors read and approved the final manuscript.
      </p>
   </div>
</section>
<section><a name="ack"></a><h3>Acknowledgements</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">The authors are grateful for insightful comments from Tom Caraco, Andrew Rutenberg,
         Gillian Ryan, Samuel Sheppard and several anonymous reviewers. The authors would also
         like to thank Yongping Shao for the initial setup of the experimental apparatus and
         Kuangnan Xiong for technical assistance. This work was supported by grant GM072815
         from the National Institutes of Health to INW. During manuscript preparation, JJD
         was supported by grants from the Professional Staff Congress of the City University
         of New York and the National Science Foundation (Division of Environmental Biology
         Award #0804039 and Division of Molecular and Cellular Biosciences Award #0918199).
      </p>
   </div>
</section>
<section><a name="refs"></a><h3>References</h3>
   <div id="article-references" class="collapsible-content">
      <ol id="references">
         <li id="B1">
            <p><a name="B1"></a> Avery SV: <strong> Microbial cell individuality and the underlying sources of heterogeneity. </strong><p><em>Nat Rev Microbiol</em> 2006,  <strong>4</strong><strong>:</strong>577-587.  <a target="_blank" href="/pubmed/16845428">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16845428">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B2">
            <p><a name="B2"></a> Longo D,  Hasty J: <strong> Dynamics of single-cell gene expression. </strong><p><em>Mol Syst Biol</em> 2006,  <strong>2</strong><strong>:</strong>64.  <a target="_blank" href="/pubmed/17130866">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17130866">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17130866">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B3">
            <p><a name="B3"></a> Losick R,  Desplan C: <strong> Stochasticity and cell fate. </strong><p><em>Science</em> 2008,  <strong>320</strong><strong>:</strong>65-68.  <a target="_blank" href="/pubmed/18388284">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18388284">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18388284">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B4">
            <p><a name="B4"></a> Rao CV,  Wolf DM,  Arkin AP: <strong> Control, exploitation and tolerance of intracellular noise. </strong><p><em>Nature</em> 2002,  <strong>420</strong><strong>:</strong>231-237.  <a target="_blank" href="/pubmed/12432408">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12432408">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B5">
            <p><a name="B5"></a> Raser JM,  O'Shea EK: <strong> Noise in gene expression: origins, consequences, and control. </strong><p><em>Science</em> 2005,  <strong>309</strong><strong>:</strong>2010-2013.  <a target="_blank" href="/pubmed/16179466">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16179466">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16179466">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B6">
            <p><a name="B6"></a> Davidson CJ,  Surette MG: <strong> Individuality in bacteria. </strong><p><em>Annu Rev Genet</em> 2008,  <strong>42</strong><strong>:</strong>253-268.  <a target="_blank" href="/pubmed/18652543">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18652543">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B7">
            <p><a name="B7"></a> Fraser D,  Kaern M: <strong> A chance at survival: gene expression noise and phenotypic diversification strategies. </strong><p><em>Mol Microbiol</em> 2009,  <strong>71</strong><strong>:</strong>1333-1340.  <a target="_blank" href="/pubmed/19220745">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19220745">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B8">
            <p><a name="B8"></a> McAdams HH,  Arkin A: <strong> It's a noisy business! Genetic regulation at the nanomolar scale. </strong><p><em>Trends Genet</em> 1999,  <strong>15</strong><strong>:</strong>65-69.  <a target="_blank" href="/pubmed/10098409">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10098409">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B9">
            <p><a name="B9"></a> Veening JW,  Smits WK,  Kuipers OP: <strong> Bistability, epigenetics, and bet-hedging in bacteria. </strong><p><em>Annu Rev Microbiol</em> 2008,  <strong>62</strong><strong>:</strong>193-210.  <a target="_blank" href="/pubmed/18537474">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18537474">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B10">
            <p><a name="B10"></a> Amir A,  Kobiler O,  Rokney A,  Oppenheim AB,  Stavans J: <strong> Noise in timing and precision of gene activities in a genetic cascade. </strong><p><em>Mol Syst Biol</em> 2007,  <strong>3</strong><strong>:</strong>71.  <a target="_blank" href="/pubmed/17299413">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17299413">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17299413">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B11">
            <p><a name="B11"></a> Arkin A,  Ross J,  McAdams HH: <strong> Stochastic kinetic analysis of developmental pathway bifurcation in phage λ-infected
                  <em>Escherichia coli </em>cells. </strong><p><em>Genetics</em> 1998,  <strong>149</strong><strong>:</strong>1633-1648.  <a target="_blank" href="/pubmed/9691025">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9691025">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9691025">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B12">
            <p><a name="B12"></a> Pearl S,  Gabay C,  Kishony R,  Oppenheim A,  Balaban NQ: <strong> Nongenetic individuality in the host-phage interaction. </strong><p><em>PLoS Biol</em> 2008,  <strong>6</strong><strong>:</strong>e120.  <a target="_blank" href="/pubmed/18494559">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18494559">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18494559">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B13">
            <p><a name="B13"></a> St-Pierre F,  Endy D: <strong> Determination of cell fate selection during phage lambda infection. </strong><p><em>Proc Natl Acad Sci USA</em> 2008,  <strong>105</strong><strong>:</strong>20705-20710.  <a target="_blank" href="/pubmed/19098103">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19098103">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19098103">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B14">
            <p><a name="B14"></a> Cai L,  Friedman N,  Xie XS: <strong> Stochastic protein expression in individual cells at the single molecule level. </strong><p><em>Nature</em> 2006,  <strong>440</strong><strong>:</strong>358-362.  <a target="_blank" href="/pubmed/16541077">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16541077">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B15">
            <p><a name="B15"></a> Elowitz MB,  Levine AJ,  Siggia ED,  Swain PS: <strong> Stochastic gene expression in a single cell. </strong><p><em>Science</em> 2002,  <strong>297</strong><strong>:</strong>1183-1186.  <a target="_blank" href="/pubmed/12183631">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12183631">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B16">
            <p><a name="B16"></a> Ito Y,  Toyota H,  Kaneko K,  Yomo T: <strong> How selection affects phenotypic fluctuation. </strong><p><em>Mol Syst Biol</em> 2009,  <strong>5</strong><strong>:</strong>264.  <a target="_blank" href="/pubmed/19401676">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19401676">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19401676">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B17">
            <p><a name="B17"></a> Ozbudak EM,  Thattai M,  Kurtser I,  Grossman AD,  van Oudenaarden A: <strong> Regulation of noise in the expression of a single gene. </strong><p><em>Nat Genet</em> 2002,  <strong>31</strong><strong>:</strong>69-73.  <a target="_blank" href="/pubmed/11967532">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11967532">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B18">
            <p><a name="B18"></a> Maamar H,  Raj A,  Dubnau D: <strong> Noise in gene expression determines cell fate in <em>Bacillus subtilis</em>. </strong><p><em>Science</em> 2007,  <strong>317</strong><strong>:</strong>526-529.  <a target="_blank" href="/pubmed/17569828">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17569828">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B19">
            <p><a name="B19"></a> Bar-Even A,  Paulsson J,  Maheshri N,  Carmi M,  O'Shea E,  Pilpel Y,  Barkai N: <strong> Noise in protein expression scales with natural protein abundance. </strong><p><em>Nat Genet</em> 2006,  <strong>38</strong><strong>:</strong>636-643.  <a target="_blank" href="/pubmed/16715097">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16715097">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B20">
            <p><a name="B20"></a> Blake WJ,  M KA,  Cantor CR,  Collins JJ: <strong> Noise in eukaryotic gene expression. </strong><p><em>Nature</em> 2003,  <strong>422</strong><strong>:</strong>633-637.  <a target="_blank" href="/pubmed/12687005">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12687005">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B21">
            <p><a name="B21"></a> Fraser HB,  Hirsh AE,  Giaever G,  Kumm J,  Eisen MB: <strong> Noise minimization in eukaryotic gene expression. </strong><p><em>PLoS Biol</em> 2004,  <strong>2</strong><strong>:</strong>e137.  <a target="_blank" href="/pubmed/15124029">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15124029">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15124029">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B22">
            <p><a name="B22"></a> Acar M,  Mettetal JT,  van Oudenaarden A: <strong> Stochastic switching as a survival strategy in fluctuating environments. </strong><p><em>Nat Genet</em> 2008,  <strong>40</strong><strong>:</strong>471-475.  <a target="_blank" href="/pubmed/18362885">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18362885">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B23">
            <p><a name="B23"></a> Ansel J,  Bottin H,  Rodriguez-Beltran C,  Damon C,  Nagarajan M,  Fehrmann S,  Francois J,  Yvert G: <strong> Cell-to-cell stochastic variation in gene expression is a complex genetic trait. </strong><p><em>PLoS Genet</em> 2008,  <strong>4</strong><strong>:</strong>e1000049.  <a target="_blank" href="/pubmed/18404214">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18404214">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18404214">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B24">
            <p><a name="B24"></a> Blake WJ,  Balazsi G,  Kohanski MA,  Isaacs FJ,  Murphy KF,  Kuang Y,  Cantor CR,  Walt DR,  Collins JJ: <strong> Phenotypic consequences of promoter-mediated transcriptional noise. </strong><p><em>Mol Cell</em> 2006,  <strong>24</strong><strong>:</strong>853-865.  <a target="_blank" href="/pubmed/17189188">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17189188">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B25">
            <p><a name="B25"></a> Bishop AL,  Rab FA,  Sumner ER,  Avery SV: <strong> Phenotypic heterogeneity can enhance rare-cell survival in 'stress-sensitive' yeast
                  populations. </strong><p><em>Mol Microbiol</em> 2007,  <strong>63</strong><strong>:</strong>507-520.  <a target="_blank" href="/pubmed/17176259">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17176259">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B26">
            <p><a name="B26"></a> Wang IN,  Smith DL,  Young R: <strong> HOLINS: The Protein Clocks of Bacteriophage Infections. </strong><p><em>Annu Rev Microbiol</em> 2000,  <strong>54</strong><strong>:</strong>799-825.  <a target="_blank" href="/pubmed/11018145">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11018145">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B27">
            <p><a name="B27"></a> Young R,  Wang IN,  Roof WD: <strong> Phages will out: strategies of host cell lysis. </strong><p><em>Trends Microbiol</em> 2000,  <strong>8</strong><strong>:</strong>120-128.  <a target="_blank" href="/pubmed/10707065">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10707065">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B27','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B28">
            <p><a name="B28"></a> Wang IN,  Deaton J,  Young R: <strong> Sizing the holin lesion with an endolysin-β-galactosidase fusion. </strong><p><em>J Bacteriol</em> 2003,  <strong>185</strong><strong>:</strong>779-787.  <a target="_blank" href="/pubmed/12533453">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12533453">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12533453">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B29">
            <p><a name="B29"></a> Savva CG,  Dewey JS,  Deaton J,  White RL,  Struck DK,  Holzenburg A,  Young R: <strong> The holin of bacteriophage lambda forms rings with large diameter. </strong><p><em>Mol Microbiol</em> 2008,  <strong>69</strong><strong>:</strong>784-793.  <a target="_blank" href="/pubmed/18788120">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1365-2958.2008.06298.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B29','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B30">
            <p><a name="B30"></a> Park T,  Struck DK,  Dankenbring CA,  Young R: <strong> The pinholin of lambdoid phage 21: control of lysis by membrane depolarization. </strong><p><em>J Bacteriol</em> 2007,  <strong>189</strong><strong>:</strong>9135-9139.  <a target="_blank" href="/pubmed/17827300">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17827300">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17827300">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B30" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B30','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B31">
            <p><a name="B31"></a> Xu M,  Arulandu A,  Struck DK,  Swanson S,  Sacchettini JC,  Young R: <strong> Disulfide isomerization after membrane release of its SAR domain activates P1 lysozyme. </strong><p><em>Science</em> 2005,  <strong>307</strong><strong>:</strong>113-117.  <a target="_blank" href="/pubmed/15637279">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15637279">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B31" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B31','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B32">
            <p><a name="B32"></a> Xu M,  Struck DK,  Deaton J,  Wang IN,  Young R: <strong> A signal-arrest-release sequence mediates export and control of the phage P1 endolysin. </strong><p><em>Proc Natl Acad Sci USA</em> 2004,  <strong>101</strong><strong>:</strong>6415-6420.  <a target="_blank" href="/pubmed/15090650">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15090650">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15090650">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B32" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B32','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B33">
            <p><a name="B33"></a> Zhang N,  Young R: <strong> Complementation and characterization of the nested <em>Rz </em>and <em>Rz1 </em>reading frames in the genome of bacteriophage lambda. </strong><p><em>Mol Gen Genet</em> 1999,  <strong>262</strong><strong>:</strong>659-667.  <a target="_blank" href="/pubmed/10628848">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1007/s004380051128">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B33" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B33','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B34">
            <p><a name="B34"></a> Berry J,  Summer EJ,  Struck DK,  Young R: <strong> The final step in the phage infection cycle: the Rz and Rz1 lysis proteins link the
                  inner and outer membranes. </strong><p><em>Mol Microbiol</em> 2008,  <strong>70</strong><strong>:</strong>341-351.  <a target="_blank" href="/pubmed/18713319">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18713319">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B34" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B34','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B35">
            <p><a name="B35"></a> Young R,  Way J,  Way S,  Yin J,  Syvanen M: <strong> Transposition mutagenesis of bacteriophage lambda: a new gene affecting cell lysis. </strong><p><em>J Mol Biol</em> 1979,  <strong>132</strong><strong>:</strong>307-322.  <a target="_blank" href="/pubmed/160463">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=160463">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B35" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B35','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B36">
            <p><a name="B36"></a> Friedman DI,  Gottesman M: <strong> Lytic mode of lambda development. </strong>In <em>Lambda II</em>.  Edited by Hendrix RW,Roberts JW,Stahl FW,Weisberg RA.  Cold Spring Harbor, New York: Cold Spring Harbor Laboratory;  1983:21-51.  <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B36" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B36','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B37">
            <p><a name="B37"></a> Gründling A,  Bläsi U,  Young R: <strong> Genetic and biochemical analysis of dimer and oligomer interactions of the lambda
                  S holin. </strong><p><em>J Bacteriol</em> 2000,  <strong>182</strong><strong>:</strong>6082-6090.  <a target="_blank" href="/pubmed/11029428">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11029428">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11029428">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B37" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B37','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B38">
            <p><a name="B38"></a> Dewey JS,  Savva CG,  White RL,  Vitha S,  Holzenburg A,  Young R: <strong> Micron-scale holes terminate the phage infection cycle. </strong><p><em>Proc Natl Acad Sci USA</em> 2010,  <strong>107</strong><strong>:</strong>2219-2223.  <a target="_blank" href="/pubmed/20080651">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20080651">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20080651">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B38" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B38','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B39">
            <p><a name="B39"></a> Ryan GL,  Rutenberg AD: <strong> Clocking out: modeling phage-induced lysis of <em>Escherichia coli</em>. </strong><p><em>J Bacteriol</em> 2007,  <strong>189</strong><strong>:</strong>4749-4755.  <a target="_blank" href="/pubmed/17468251">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17468251">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17468251">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B39" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B39','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B40">
            <p><a name="B40"></a> White R,  Chiba S,  Pang T,  Dewey JS,  Savva CG,  Holzenburg A,  Pogliano K,  Young R: <strong> Holin triggering in real time. </strong><p><em>Proc Natl Acad Sci USA</em> 2010,  <strong>108</strong><strong>:</strong>798-803.  <a target="_blank" href="/pubmed/21187415">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21187415">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21187415">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B40" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B40','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B41">
            <p><a name="B41"></a> Ellis EL,  Delbrück M: <strong> The growth of bacteriophage. </strong><p><em>J Gen Physiol</em> 1939,  <strong>22</strong><strong>:</strong>365-384.  <a target="_blank" href="/pubmed/19873108">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1085/jgp.22.3.365">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19873108">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B41" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B41','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B42">
            <p><a name="B42"></a> Delbrück M: <strong> The growth of bacteriophage and lysis of the host. </strong><p><em>J Gen Physiol</em> 1940,  <strong>23</strong><strong>:</strong>643-660.  <a target="_blank" href="/pubmed/19873180">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1085/jgp.23.5.643">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19873180">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B42" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B42','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B43">
            <p><a name="B43"></a> Doermann AH: <strong> The intracellular growth of bacteriophages. I. Liberation of intracellular bacteriophage
                  T4 by premature lysis with another phage or with cyanide. </strong><p><em>J Gen Physiol</em> 1952,  <strong>35</strong><strong>:</strong>645-656.  <a target="_blank" href="/pubmed/14898042">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1085/jgp.35.4.645">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=14898042">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B43" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B43','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B44">
            <p><a name="B44"></a> Young R: <strong> Bacteriophage lysis: mechanism and regulation. </strong><p><em>Microbiol Rev</em> 1992,  <strong>56</strong><strong>:</strong>430-481.  <a target="_blank" href="/pubmed/1406491">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=1406491">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B44" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B44','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B45">
            <p><a name="B45"></a> Gründling A,  Manson MD,  Young R: <strong> Holins kill without warning. </strong><p><em>Proc Natl Acad Sci USA</em> 2001,  <strong>98</strong><strong>:</strong>9348-9352.  <a target="_blank" href="/pubmed/11459934">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11459934">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11459934">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B45" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B45','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B46">
            <p><a name="B46"></a> Wang IN: <strong> Lysis timing and bacteriophage fitness. </strong><p><em>Genetics</em> 2006,  <strong>172</strong><strong>:</strong>17-26.  <a target="_blank" href="/pubmed/16219778">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16219778">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16219778">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B46" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B46','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B47">
            <p><a name="B47"></a> Raab R,  Neal G,  Garrett J,  Grimaila R,  Fusselman R,  Young R: <strong> Mutational analysis of bacteriophage lambda lysis gene S. </strong><p><em>J Bacteriol</em> 1986,  <strong>167</strong><strong>:</strong>1035-1042.  <a target="_blank" href="/pubmed/2943725">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2943725">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=2943725">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B47" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B47','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B48">
            <p><a name="B48"></a> Swain PS,  Elowitz MB,  Siggia ED: <strong> Intrinsic and extrinsic contributions to stochasticity in gene expression. </strong><p><em>Proc Natl Acad Sci USA</em> 2002,  <strong>99</strong><strong>:</strong>12795-12800.  <a target="_blank" href="/pubmed/12237400">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12237400">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12237400">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B48" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B48','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B49">
            <p><a name="B49"></a> Raj A,  Peskin CS,  Tranchina D,  Vargas DY,  Tyagi S: <strong> Stochastic mRNA synthesis in mammalian cells. </strong><p><em>PLoS Biol</em> 2006,  <strong>4</strong><strong>:</strong>1707-1719.  <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B49" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B49','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B50">
            <p><a name="B50"></a> Shao Y,  Wang IN: <strong> Effect of late promoter activity on bacteriophage λ fitness. </strong><p><em>Genetics</em> 2009,  <strong>181</strong><strong>:</strong>1467-1475.  <a target="_blank" href="/pubmed/19171945">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19171945">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19171945">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B50" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B50','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B51">
            <p><a name="B51"></a> Gillespie DT: <strong> Exact stochastic simulation of coupled chemical reactions. </strong><p><em>J Phys Chem</em> 1977,  <strong>81</strong><strong>:</strong>2340-2361.   <a target="_blank" href="http://dx.doi.org/10.1021/j100540a008">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B51" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B51','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B52">
            <p><a name="B52"></a> McAdams HH,  Arkin A: <strong> Stochastic mechanisms in gene expression. </strong><p><em>Proc Natl Acad Sci USA</em> 1997,  <strong>94</strong><strong>:</strong>814-819.  <a target="_blank" href="/pubmed/9023339">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9023339">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9023339">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B52" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B52','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B53">
            <p><a name="B53"></a> Bremer H,  Dennis PP: <strong> Modulation of chemical composition and other parameters of the cell by growth rate. </strong>In <em>Escherichia coli and Salmonella typhimurium Cellular and Molecular Biology</em>. <em> Volume 2</em>.  Edited by Ingraham JL,Low KB,Magasanik B,Schaechter M,Umbarger HE.  Washington, D.C.: American Society for Microbiology;  1987:1527-1542.  <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B53" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B53','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B54">
            <p><a name="B54"></a> Hadas H,  Einav M,  Fishov I,  Zaritsky A: <strong> Bacteriophage T4 development depends on the physiology of its host <em>Escherichia coli</em>. </strong><p><em>Microbiology</em> 1997,  <strong>143</strong><strong>:</strong>179-185.  <a target="_blank" href="/pubmed/9025292">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9025292">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B54" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B54','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B55">
            <p><a name="B55"></a> Bertani G: <strong> Lysogeny at mid-twentieth century: P1, P2, and other experimental systems. </strong><p><em>J Bacteriol</em> 2004,  <strong>186</strong><strong>:</strong>595-600.  <a target="_blank" href="/pubmed/14729683">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14729683">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=14729683">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B55" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B55','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B56">
            <p><a name="B56"></a> Sokal RR,  Rohlf FJ: <em>Biometry. </em> 3rd edition.  New York, New York: W. H. Freeman and Company;  1995.</p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B57">
            <p><a name="B57"></a> Abedon ST: <strong> Selection for bacteriophage latent period length by bacterial density: A theoretical
                  examination. </strong><p><em>Microb Ecol</em> 1989,  <strong>18</strong><strong>:</strong>79-88.   <a target="_blank" href="http://dx.doi.org/10.1007/BF02030117">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B57" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B57','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B58">
            <p><a name="B58"></a> Abedon ST,  Herschler TD,  Stopar D: <strong> Bacteriophage latent-period evolution as a response to resource availability. </strong><p><em>Appl Environ Microbiol</em> 2001,  <strong>67</strong><strong>:</strong>4233-4241.  <a target="_blank" href="/pubmed/11526028">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11526028">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11526028">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B58" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B58','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B59">
            <p><a name="B59"></a> Heineman RH,  Bull JJ: <strong> Testing optimality with experimental evolution: lysis time in a bacteriophage. </strong><p><em>Evolution</em> 2007,  <strong>61</strong><strong>:</strong>1695-1709.  <a target="_blank" href="/pubmed/17598749">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17598749">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17598749">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B59" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B59','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B60">
            <p><a name="B60"></a> Shao Y,  Wang IN: <strong> Bacteriophage adsorption rate and optimal lysis time. </strong><p><em>Genetics</em> 2008,  <strong>180</strong><strong>:</strong>471-482.  <a target="_blank" href="/pubmed/18757924">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18757924">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18757924">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B60" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B60','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B61">
            <p><a name="B61"></a> Wang IN,  Dykhuizen DE,  Slobodkin LB: <strong> The evolution of phage lysis timing. </strong><p><em>Evol Ecol</em> 1996,  <strong>10</strong><strong>:</strong>545-558.   <a target="_blank" href="http://dx.doi.org/10.1007/BF01237884">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B61" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B61','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B62">
            <p><a name="B62"></a> Gillespie JH: <strong> Nautural selection for within-generation variance in offspring number. </strong><p><em>Genetics</em> 1974,  <strong>76</strong><strong>:</strong>601-606.  <a target="_blank" href="/pubmed/4833578">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=4833578">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=4833578">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B62" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B62','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B63">
            <p><a name="B63"></a> Gillespie JH: <strong> Natural selection for variances in offspring numbers: a new evolutionary principle. </strong><p><em>Am Nat</em> 1977,  <strong>111</strong><strong>:</strong>1010-1014.   <a target="_blank" href="http://dx.doi.org/10.1086/283230">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B63" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B63','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B64">
            <p><a name="B64"></a> Powell BS,  Rivas MP,  Court DL,  Nakamura Y,  Turnbough CL Jr: <strong> Rapid confirmation of single copy lambda prophage integration by PCR. </strong><p><em>Nucleic Acids Res</em> 1994,  <strong>22</strong><strong>:</strong>5765-5766.  <a target="_blank" href="/pubmed/7838735">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=7838735">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=7838735">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2180-11-174&amp;bibl=B64" onclick="popup('/sfx_links?ui=1471-2180-11-174&amp;bibl=B64','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
      </ol>
   </div>
</section>
	</div>
	    </div></div>
</div>


		</div>
</div>
    			
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Bottom' begin -->
		<script type="text/javascript">
		OAS_AD('Bottom');

		</script>
		<!-- OAS AD 'Bottom' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcmicrobiol/article/10.1186/1471/2180/11/174/15987221665@Bottom?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcmicrobiol/article/10.1186/1471/2180/11/174/17764508875@Bottom?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
			    <hr class="hide" />
	
    



 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="footer">

									<span class="views"><a href="?fmt_view=mobile">Mobile view</a> | <strong>Desktop view</strong></span>
			
	<div class="content">
    <div class="rounded ">
<div class="wrap-inner ">
        	
    <ul class="desktop">
                                                    
				        <li class="noborder"><a href="http://www.biomedcentral.com/about/tandc">Terms and Conditions</a></li>
<li><a href="http://www.biomedcentral.com/about/privacy">Privacy statement</a></li>
<li><a href="http://www.biomedcentral.com/presscenter">Press</a></li>
	<li><a href="http://www.biomedcentral.com/advertisers">Information for advertisers</a></li>
<li><a href="http://www.biomedcentral.com/about/bmcjobs">Jobs at BMC</a></li>
<li><a href="http://www.biomedcentral.com/support">Support</a></li>
<li><a href="http://www.biomedcentral.com/about/contact">Contact us</a></li>


    
    	    </ul>
    <p id="copyright">&copy; 2013 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business Media.</p>
        </div>
</div>

	</div>

</div>
<div class="springer">
    <a href="http://www.springer.com/" target="blank"><img style="float:right; margin:0;" src="/images/Springer_Branding_footer_logo.png" /></a>
</div>
					    

<script>
// getElementsByClassName Polyfill
var getElementsByClassName=function(e,t,n){if(document.getElementsByClassName){getElementsByClassName=function(e,t,n){n=n||document;var r=n.getElementsByClassName(e),i=t?new RegExp("\\b"+t+"\\b","i"):null,s=[],o;for(var u=0,a=r.length;u<a;u+=1){o=r[u];if(!i||i.test(o.nodeName)){s.push(o)}}return s}}else if(document.evaluate){getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i="",s="http://www.w3.org/1999/xhtml",o=document.documentElement.namespaceURI===s?s:null,u=[],a,f;for(var l=0,c=r.length;l<c;l+=1){i+="[contains(concat(' ', @class, ' '), ' "+r[l]+" ')]"}try{a=document.evaluate(".//"+t+i,n,o,0,null)}catch(h){a=document.evaluate(".//"+t+i,n,null,0,null)}while(f=a.iterateNext()){u.push(f)}return u}}else{getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i=[],s=t==="*"&&n.all?n.all:n.getElementsByTagName(t),o,u=[],a;for(var f=0,l=r.length;f<l;f+=1){i.push(new RegExp("(^|\\s)"+r[f]+"(\\s|$)"))}for(var c=0,h=s.length;c<h;c+=1){o=s[c];a=false;for(var p=0,d=i.length;p<d;p+=1){a=i[p].test(o.className);if(!a){break}}if(a){u.push(o)}}return u}}return getElementsByClassName(e,t,n)}
</script>


<script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>

<script type="text/javascript">
if (typeof jQuery == 'undefined') { document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.min.js' type='text/javascript'%3E%3C/script%3E")); }


document.documentElement.id = "js";

var site = {
			abbreviation: "/bmcmicrobiol",
	        type: "BMC_CORE",
        portalId: "9001",
                section: "browse",
                page: "fulltext",
    		
									homePortal: "biomedcentral.com",
		  

			siteShowAds: "true",
				showAds: "true",
				url: "biomedcentral.com/bmcmicrobiol",
			portal: false,
	        adpage_url: "biomedcentral.com/bmcmicrobiol/article/10.1186/1471/2180/11/174",
        id: "1010",
	name: "BMC Microbiology",
	plugins: {
		scrollable: (getElementsByClassName("scrollable").length > 0),
		flowplayer: ((document.getElementById("flowplayer") != null) || (getElementsByClassName("myPlayer").length > 0) || (document.getElementById("videoarticle") != null) || (getElementsByClassName("flowplayer").length > 0) || (document.getElementById("audio") != null) || (document.getElementById("elifeplayer") != null)),
		lightbox: ((document.getElementById("image-highlight") != null) || (getElementsByClassName("lightbox").length > 0)),
		thickbox: (getElementsByClassName("thickbox").length > 0),
		altmetric: ((document.getElementById("altmetric") != null) || (getElementsByClassName("altmetric").length > 0))
	}
	
};



if(typeof quoteSearchUri == "undefined") { quoteSearchUri=null; }
</script>


        <script type="text/javascript" src="/javascript/journals/behaviours-0.js"></script>







		<script>
	if (site.plugins.scrollable == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.scrollable-1.0.1.min.js' %3E%3C/script%3E")); 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.tools.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.flowplayer == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/flowplayer-3.2.6.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.lightbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/browse_imagehighlight.js'%3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.colorbox.js' %3E%3C/script%3E"));
	}

	if (site.plugins.thickbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-compressed.js' %3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-flashhidden-patch.js' %3E%3C/script%3E"));
	}
	</script>




				        <script type="text/javascript" src="/javascript/plugins/mobile-0.js"></script>
	        <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/jquery-ui.min.js"></script>
	        	        <script src="/javascript/plugins/hammer.js"></script>
		



            <script type="text/javascript" src="/javascript/plugins/jquery.fancybox-1.3.1.pack-0.js"></script>
            <script type="text/javascript" src="/javascript/articles/articles-0.js"></script>
		<div id="fb-root"></div>










<!--[if lte IE 6]>
            <script type="text/javascript" src="/javascript/plugins/DD_belatedPNG_0.0.8a-0.js"></script>
    <script type="text/javascript">
        DD_belatedPNG.fix('.png_bg, ul.primary-nav li a, ul.primary-nav li a span, ul.secondary-nav li a, ul.secondary-nav li a span, ul li i, .plus-button, .facebook, .twitter, .fancy-bg, #fancybox-close');
    </script>
	                <link rel="stylesheet" type="text/css" href="/css/hacks/ie6-0.css"/>
    <![endif]-->

<!-- Claudia Whitcombe wanted this comment here -->

       <div class="hide">
      <dl class="google-ad" id="x96banner">
        <dt style="display: block;" class="hide">
          <a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
       </dt>
        <dd>
          <!-- OAS AD 'x96' begin -->
          <script type="text/javascript">
            OAS_AD('x96');
          </script>
        </dd>
      </dl>
    </div>  
    
</body>
</html>

